{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "# System packages\n",
    "import os\n",
    "import sys\n",
    "import warnings\n",
    "# Data related\n",
    "import numpy as np \n",
    "import pandas as pd \n",
    "\n",
    "# sklearn tools \n",
    "from sklearn.metrics import  accuracy_score\n",
    "from sklearn.model_selection import train_test_split, cross_val_predict, StratifiedKFold\n",
    "from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer, TfidfVectorizer\n",
    "from sklearn.pipeline import FeatureUnion, Pipeline, make_pipeline, make_union\n",
    "from sklearn.preprocessing import FunctionTransformer, OneHotEncoder\n",
    "\n",
    "# sklearn models\n",
    "from sklearn.naive_bayes import MultinomialNB\n",
    "from sklearn.linear_model import SGDClassifier\n",
    "import xgboost as xgb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Using TensorFlow backend.\n"
     ]
    }
   ],
   "source": [
    "warnings.filterwarnings('ignore')\n",
    "pd.set_option('display.max_columns', 50)\n",
    "# Import custom functions\n",
    "from utils_functions import *"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1. Load full data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('../data/processed/train_variants_text.csv')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Use a sample data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "'''\n",
    "sample =df.groupby('Class').apply(lambda x: x.sample(frac=0.2))\n",
    "sample =sample.drop([sample.columns[1], \n",
    "                     sample.columns[2], \n",
    "                     sample.columns[5]], \n",
    "                    axis='columns')\n",
    "sample.dropna(subset=['Text'])\n",
    "sample.head(1)\n",
    "sample.to_csv('../data/processed/train_variants_text_sample.csv')\n",
    "'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample = pd.read_csv('../data/processed/train_variants_text_sample.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Class</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>NF2</td>\n",
       "      <td>L46R</td>\n",
       "      <td>Neurofibromatosis type 2 (NF2) is a multiple n...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>FGFR2</td>\n",
       "      <td>E219K</td>\n",
       "      <td>Introduction  Melanoma is the most lethal of a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>1</td>\n",
       "      <td>BRCA1</td>\n",
       "      <td>F1704S</td>\n",
       "      <td>Abstract  The BRCA1 gene from individuals at ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>1</td>\n",
       "      <td>TP53</td>\n",
       "      <td>R337H</td>\n",
       "      <td>The tumor suppressor protein p53 is a transcri...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>1</td>\n",
       "      <td>TSC2</td>\n",
       "      <td>E75G</td>\n",
       "      <td>Tuberous sclerosis complex (TSC) is an autosom...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Class   Gene Variation                                               Text\n",
       "0      1    NF2      L46R  Neurofibromatosis type 2 (NF2) is a multiple n...\n",
       "1      1  FGFR2     E219K  Introduction  Melanoma is the most lethal of a...\n",
       "2      1  BRCA1    F1704S   Abstract  The BRCA1 gene from individuals at ...\n",
       "3      1   TP53     R337H  The tumor suppressor protein p53 is a transcri...\n",
       "4      1   TSC2      E75G  Tuberous sclerosis complex (TSC) is an autosom..."
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sample.head()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Split sample data into train and validation data set"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 263,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_tr, X_val, y_tr, y_val = split_data(sample,\n",
    "                                      'Text',\n",
    "                                      'Class',\n",
    "                                      0.1,\n",
    "                                      0,\n",
    "                                      stratify='Class')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Take a look at the first y_tr and X_tr\n",
    "#print(y_tr[0], \"-is the predicted Class for text -\", X_tr[0],)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 2. Feature extraction"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.1 Bag of words\n",
    "Here we will use \n",
    " * CountVectorizer: Counts the number of times a word appears in the text\n",
    " * TfidfVectorizer: Weighs the words according to the importance of the word in the context of whole collection\n",
    " "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2.2 Word2Vec"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found w2vmodel\n"
     ]
    }
   ],
   "source": [
    "# Use document df\n",
    "w2vec = get_word2vec(\n",
    "    MySentences(\n",
    "        sample['Text'].values, \n",
    "    ),\n",
    "    'w2vmodel'\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3. Training"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.1 Bag of words + Model"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.1 Naive Bayes classifier for multinomial models\n",
    "Suitable for classification with discrete features (e.g., word counts for text classification). The multinomial distribution normally requires integer feature counts. However, in practice, fractional counts such as tf-idf may also work."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', CountVectorizer(preprocessor=clean_text_stemmed, stop_words =stop_words)),\n",
    "                         ('tfidf', TfidfTransformer()),\n",
    "                         ('clf',  MultinomialNB())])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.44776119402985076\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)\n",
    "#print(classification_report(y_val, predicted))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.2 SGD \n",
    "\n",
    "This estimator implements regularized linear models with stochastic gradient descent (SGD) learning: the gradient of the loss is estimated each sample at a time and the model is updated along the way with a decreasing strength schedule (aka learning rate). SGD allows minibatch (online/out-of-core) learning, see the partial_fit method. For best results using the default learning rate schedule, the data should have zero mean and unit variance."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', CountVectorizer(preprocessor=clean_text_stemmed, stop_words =stop_words)),\n",
    "                ('tfidf', TfidfTransformer()),\n",
    "                ('clf',  SGDClassifier(n_jobs=))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.4925373134328358\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.3 TfidfTransformer + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 261,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Without text cleaning\n",
    "clf = Pipeline([('vect', CountVectorizer()),\n",
    "                ('tfidf', TfidfTransformer()),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 264,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.5223880597014925\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "# With text cleaning\n",
    "clf = Pipeline([('vect', CountVectorizer(preprocessor=clean_text_stemmed, stop_words =stop_words)),\n",
    "                ('tfidf', TfidfTransformer()),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.5671641791044776\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.4 DictVectorizer(Gene) +  DictVectorizer(Variation) + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample['Class'] = sample['Class'].astype(int)\n",
    "sample['Gene'] = sample['Gene'].astype(str)\n",
    "sample['Variation'] = sample['Variation'].astype(str)\n",
    "\n",
    "y= sample['Class']\n",
    "X= sample.drop('Class',axis =1)\n",
    "X_tr, X_val, y_tr, y_val = train_test_split(X,\n",
    "                                            y,\n",
    "                                            test_size=0.2,\n",
    "                                            stratify=y,\n",
    "                                            random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 177,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(\n",
    "        make_pipeline(\n",
    "            PandasSelector([\"Gene\",\"Variation\"]),\n",
    "            PandasToDict(),\n",
    "            DictVectorizer(sparse=False)\n",
    "            # select categorical data\n",
    "        )\n",
    "    \n",
    "    )\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 178,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 179,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = make_pipeline(processing_pipeline, clf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 181,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('pandasselector', PandasSelector(columns=['Gene', 'Variation'])), ('pandastodict', PandasToDict()), ('dictvectorizer', DictVectori...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 181,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 182,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.42105263157894735\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.5 TFIDF(Text) + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 251,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(\n",
    "        \n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "                    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', binary...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 252,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 253,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.631578947368421\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.6 OneHot(Gene) + TFIDF(Text)  + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 235,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(\n",
    "        \n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "                    ),    \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            OneHotEncoder()\n",
    "    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 236,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', bina...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 236,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 237,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6240601503759399\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.7 OneHot(Variation) + TFIDF(Text) + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(    \n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "                    ),\n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Variation\"),\n",
    "            Converter(),\n",
    "            OneHotEncoder()\n",
    "    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', bina...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 248,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.631578947368421\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 254,
   "metadata": {},
   "outputs": [],
   "source": [
    "def model_predict(processing_pipeline,clf,X_tr,y_tr,X_val,y_val):\n",
    "    model = make_pipeline(processing_pipeline, clf)\n",
    "    model.fit(X_tr, y_tr)\n",
    "    predicted = model.predict(X_val)\n",
    "    acc=np.mean(predicted == y_val)\n",
    "    print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_predict(processing_pipeline,clf,X_tr,y_tr,X_val,y_val)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.8 OneHot(Variation) + OneHot(Gene) + TFIDF(Text)  + xgboost\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 241,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline2= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(\n",
    "        \n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "                    ),    \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            OneHotEncoder()\n",
    "        ),\n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Variation\"),\n",
    "            Converter(),\n",
    "            OneHotEncoder()\n",
    "    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 242,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', bina...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 242,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model2 = make_pipeline(processing_pipeline2, clf)\n",
    "model2.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 243,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6240601503759399\n"
     ]
    }
   ],
   "source": [
    "predicted2 = model2.predict(X_val)\n",
    "\n",
    "acc2=np.mean(predicted2 == y_val)\n",
    "print(acc2)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.9 TFIDF(Gene) + TFIDF(Text)  + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(  \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Gene')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', binary...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.3458646616541353\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.10 TFIDF(Variation) +  TFIDF(Gene) +xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(  \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Variation\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "        ),\n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "    )\n",
    "   \n",
    "    )\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Variation')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word',...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.41353383458646614\n"
     ]
    }
   ],
   "source": [
    "# prediction\n",
    "predicted = model.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "150\n",
      "617\n"
     ]
    }
   ],
   "source": [
    "print(sample.Gene.nunique())\n",
    "print(sample.Variation.nunique())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.11 TFIDF(Gene) + TFIDF(Text)  + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(\n",
    "        \n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "                    ),    \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "    )\n",
    ")\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word', bina...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6240601503759399\n"
     ]
    }
   ],
   "source": [
    "predicted = model.predict(X_val)\n",
    "\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.11 make_pipeline + make_unionn & TFIDF(Variation) + TFIDF(Gene) + TFIDF(Text) + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "processing_pipeline= make_pipeline(\n",
    "    # combine features\n",
    "    make_union(  \n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Variation\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "        ),\n",
    "        make_pipeline(\n",
    "            ItemSelector(key=\"Gene\"),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "    ),\n",
    "        make_pipeline(\n",
    "            ItemSelector(key='Text'),\n",
    "            Converter(),\n",
    "            TfidfVectorizer()\n",
    "        )\n",
    "   \n",
    "    )\n",
    ")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('pipeline', Pipeline(memory=None,\n",
       "     steps=[('featureunion', FeatureUnion(n_jobs=None,\n",
       "       transformer_list=[('pipeline-1', Pipeline(memory=None,\n",
       "     steps=[('itemselector', ItemSelector(key='Variation')), ('converter', Converter()), ('tfidfvectorizer', TfidfVectorizer(analyzer='word',...\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1))])"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clf =xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model = make_pipeline(processing_pipeline, clf)\n",
    "model.fit(X_tr, y_tr)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.6240601503759399\n"
     ]
    }
   ],
   "source": [
    "# prediction\n",
    "predicted = model.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.1.12 Pipeline + featureunion + explain _features & TFIDF(Variation) + TFIDF(Gene) + TFIDF(Text) + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "Variation = Pipeline([\n",
    "    ('selector',ItemSelector(key=\"Variation\")),\n",
    "    ('converter',Converter()),\n",
    "    ('tfidf', TfidfVectorizer())\n",
    "])\n",
    "\n",
    "Gene = Pipeline([\n",
    "    ('selector',ItemSelector(key=\"Gene\")),\n",
    "    ('converter',Converter()),\n",
    "    ('tfidf', TfidfVectorizer())\n",
    "])\n",
    "\n",
    "Text = Pipeline([\n",
    "    ('selector',ItemSelector(key=\"Text\")),\n",
    "    ('converter',Converter()),\n",
    "    ('tfidf', TfidfVectorizer())\n",
    "    \n",
    "])\n",
    "\n",
    "ppl = Pipeline([\n",
    "    ('features',FeatureUnion([\n",
    "        ('Variation',Variation),\n",
    "        ('Gene',Gene),\n",
    "        ('Text',Text)\n",
    "    ])),\n",
    "    ('clf',xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))\n",
    "])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = ppl.fit(X_tr,y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.6240601503759399"
      ]
     },
     "execution_count": 117,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Accuracy score \n",
    "model.score(X_val,y_val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<matplotlib.lines.Line2D at 0x1a3e1e1550>]"
      ]
     },
     "execution_count": 167,
     "metadata": {},
     "output_type": "execute_result"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAXcAAAD8CAYAAACMwORRAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAHBhJREFUeJzt3XtwXOd53/Hvs7u4ENgLSVxI7PIugRK5O44lc2QpjlOndlwp9UjtTNpIHXfaTibqTWlcZ9Kx246bun+0dTq9/KF6qthJnNixRnHimpOyUTq13CQdSRVJyTYBSrxJJEGQAHgDsCBx3ad/7AJcgLgsQIBn9+zvM8PB7sHL3Uca8Lcv3uec85q7IyIi4RIJugAREVl/CncRkRBSuIuIhJDCXUQkhBTuIiIhpHAXEQkhhbuISAgp3EVEQkjhLiISQrGg3ri9vd337NkT1NuLiNSkY8eOXXX3jpXGBRbue/bs4ejRo0G9vYhITTKz85WM07KMiEgIKdxFREJI4S4iEkIKdxGREFK4i4iEkMJdRCSEFO4iIiFUc+F+7Px1/v2fvIu2BxQRWVrNhfuJSyN89QdnGRiZCLoUEZGqVXPhnk0nAThxaTjgSkREqlfNhfuBriRmcKJf4S4ispSaC/fWphj72ls5cWkk6FJERKpWzYU7QC6TolczdxGRJdVmuKdT9A+Pcy2vpqqIyGJqMtyzmWJTtadfSzMiIoupzXDvSgFqqoqILKUmwz3V0sDOrZvoUVNVRGRRNRnuUFx318xdRGRxtRvumRTnr91iZHwq6FJERKpOzYb77JWqvWqqiojcpYbDvdRU1W0IRETuUlG4m9mTZvaemZ0xsy8s8v1dZvaamb1tZj8ys59b/1Ln60g0sS3ZpNMhRUQWsWK4m1kUeBF4CjgIPGdmBxcM+5fAK+7+CPAs8F/Xu9DF5NIpzdxFRBZRycz9MeCMu59z90ngZeCZBWMcSJYep4D+9StxadlMirNDeW5PztyPtxMRqRmVhHsGuFj2vK90rNyvA581sz7gCPDLi72QmT1vZkfN7OjQ0NAayp0vl05ScDh5RUszIiLlKgl3W+TYwm2QngN+x913AD8H/J6Z3fXa7v6Sux9y90MdHR2rr3aBXKbYVO3R0oyIyDyVhHsfsLPs+Q7uXnb5ReAVAHd/HWgG2tejwOV0pZrZ0tKg2/+KiCxQSbi/BXSb2V4za6TYMD28YMwF4JMAZnaAYrjf+7rLCsyMXEZXqoqILLRiuLv7NPAC8CpwkuJZMT1m9mUze7o07FeBXzKzHwLfBv6u36cdrLPpFKcGRpmcLtyPtxMRqQmxSga5+xGKjdLyY18qe9wLfGx9S6tMLpNkasY5NTA6twYvIlLvavYK1Vm50pWqPVqaERGZU/PhvmtrC/GmmJqqIiJlaj7cIxHjYDqppqqISJmaD3coLs2cvDzCTOG+9HBFRKpeOMI9k2R8qsC5oXzQpYiIVIWQhLv2VBURKReKcN/X3kpTLKKmqohISSjCPRaNcKArqdv/ioiUhCLcobju3ts/QkFNVRGREIV7OsXoxDQXb9wKuhQRkcCFJ9xnm6padxcRCU+4d2+L0xA1nTEjIkKIwr0pFqW7M6GmqogIIQp3KDZVe/pHuE93GxYRqVohC/cU18cmuTIyHnQpIiKBClW4Z9NqqoqIQMjC/UBXgoihdXcRqXuhCveWxhj7OuLauENE6l6owh0gl05qWUZE6l74wj2T4srIOFfzE0GXIiISmNCFe3ZuT1XN3kWkfoUu3A+mk4CaqiJS30IX7qlNDexua1FTVUTqWujCHSCrpqqI1LmQhnuKC9dvMXx7KuhSREQCEcpwn739b6+aqiJSp0IZ7tlSU1Xr7iJSr0IZ7u3xJrpSzTpjRkTqVijDHUpNVS3LiEidCnG4pzg7lOfW5HTQpYiI3HehDfdcJoU7nLw8GnQpIiL3XYjDXU1VEalfoQ337clm2lob1VQVkboU2nA3M7KZlK5UFZG6FNpwh+IZM6cGRpmYngm6FBGR+yrU4Z5Lp5guOKeu5IMuRUTkvqoo3M3sSTN7z8zOmNkXlhjzN82s18x6zOz317fMtVFTVUTqVWylAWYWBV4EfhboA94ys8Pu3ls2phv4IvAxd79hZp0bVfBq7NraQqI5xgmFu4jUmUpm7o8BZ9z9nLtPAi8DzywY80vAi+5+A8DdB9e3zLUxM93+V0TqUiXhngEulj3vKx0rtx/Yb2b/18zeMLMn16vAe5VNpzh5eYTpmULQpYiI3DeVhLstcswXPI8B3cAngOeAr5nZ5rteyOx5MztqZkeHhoZWW+ua5DJJJqYLnB0auy/vJyJSDSoJ9z5gZ9nzHUD/ImO+5+5T7v4+8B7FsJ/H3V9y90Pufqijo2OtNa9Kbm7DbK27i0j9qCTc3wK6zWyvmTUCzwKHF4z578DPAJhZO8VlmnPrWeha7euI09wQ0bq7iNSVFcPd3aeBF4BXgZPAK+7eY2ZfNrOnS8NeBa6ZWS/wGvBr7n5to4pejWjEONiV1BkzIlJXVjwVEsDdjwBHFhz7UtljBz5f+lN1sukU3337EoWCE4ks1kIQEQmXUF+hOiuXSZKfmOb89VtBlyIicl/URbhn1VQVkTpTF+G+f1uChqipqSoidaMuwr0xFuGh7QnN3EWkbtRFuEPxfPcTl4Yp9n5FRMKtbsI9m05y49YU/cPjQZciIrLh6ifcM8WmqrbdE5F6UDfhfmB7kohBT7+aqiISfnUT7psaozzYGadHM3cRqQN1E+5QaqrqjBkRqQN1Fe4H00kGRiYYHFVTVUTCra7CPZeZvVJV6+4iEm51Fe4H08UNs3sV7iIScnUV7snmBva0teh0SBEJvboKdyie766mqoiEXd2Fey6d4uL12wzfmgq6FBGRDVN34Z4trbvrJmIiEmZ1G+5amhGRMKu7cG+LN5FONet0SBEJtboLdyg1VXXGjIiEWF2Gey6d4tzVMcYmpoMuRURkQ9RluGfTSdzh5GUtzYhIONVluOd0b3cRCbm6DPdtySba441qqopIaNVluJsZ2XSKEwp3EQmpugx3gFwmyemBUcanZoIuRURk3dVvuKdTTBecUwOjQZciIrLu6jbcs+nZpqqWZkQkfOo23Hdu3USiOaZ7zIhIKNVtuJtZaU9VzdxFJHzqNtyh2FQ9eXmEqZlC0KWIiKyrOg/3FJPTBc4O5YMuRURkXdV1uM/d/ldNVREJmboO973tcTY1RHUbAhEJnboO92jEOJhO0qumqoiETF2HO0AunaSnf5hCwYMuRURk3dR9uGczKcYmZ/jg2ljQpYiIrJuKwt3MnjSz98zsjJl9YZlxP29mbmaH1q/EjXVnT1UtzYhIeKwY7mYWBV4EngIOAs+Z2cFFxiWAfwK8ud5FbqTuzgSN0Qg9aqqKSIhUMnN/DDjj7ufcfRJ4GXhmkXH/BvgKML6O9W24xliEh7YndG93EQmVSsI9A1wse95XOjbHzB4Bdrr7Hy/3Qmb2vJkdNbOjQ0NDqy52o+QySU70D+OupqqIhEMl4W6LHJtLQTOLAP8J+NWVXsjdX3L3Q+5+qKOjo/IqN1g2neLmrSku3bwddCkiIuuiknDvA3aWPd8B9Jc9TwA54Adm9gHwOHC4lpqqd/ZU1dKMiIRDJeH+FtBtZnvNrBF4Fjg8+013H3b3dnff4+57gDeAp9396IZUvAEe3p4gGjHd/ldEQmPFcHf3aeAF4FXgJPCKu/eY2ZfN7OmNLvB+aG6I8mBHXE1VEQmNWCWD3P0IcGTBsS8tMfYT917W/ZfNJPmL01eDLkNEZF3U/RWqs3LpFIOjEwyO1NSZnCIii1K4l8w2VbU0IyJhoHAvOdCVANDtf0UkFBTuJYnmBva2t3JCZ8yISAgo3Mtk00kty4hIKCjcy+QyKfpu3ObmrcmgSxERuScK9zK5tJqqIhIOCvcydzbM1rq7iNQ2hXuZLa2NZDZv0sYdIlLzFO4LZEt7qoqI1DKF+wK5TIr3r46Rn5gOuhQRkTVTuC+QyyRxh5OXtTQjIrVL4b7A7BkzaqqKSC1TuC/QmWymPd6kjTtEpKYp3BeRy6ipKiK1TeG+iFw6xenBPONTM0GXIiKyJgr3ReQySWYKzntXRoMuRURkTRTui8jONlW1NCMiNUrhvogdWzaR2tSgpqqI1CyF+yLMTFeqikhNU7gvIZdJ8e6VUaZmCkGXIiKyagr3JWTTSSanC5wZzAddiojIqinclzC7YbauVBWRWqRwX8LetlZaG6PauENEapLCfQmRiHGgK6mZu4jUJIX7MnKZFL2XRygUPOhSRERWReG+jGw6ya3JGd6/NhZ0KSIiq6JwX4aaqiJSqxTuy3iwM05jLKKmqojUHIX7MhqiEQ5sT2jmLiI1R+G+goPpFD39I7irqSoitUPhvoJcJsnw7Sn6btwOuhQRkYop3Fcwu6eqbiImIrVE4b6Ch7YniEZMt/8VkZqicF9Bc0OU7s64Nu4QkZqicK9ANp3SzF1EakpF4W5mT5rZe2Z2xsy+sMj3P29mvWb2IzP732a2e/1LDU4uk+RqfoLBkfGgSxERqciK4W5mUeBF4CngIPCcmR1cMOxt4JC7fwj4DvCV9S40SHNXqmppRkRqRCUz98eAM+5+zt0ngZeBZ8oHuPtr7n6r9PQNYMf6lhmsA11JzNDSjIjUjErCPQNcLHveVzq2lF8E/ue9FFVt4k0x9ra36kpVEakZsQrG2CLHFr1c08w+CxwC/tIS338eeB5g165dFZZYHbLpFMfP3wi6DBGRilQyc+8DdpY93wH0LxxkZp8C/gXwtLtPLPZC7v6Sux9y90MdHR1rqTcwuXSSSzdvc2NsMuhSRERWVEm4vwV0m9leM2sEngUOlw8ws0eA/0Yx2AfXv8zgzTZVdYdIEakFK4a7u08DLwCvAieBV9y9x8y+bGZPl4b9BhAH/sDM3jGzw0u8XM3KppOAzpgRkdpQyZo77n4EOLLg2JfKHn9qneuqOptbGtmxZZOaqiJSE3SF6irkSrf/FRGpdgr3Vcimk7x/dYzR8amgSxERWZbCfRVmm6onL48GXImIyPIU7quQzZSaqlp3F5Eqp3Bfhc5EM52JJp0xIyJVT+G+SrlMih7dY0ZEqpzCfZWy6SRnhvKMT80EXYqIyJIU7quUTaeYKTjvXlFTVUSql8J9lXJqqopIDVC4r1Jm8yY2tzTQo6aqiFQxhfsqmRk57akqIlVO4b4G2UyS966MMjVTCLoUEZFFKdzXIJtOMTlT4PRAPuhSREQWpXBfg5xu/ysiVU7hvgZ72lppbYzSozNmRKRKKdzXIBIxsukUJ3T7XxGpUgr3NcpmkvT2jzBTWHSvcBGRQCnc1yibTnF7aob3r44FXYqIyF0U7ms0e6WqLmYSkWqkcF+jBzviNMUiug2BiFQlhfsaxaIRHu5K6kpVEalKCvd7kEsnOdE/jLuaqiJSXRTu9yCXSTE6Ps3F67eDLkVEZB6F+z3IptVUFZHqpHC/B/u3JYhFTLchEJGqo3C/B80NUbq3JdRUFZGqo3C/R7l0khOX1FQVkeqicL9HuUyKa2OTDIxMBF2KiMgchfs9UlNVRKpRLOgCat2BriRmcOLSCJ88sC3ocqpeoeCcHszz+tmrnBrMs7etlQNdSQ6mk2xtbQy6PJHQULjfo9amGPvaW3XGzBLcnTODeV4/d403zl3jjXPXuT42CUCiKcboxPTc2O3JZg50JTiYThYDvyvJ7rZWohELqnxZxrX8BKcG8ly8cYsHOuLkMkmaYtGgy5IShfs6yGVSvPX+9aDLqAruztmhsbkwf/PcNa7mi2GeTjXziYc6eHxfG0/sa2Pn1hau5Sc4eXmU3svDxa/9I/zZ6atzt1Le1BDl4a4EB7uSczP8h7cnaGnUj+794O5czU9yenCU0wN5Tg+Ocmogz5nB/NyH9KzGaIRcJslHdm/hI7u38OiuLXQmmwOqXPQvZB3k0im+904/18cm625pwd15/+oYb5y7PhfoQ6PF5vL2ZDMf7+7g8X1beWJfOzu3bsJs/iy8Ld7ET3U38VPd7XPHxqdmODOYp/fyCL39I5y8PMLhH/bzrTcvAGDGvOWc2eDflmy66/WlMu7O0OgEpwfznB4Y5dRgnjMDeU4NjnLz1tTcuERzjP3bEnz64Da6tyXo7oyT2bKJ0wN5jl+4wfHzN/jG6+f5zT9/H4AdWzbNC/uHtyeIRdXqux8U7uugvKn68e6OgKvZWO7O+Wu3eOPctbkwnz1TqDPRxE8+0DY3M9/d1rKmsG1uiJLLpMhlUvPe99LN2/T2j9B7uRj4P7p0k//x48tzY7a2NhaXdUqhf6AryQMdcRoUJnPcncHRCU4N3JmJF7/mGb59J8RTmxrYvy3OU7kuujvj7N+WoHtbnM7E4h+gD3TEeTK3HYCJ6Rl6+kc4fv4Gxy/c4PWz1/jeO/0AtDRG+Ykdm4thv3szj+7awuaW+poQ3S8K93WQTRdD6MSlkdCFu7tz8frteWF+eXgcgPZ4U3FWXgr0fe2tGzZzNjN2bGlhx5YWPp3dPnd8ZHyKdy+PcnJ2ln9lhG+8fp7J6QJQXCro3haft6xzoCtJalPDhtRZLdydKyPjnB7Ic2pglDODxa+nB/OMjt/pc2xuaWB/Z4LPfKgY4t2lEO+Ir/23oKZYlEd3FWfqs7Vcunmb4xducvz8DY6dv8FX/8/ZuaW3Bzpa52b2H9m9hQc64kTUZ7lnFtTFN4cOHfKjR48G8t4b4eNf+T4f2rGZF//Wo0GXcs8uXr8zM3/z3HUu3SzeGK2ttZHH97Xx+ANtPLFvKw90xKtyGWR6psD7V8eKyzplSzuza/8Amc2b5jVuD3YlF102qnbuTv/wOKcXBPiZgfy8ZvXW1sZSeBdn4Q92xunuTNAebwzkv/nW5DQ/6hvm2PniUs6xCzfmln+SzTEeLQv7n9i5mXiT5qGzzOyYux9acZzCfX38w28e4+TlEX7waz8TdCmr1n/zNq+fvTYX6H03imG+paWhuMRSmpl3d1ZnmFdqcHS8FPSjc0s754byzG6Dm2iK8XBXgu5tCTY1RIlGjIgZ0QhEzYhE7M7X8sdGcey8YzZ3LDb3OsXXmntc9lrzj7Hg9YvfmykU+xuzSynFdfFRxiZn5v4b2+ONPDi7jNIZ58HOBPu3xWmLNwX0f70ys72bY6WlnOPnb3JqcBR3iBg8vD3Jo7uLyzkf2bW1Jj+I18u6hruZPQn8FyAKfM3d/92C7zcBvwt8BLgG/IK7f7Dca4Yt3F987Qy/8ep7JJpitDbFaG2KEm9uIN4UpbUxRnzueIxEc4zWxiitTfOPx5tixJtjxBuLf3+jGk9Xhsd5/dxV3jhbbIJeuH4LKP6K/tG9W3miNDvf35kI/a/HtydnODVwJ+x7+0c4O5RncrrAjDuFAsy4V91G6O3xJvZvi99ZSil9DVNDf/j2FO9cvMmx8zd4+8IN3r5wk3zpt5H2eBOP7to816zNZVI0N9THaZjrFu5mFgVOAT8L9AFvAc+5e2/ZmH8EfMjd/4GZPQv8dXf/heVeN2zhfjU/wTffOM/w7SnGJqYZm5ghPzHN2MQ0+dKf2eOTM4WKXrMpFpn/oVD60Lj7QyFKvKmh+IFS9kEx+yEzPeO89cH14sz87DU+uFYM82RzjI/uu9MAfXh7+MP8XhQKPhf0hdmvZeE/e2z28XTB5/+dRcbe/ZrM/37Za5oZu9taeLAjzpYQhXilZgrOqYHRstn9jbmf5YaokU2n5p2Zsz0VztMw1zPcnwB+3d3/Sun5FwHc/d+WjXm1NOZ1M4sBV4AOX+bFwxbuqzE5Xbgr9POl4B+bmGZ07oOg/Psz5Cem5sbMfViU/Uq+kkRzjI/u3VpcN9/XxoGupC4Qkpp2NT9ROiun2Kz9Yd9NJkrN9O3JZuLNS6/Vr5R9y353hV/kVvo973Of6uaZD2dWGLW4SsO9ki5FBrhY9rwP+OhSY9x92syGgTbg6oKingeeB9i1a1cFbx1OjbEIjbHGdZl9FQrO2OT83xQWfnAUHB7ZtZlsOqUwl1Bpjzfx6ez2uTOoJqcL9F4unob540vDc2dNLWmFfw7LfXulNf/lvtvWuvE9kErCfbEaF34wVTIGd38JeAmKM/cK3ltWEIkYieYGEs3hPrVPpBKNsQgf3rmZD+/cHHQpgaukY9cH7Cx7vgPoX2pMaVkmBeh6fBGRgFQS7m8B3Wa218wagWeBwwvGHAb+TunxzwPfX269XURENtaKyzKlNfQXgFcpngr5W+7eY2ZfBo66+2Hg68DvmdkZijP2ZzeyaBERWV5Fl325+xHgyIJjXyp7PA78jfUtTURE1kp3VBIRCSGFu4hICCncRURCSOEuIhJCgd0V0syGgPNr/OvtLLj6tUqortVRXatXrbWprtW5l7p2u/uKG0cEFu73wsyOVnJvhftNda2O6lq9aq1Nda3O/ahLyzIiIiGkcBcRCaFaDfeXgi5gCaprdVTX6lVrbaprdTa8rppccxcRkeXV6sxdRESWUXPhbmZPmtl7ZnbGzL4QdD0AZvZbZjZoZieCrqWcme00s9fM7KSZ9ZjZrwRdE4CZNZvZ/zOzH5bq+tdB11TOzKJm9raZ/XHQtcwysw/M7Mdm9o6ZVc0WZma22cy+Y2bvln7OnqiCmh4q/X+a/TNiZp8Lui4AM/unpZ/5E2b2bTPbsL0Aa2pZppL9XAOq66eBPPC77p4LspZyZtYFdLn7cTNLAMeAv1YF/78MaHX3vJk1AH8B/Iq7vxFkXbPM7PPAISDp7p8Juh4ohjtwyN2r6pxtM/sG8Ofu/rXSLcFb3P1m0HXNKmXGJeCj7r7W62rWq5YMxZ/1g+5+28xeAY64++9sxPvV2sz9MeCMu59z90ngZeCZgGvC3f+MKtycxN0vu/vx0uNR4CTFLRED5UX50tOG0p+qmGWY2Q7grwJfC7qWamdmSeCnKd7yG3efrKZgL/kkcDboYC8TAzaVNjVq4e6Nj9ZNrYX7Yvu5Bh5WtcDM9gCPAG8GW0lRaenjHWAQ+F/uXhV1Af8Z+GfACptv3ncO/KmZHSvtRVwN9gFDwG+XlrG+ZmatQRe1wLPAt4MuAsDdLwH/AbgAXAaG3f1PN+r9ai3cK9qrVeYzszjwh8Dn3H0k6HoA3H3G3T9McdvGx8ws8OUsM/sMMOjux4KuZREfc/dHgaeAf1xaCgxaDHgU+Kq7PwKMAVXRBwMoLRM9DfxB0LUAmNkWiisNe4E00Gpmn92o96u1cK9kP1cpU1rT/kPgW+7+R0HXs1Dp1/gfAE8GXArAx4CnS+vbLwN/2cy+GWxJRe7eX/o6CHyX4hJl0PqAvrLfur5DMeyrxVPAcXcfCLqQkk8B77v7kLtPAX8E/ORGvVmthXsl+7lKSalx+XXgpLv/x6DrmWVmHWa2ufR4E8Uf+neDrQrc/YvuvsPd91D82fq+u2/YzKpSZtZaaohTWvb4NBD4mVnufgW4aGYPlQ59Egi0Wb/Ac1TJkkzJBeBxM2sp/dv8JMU+2IaoaJu9arHUfq4Bl4WZfRv4BNBuZn3Av3L3rwdbFVCcif5t4Mel9W2Af17aNjFIXcA3SmcyRIBX3L1qTjusQtuA7xbzgBjw++7+J8GWNOeXgW+VJlvngL8XcD0AmFkLxbPq/n7Qtcxy9zfN7DvAcWAaeJsNvFK1pk6FFBGRytTasoyIiFRA4S4iEkIKdxGREFK4i4iEkMJdRCSEFO4iIiGkcBcRCSGFu4hICP1/DRYeuHgdqoIAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Calculate predict probability for each class\n",
    "y_pred = model.predict_proba(X_val)[0]\n",
    "y_pred \n",
    "\n",
    "# plot the probability \n",
    "plt.plot(y_pred)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('Variation', Pipeline(memory=None,\n",
       "       steps=[('selector', ItemSelector(key='Variation')), ('converter', Converter()), ('tfidf', TfidfVectorizer(analyzer='word', binary=False, decode_error='strict',\n",
       "          dtype=<class 'numpy.float64'>, encoding='utf-8', input='content',\n",
       "          lowercase=True, max_df=1.0, max_features=None, min_df=1,\n",
       "   ...\n",
       "          token_pattern='(?u)\\\\b\\\\w\\\\w+\\\\b', tokenizer=None, use_idf=True,\n",
       "          vocabulary=None))])), ('Gene', Pipeline(memory=None,\n",
       "       steps=[('selector', ItemSelector(key='Gene')), ('converter', Converter()), ('tfidf', TfidfVectorizer(analyzer='word', binary=False, decode_error='strict',\n",
       "          dtype=<class 'numpy.float64'>, encoding='utf-8', input='content',\n",
       "          lowercase=True, max_df=1.0, max_features=None, min_df=1,\n",
       "        ...\n",
       "          token_pattern='(?u)\\\\b\\\\w\\\\w+\\\\b', tokenizer=None, use_idf=True,\n",
       "          vocabulary=None))])), ('Text', Pipeline(memory=None,\n",
       "       steps=[('selector', ItemSelector(key='Text')), ('converter', Converter()), ('tfidf', TfidfVectorizer(analyzer='word', binary=False, decode_error='strict',\n",
       "          dtype=<class 'numpy.float64'>, encoding='utf-8', input='content',\n",
       "          lowercase=True, max_df=1.0, max_features=None, min_df=1,\n",
       "        ...\n",
       "          token_pattern='(?u)\\\\b\\\\w\\\\w+\\\\b', tokenizer=None, use_idf=True,\n",
       "          vocabulary=None))]))]"
      ]
     },
     "execution_count": 96,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model.named_steps['features'].transformer_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "metadata": {},
   "outputs": [],
   "source": [
    "f1=model.named_steps['features'].transformer_list[0][1].named_steps['tfidf'].get_feature_names()\n",
    "f2=model.named_steps['features'].transformer_list[1][1].named_steps['tfidf'].get_feature_names()\n",
    "f3=model.named_steps['features'].transformer_list[2][1].named_steps['tfidf'].get_feature_names()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {},
   "outputs": [],
   "source": [
    "list_features=f1\n",
    "# Add all the elements of f2 and f3 to f1\n",
    "list_features.extend(f2)\n",
    "list_features.extend(f3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 172,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <style>\n",
       "    table.eli5-weights tr:hover {\n",
       "        filter: brightness(85%);\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        <table class=\"eli5-weights eli5-feature-importances\" style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto;\">\n",
       "    <thead>\n",
       "    <tr style=\"border: none;\">\n",
       "        <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">Weight</th>\n",
       "        <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "    </tr>\n",
       "    </thead>\n",
       "    <tbody>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 80.00%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0128\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                unclassified\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 81.47%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0115\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                select\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 83.26%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0099\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                mutagenized\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 83.49%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0098\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                erk\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 85.73%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0079\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                kras\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 85.94%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0078\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                formation\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 86.10%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0076\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                57\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 86.55%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0073\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                partial\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 86.98%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0069\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                25th\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.06%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0069\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                calculate\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "    \n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 87.06%); border: none;\">\n",
       "                <td colspan=\"2\" style=\"padding: 0 0.5em 0 0.5em; text-align: center; border: none; white-space: nowrap;\">\n",
       "                    <i>&hellip; 64611 more &hellip;</i>\n",
       "                </td>\n",
       "            </tr>\n",
       "        \n",
       "    \n",
       "    </tbody>\n",
       "</table>\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "Explanation(estimator=\"XGBClassifier(base_score=0.5, booster='gbtree', colsample_bylevel=1,\\n       colsample_bytree=1, gamma=0, learning_rate=0.1, max_delta_step=0,\\n       max_depth=3, min_child_weight=1, missing=None, n_estimators=100,\\n       n_jobs=1, nthread=None, objective='multi:softprob', random_state=42,\\n       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\\n       silent=True, subsample=1)\", description='\\nXGBoost feature importances; values are numbers 0 <= x <= 1;\\nall values sum to 1.\\n', error=None, method='feature importances', is_regression=False, targets=None, feature_importances=FeatureImportances(importances=[FeatureWeight(feature='unclassified', weight=0.012829316, std=None, value=None), FeatureWeight(feature='select', weight=0.011503947, std=None, value=None), FeatureWeight(feature='mutagenized', weight=0.00994953, std=None, value=None), FeatureWeight(feature='erk', weight=0.0097536715, std=None, value=None), FeatureWeight(feature='kras', weight=0.00792183, std=None, value=None), FeatureWeight(feature='formation', weight=0.0077546854, std=None, value=None), FeatureWeight(feature='57', weight=0.0076285223, std=None, value=None), FeatureWeight(feature='partial', weight=0.007281949, std=None, value=None), FeatureWeight(feature='25th', weight=0.0069469423, std=None, value=None), FeatureWeight(feature='calculate', weight=0.0068843956, std=None, value=None)], remaining=64611), decision_tree=None, highlight_spaces=None, transition_features=None)"
      ]
     },
     "execution_count": 172,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#!pip install eli5\n",
    "from eli5.explain import explain_weights\n",
    "from eli5.formatters import explain_weights_df\n",
    "\n",
    "explain_weights(model.named_steps['clf'], \n",
    "                 vec=None, top=10, \n",
    "                 target_names=ppl.classes_, \n",
    "                 feature_names=list_features)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Save to csv"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Summary\n",
    "1. Fill NA, define dtype \n",
    "2. Feature extraction and Union\n",
    "3. Define model Pipeline\n",
    "4. model = Pipeline.fit(X,y)\n",
    "5. model.score(X_val,y_val)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 205,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample['Class'] = sample['Class'].astype(int)\n",
    "sample['Gene'] = sample['Gene'].astype(str)\n",
    "sample['Variation'] = sample['Variation'].astype(str)\n",
    "\n",
    "y= sample['Class']\n",
    "X= sample.drop('Class',axis =1)\n",
    "X_tr, X_val, y_tr, y_val = train_test_split(X,\n",
    "                                            y,\n",
    "                                            test_size=0.2,\n",
    "                                            stratify=y,\n",
    "                                            random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 175,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2. Feature extraction and Union\n",
    "default_preprocessor = CountVectorizer().build_preprocessor()\n",
    "def build_preprocessor(df,field):\n",
    "    field_idx = list(df.columns).index(field)\n",
    "    return lambda x: default_preprocessor(x[field_idx])\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 213,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 2. Feature extraction and Union\n",
    "vectorizer = FeatureUnion([\n",
    "    ('Variation',TfidfVectorizer(preprocessor=build_preprocessor(X,'Variation'))),\n",
    "        ('Gene',TfidfVectorizer(preprocessor=build_preprocessor(X,'Gene'))),\n",
    "        ('Text',TfidfVectorizer(preprocessor=build_preprocessor(X,'Text'))),\n",
    "])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 214,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_v = vectorizer.fit_transform(X.values)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 215,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_tr, X_val, y_tr, y_val = train_test_split(X_v,\n",
    "                                            y,\n",
    "                                            test_size=0.2,\n",
    "                                            stratify=y,\n",
    "                                            random_state=42)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "XGBClassifier(base_score=0.5, booster='gbtree', colsample_bylevel=1,\n",
       "       colsample_bytree=1, gamma=0, learning_rate=0.1, max_delta_step=0,\n",
       "       max_depth=3, min_child_weight=1, missing=None, n_estimators=100,\n",
       "       n_jobs=1, nthread=None, objective='multi:softprob', random_state=42,\n",
       "       reg_alpha=0, reg_lambda=1, scale_pos_weight=1, seed=None,\n",
       "       silent=True, subsample=1)"
      ]
     },
     "execution_count": 216,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42)\n",
    "model.fit(X_tr, y_tr)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 217,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0.6390977443609023"
      ]
     },
     "execution_count": 217,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Accuracy score \n",
    "model.score(X_val,y_val)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 218,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <style>\n",
       "    table.eli5-weights tr:hover {\n",
       "        filter: brightness(85%);\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        <table class=\"eli5-weights eli5-feature-importances\" style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto;\">\n",
       "    <thead>\n",
       "    <tr style=\"border: none;\">\n",
       "        <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">Weight</th>\n",
       "        <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "    </tr>\n",
       "    </thead>\n",
       "    <tbody>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 80.00%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0135\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__formation\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 84.79%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0091\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__mutagenized\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 86.37%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0078\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__variant\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.42%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0070\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__25th\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.52%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0069\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__calculate\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.74%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0067\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__activated\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.04%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0065\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__kras\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.05%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0065\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__21\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.41%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0062\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__6b\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.53%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0061\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__treatment\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.58%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0061\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__erk\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.71%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0060\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__ability\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.78%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0059\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__kinase\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.92%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0058\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__annotated\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.26%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0055\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__fewer\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.33%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0055\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__cytopenia\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.35%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0055\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__deletions\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.51%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0054\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__nonsense\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.65%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0053\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__partial\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 89.79%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0052\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__this\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "    \n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 89.79%); border: none;\">\n",
       "                <td colspan=\"2\" style=\"padding: 0 0.5em 0 0.5em; text-align: center; border: none; white-space: nowrap;\">\n",
       "                    <i>&hellip; 71920 more &hellip;</i>\n",
       "                </td>\n",
       "            </tr>\n",
       "        \n",
       "    \n",
       "    </tbody>\n",
       "</table>\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 218,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "eli5.show_weights(model, vec=vectorizer)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 221,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <style>\n",
       "    table.eli5-weights tr:hover {\n",
       "        filter: brightness(85%);\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        <table class=\"eli5-weights eli5-feature-importances\" style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto;\">\n",
       "    <thead>\n",
       "    <tr style=\"border: none;\">\n",
       "        <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">Weight</th>\n",
       "        <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "    </tr>\n",
       "    </thead>\n",
       "    <tbody>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 80.00%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0135\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__formation\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 84.79%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0091\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__mutagenized\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 86.37%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0078\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__variant\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.42%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0070\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__25th\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.52%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0069\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__calculate\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 87.74%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0067\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__activated\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.04%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0065\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__kras\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.05%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0065\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__21\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.41%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0062\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__6b\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "        <tr style=\"background-color: hsl(120, 100.00%, 88.53%); border: none;\">\n",
       "            <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "                0.0061\n",
       "                \n",
       "            </td>\n",
       "            <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "                Text__treatment\n",
       "            </td>\n",
       "        </tr>\n",
       "    \n",
       "    \n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 88.53%); border: none;\">\n",
       "                <td colspan=\"2\" style=\"padding: 0 0.5em 0 0.5em; text-align: center; border: none; white-space: nowrap;\">\n",
       "                    <i>&hellip; 71930 more &hellip;</i>\n",
       "                </td>\n",
       "            </tr>\n",
       "        \n",
       "    \n",
       "    </tbody>\n",
       "</table>\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 221,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "eli5.show_weights(model, vec=vectorizer, top=10, feature_filter=lambda x: x != '<BIAS>')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 227,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_name = explain_weights_df(model, vec=vectorizer, top=10, feature_filter=lambda x: x != '<BIAS>')\n",
    "                \n",
    "df_name.to_csv('../data/features/20190609sample_union_3groups_tfidf_feature_weights.csv')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "\n",
       "    <style>\n",
       "    table.eli5-weights tr:hover {\n",
       "        filter: brightness(85%);\n",
       "    }\n",
       "</style>\n",
       "\n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=1\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.485</b>, score <b>1.971</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 89.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +1.182\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 93.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.571\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.10%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.104\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__level\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.060\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__akt\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.20%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__history\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__enzymatic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__deviation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cdnas\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__allele\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__week\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clinical\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inhibitors\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__caused\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__laser\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__steps\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kb\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__documented\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__problem\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__iii\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__frequently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__3a\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__analogous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__progressive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__alanine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__base\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__relation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__value\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__little\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__confer\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__contributed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__characterize\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__too\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__efficiently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__like\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__biallelic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__allelic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__exon\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mdm2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cultures\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__prone\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__scored\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__codons\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__damage\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.43%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hit\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__09\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Variation__mutations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__flow\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> skin </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">cancers</span><span style=\"opacity: 0.80\">. the american cancer society estimated that there would be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic expression of growth factors and/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends on fgf2, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf2 resulted in phenotypic changes consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been shown to result in </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">inhibition</span><span style=\"opacity: 0.80\"> of tumor growth and induction of apoptosis (10, 11).  </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">constitutive</span><span style=\"opacity: 0.80\"> activation of fgfrs through chromosomal translocation, aberrant splicing, or missense </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> has been reported in several </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">cancers</span><span style=\"opacity: 0.80\">. the catalog of somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in cancer (cosmic)7 provides a repository of </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have recently reported the presence of activating </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 in 16% of endometrioid endometrial </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">cancers</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">).  we report here the identification of novel fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\"> of 113 (</span><span style=\"background-color: hsl(120, 100.00%, 98.02%); opacity: 0.80\" title=\"0.005\">13</span><span style=\"opacity: 0.80\">%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma tumors. mapping of these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> onto the known crystal structures of fgfr2 together </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional analyses shows that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> result in receptor loss of function.  results  identification of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in melanoma cell lines and uncultured primary and metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in the fgfr1 to fgfr4 genes by sequencing. following the identification of </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (table 1 ). none of these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), </span><span style=\"background-color: hsl(120, 100.00%, 99.69%); opacity: 0.80\" title=\"0.000\">suggesting</span><span style=\"opacity: 0.80\"> that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> arose somatically. of the </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\"> cell lines carrying fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, 11 also carried </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in either braf or nras (table 1). no </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> next, we screened a panel of 28 metastatic melanoma tumors and a panel of 72 vertical growth phase primary melanomas </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">representing</span><span style=\"opacity: 0.80\"> samples from different melanoma subtypes for the presence of </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 gene. we identified </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in 3 of 28 metastatic samples and 5 of 72 primary tumors (table 1). subset </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> revealed fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding normal tissue, and in </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> cases the mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> only present in the tumor, confirming the somatic origin of these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (</span><span style=\"background-color: hsl(120, 100.00%, 98.02%); opacity: 0.80\" title=\"0.005\">13</span><span style=\"opacity: 0.80\">), one metastatic tumor </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> found to carry four different fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (v77m, e574k, s688f, and p708s), </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> were not further analyzed because there </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> a high probability that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> represented passenger </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> rather </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">than</span><span style=\"opacity: 0.80\"> driver </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">. notably, two primary tumors carried nonsense </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (table 1). the majority (20 of 22) of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in both nras and braf were observed in tumors </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> at similar frequencies to that reported in the literature.  mapping of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> within known fgfr2 structures  extracellular </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">. to gain insights into how the different fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> might affect receptor function, we analyzed them in the light of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> affect residues in the extracellular </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; fig. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c </span><span style=\"background-color: hsl(120, 100.00%, 82.96%); opacity: 0.86\" title=\"0.118\">monomer</span><span style=\"opacity: 0.80\"> to evaluate the effect of the </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> each other directly and each receptor </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> both ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal structures suggests that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> should impair fgfr2c activity. the locations of mutated residues in the extracellular </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand binding </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> where the mutated receptor residues are located. in each image, in addition to the mutated residue, other relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> correspond to the mutated g227 and v248 in fgfr2. g226 in f strand </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall </span><span style=\"background-color: hsl(120, 100.00%, 91.68%); opacity: 0.82\" title=\"0.042\">stability</span><span style=\"opacity: 0.80\"> of d2. v247 engages in hydrophobic contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in the interior </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of </span><span style=\"background-color: hsl(120, 100.00%, 94.41%); opacity: 0.81\" title=\"0.024\">turn</span><span style=\"opacity: 0.80\"> between a and b by making hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in d3. therefore, mutation of g271 in fgfr2 should </span><span style=\"background-color: hsl(120, 100.00%, 98.01%); opacity: 0.80\" title=\"0.005\">negatively</span><span style=\"opacity: 0.80\"> affect the tertiary folding of d3. n304, on the other </span><span style=\"background-color: hsl(120, 100.00%, 98.72%); opacity: 0.80\" title=\"0.003\">hand</span><span style=\"opacity: 0.80\">, maps to the large loop </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> between strand c and c, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> protrudes from d3 and does not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to d218 in fgfr1, should </span><span style=\"background-color: hsl(120, 100.00%, 98.01%); opacity: 0.80\" title=\"0.005\">negatively</span><span style=\"opacity: 0.80\"> affect the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> its primary fgf2, </span><span style=\"background-color: hsl(120, 100.00%, 99.69%); opacity: 0.80\" title=\"0.000\">suggesting</span><span style=\"opacity: 0.80\"> that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (fig. 1b). by contrast, the e219k mutation maps to the secondary ligand binding site on d2. in the dimer, d218, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to e219 of fgfr2, </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> a hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k35 of the secondary fgf2 ligand (fig. 1e). therefore, </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other extracellular </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 where its </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain is surrounded by other hydrophobic residues inside the hydrophobic </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of d2 (fig. </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.002\">1a</span><span style=\"opacity: 0.80\">). substitution of v248 </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (fig. </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.002\">1a</span><span style=\"opacity: 0.80\">). g270, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on the structure, these substitutions should cause local structural perturbations, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on the structure, the e160a and h213y </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> are likely to affect the interaction of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate. in the structure, e159 of fgfr1, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> a key lysine residue involved in heparan sulfate binding resides (fig. 1d). therefore, destabilization of the ga helix due to the e160a mutation should </span><span style=\"background-color: hsl(120, 100.00%, 98.01%); opacity: 0.80\" title=\"0.005\">negatively</span><span style=\"opacity: 0.80\"> affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), and the bulkier tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain at this location could cause steric clashes </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">. more </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">than</span><span style=\"opacity: 0.80\"> half of the melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> affect the conserved tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2, and mapping of these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> onto unphosphorylated and phosphorylated fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain structures (pdb id: 2psq and 2pvf; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\">) suggests that they also represent loss-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (fig. 2 ). notably, m640 and i642 localize onto either end of the 8 strand (fig. 2), a secondary structure element that precedes the activation loop (a-loop) of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. the hydrophobic </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chains of these residues are in contact </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the e helix in the innermost </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of the c-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. substitutions of either residues </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation maps to the activation loop of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain (fig. 2e) and is also expected to lead to loss of function. this is because in the phosphorylated activated </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2 (pdb id: 2pvf; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\">), the methyl group of a648 </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> m537 in the c helix, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (</span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\">). a threonine is unfavorable at this location because its larger </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> identified in melanoma tumors/cell lines mapped onto the crystal structures of the fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain suggest that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> should impair the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain (pdb entry: 2psq; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\">). the coloring of the intracellular tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain is as follows: the nh2-terminal lobe of </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> hinge </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> is in green; and the nh2-terminal tail of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. to assist the viewer, certain  strands and  helices, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> are relevant for explanation of the effects of the melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, are labeled. a. e636, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> maps to the sharp </span><span style=\"background-color: hsl(120, 100.00%, 94.41%); opacity: 0.81\" title=\"0.024\">turn</span><span style=\"opacity: 0.80\"> between 7 and 8 strands, </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s563 in the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> hinge </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\">. mutation of this residue to lysine could </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">influence</span><span style=\"opacity: 0.80\"> the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. b. the m640 and i642 </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chains (red mesh) point into the inner hydrophobic </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these </span><span style=\"background-color: hsl(120, 100.00%, 95.24%); opacity: 0.81\" title=\"0.019\">core</span><span style=\"opacity: 0.80\"> hydrophobic contacts and lead to a reduction in </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity. c. the hydrogen bonds between r759 and </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chains of </span><span style=\"background-color: hsl(120, 100.00%, 98.01%); opacity: 0.80\" title=\"0.005\">negatively</span><span style=\"opacity: 0.80\"> charged e731 (in helix h) and d756 (in helix i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical </span><span style=\"background-color: hsl(120, 100.00%, 94.41%); opacity: 0.81\" title=\"0.024\">turn</span><span style=\"opacity: 0.80\">. these structural changes should reduce the </span><span style=\"background-color: hsl(120, 100.00%, 91.68%); opacity: 0.82\" title=\"0.042\">stability</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain, specifically making the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> where e475 is located. the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain of e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain and backbone atom of t555 (located in the loop between 4 and 5 strands), </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> facilitate the conformation of the 4-5 loop as well of the </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> preceding e475 itself. this in </span><span style=\"background-color: hsl(120, 100.00%, 94.41%); opacity: 0.81\" title=\"0.024\">turn</span><span style=\"opacity: 0.80\"> promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> where d530 and a648 are located. in unphosphorylated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2k structure, d530 does not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\">), the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain of d530 engages in salt bridge </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain of a648, located at the beginning of the a-loop, packs against the hydrophobic </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chains of m537 in the c helix. replacement of a648 </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> threonine is expected to lead to loss of function because the larger </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is in hydrogen-bonding contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation maps to the loop </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> between the 7 and 8 strands (fig. 2a). in the structure, the </span><span style=\"background-color: hsl(120, 100.00%, 94.27%); opacity: 0.81\" title=\"0.025\">side</span><span style=\"opacity: 0.80\"> chain of e636 </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> a solvent-exposed hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s568, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is located in the hinge </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain, </span><span style=\"background-color: hsl(120, 100.00%, 99.69%); opacity: 0.80\" title=\"0.000\">suggesting</span><span style=\"opacity: 0.80\"> that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> relative to each other. </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on the structure, the e636k mutation should </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">influence</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity of the fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> t555 whose methyl group is in hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">inhibition</span><span style=\"opacity: 0.80\"> and potentially lead to activation of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (fig. 2f) and facilitates the conformation of this loop </span><span style=\"background-color: hsl(120, 100.00%, 98.11%); opacity: 0.80\" title=\"0.005\">region</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> several hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity. the r759q mutation maps to the cooh-terminal end of the i helix, the last secondary structure element of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. in the structure, r759 </span><span style=\"background-color: hsl(120, 100.00%, 86.55%); opacity: 0.84\" title=\"0.084\">makes</span><span style=\"opacity: 0.80\"> hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> two other residues in its vicinity (fig. 2c), and loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> phosphopeptide indicates that the sh2 domain of plc interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate binding pocket of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the reduced phosphorylation of y769 could, in </span><span style=\"background-color: hsl(120, 100.00%, 94.41%); opacity: 0.81\" title=\"0.024\">turn</span><span style=\"opacity: 0.80\">, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> inferred from crystal structures. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> these analyses predicting that the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> are loss-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, two melanoma samples harbored nonsense </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> and a third sample carried a splicing mutation that would result in premature termination of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> translation (table 1).  view inlineview popup table 2. summary of in silico </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> of fgfr2 melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in vitro  to test our structural predictions for the extracellular </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> on fgfr2 function, we expressed recombinant wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant ectodomains in e. coli, and then isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have previously shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> yields similar to those of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and s252w mutant receptors. we then used surface plasmon resonance to analyze the effect of the r251q mutation on ligand binding affinity of fgfr2c. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the other </span><span style=\"background-color: hsl(120, 100.00%, 98.72%); opacity: 0.80\" title=\"0.003\">hand</span><span style=\"opacity: 0.80\">, as expected, the s252w mutation </span><span style=\"background-color: hsl(120, 100.00%, 97.92%); opacity: 0.80\" title=\"0.006\">enhanced</span><span style=\"opacity: 0.80\"> the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> shows that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> full-length human fgf2 </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> immobilized. the biosensor chip response is plotted as a function of time. surface plasmon resonance </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> done as previously </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> (19). to test the effect of </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, we prepared recombinant wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain proteins harboring the e475k, d530n, i642v, and a648t melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> and subjected them to in vitro </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> assays. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the d530n, i642v, and a648t mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domains exhibited reduced </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain (fig. 4 ). interestingly, the e475k mutant exhibited a slight </span><span style=\"background-color: hsl(120, 100.00%, 97.27%); opacity: 0.80\" title=\"0.009\">increase</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain. notably, the </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> expression yield for the e475k mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> severalfold less </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">than</span><span style=\"opacity: 0.80\"> that of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(120, 100.00%, 99.69%); opacity: 0.80\" title=\"0.000\">suggesting</span><span style=\"opacity: 0.80\"> that this mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> partially destabilizing the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domains harboring melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> have reduced tyrosine autophosphorylation activity. wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2k and four mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> assays. columns, average of three independent </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">experiments</span><span style=\"opacity: 0.80\">; bars, sd. the tyrosine autophosphorylation activities of these wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2ks were determined using a continuous spectrophotometric assay as previously </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> (43). </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in cultured cells  we next analyzed the effect of several </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crouzon syndrome </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> used to assess the effect of destabilizing d3 on both receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">constitutive</span><span style=\"opacity: 0.80\"> dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation observed in patients </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> apert syndrome has no effect on receptor folding (19). the k517r mutant receptor served as a </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\">-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously shown to abolish </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity (21).  following transfection of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the reduction in </span><span style=\"background-color: hsl(120, 100.00%, 98.26%); opacity: 0.80\" title=\"0.005\">size</span><span style=\"opacity: 0.80\"> of both bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (fig. 5a ). endohf digestion, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and </span><span style=\"background-color: hsl(120, 100.00%, 97.22%); opacity: 0.80\" title=\"0.009\">early</span><span style=\"opacity: 0.80\"> golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 (fig. 5a and b). the r251q and s252w </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> did not reduce the relative proportion of the endoh-resistant 130-kda mature form, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is fully consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our structural predictions of no adverse effects on </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 91.68%); opacity: 0.82\" title=\"0.042\">stability</span><span style=\"opacity: 0.80\"> by these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, </span><span style=\"background-color: hsl(120, 100.00%, 99.69%); opacity: 0.80\" title=\"0.000\">suggesting</span><span style=\"opacity: 0.80\"> impaired receptor processing of these fgfr2 mutants. this is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our structural predictions that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, top). there </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> a small </span><span style=\"background-color: hsl(120, 100.00%, 97.27%); opacity: 0.80\" title=\"0.009\">increase</span><span style=\"opacity: 0.80\"> in the relative proportion of the 110-kda band for the e475k mutant (fig. 5b, bottom), consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the decreased </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> yield observed in the in vitro </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> assays.  figure 5. download figureopen in new tabdownload powerpoint figure 5. </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. sensitivity to digestion </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and </span><span style=\"background-color: hsl(120, 100.00%, 97.22%); opacity: 0.80\" title=\"0.009\">early</span><span style=\"opacity: 0.80\"> golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 or mutant fgfr2. densitometry </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> revealed that the endoh-resistant 130-kda band comprises 60% of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutated fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> trafficking of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> receptor to the cell surface. d. immunofluorescent </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> of the colocalization of fgfr2 (red, left) and the er-resident marker </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the cell surface localization of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 and e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2, positive immunofluorescence signal </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> distributed primarily in the plasma membrane and also in the rough er and golgi (fig. 5c). in cells transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the g227e mutant (fig. 5c), however, there </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we then performed similar immunofluorescence studies looking for colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the er-resident </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in contrast, marked cell surface localization and limited colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase were observed for both the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 and the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay and p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mitogen-activated </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the effects of the melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> on fgfr2c activity. given that melanocytes and melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 91.07%); opacity: 0.82\" title=\"-0.047\">express</span><span style=\"opacity: 0.80\"> other fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it does not </span><span style=\"background-color: hsl(0, 100.00%, 91.07%); opacity: 0.82\" title=\"-0.047\">express</span><span style=\"opacity: 0.80\"> endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (</span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">) and an identical mutation at the paralogous positions in fgfr3 has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were then done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in </span><span style=\"background-color: hsl(120, 100.00%, 95.18%); opacity: 0.81\" title=\"0.019\">levels</span><span style=\"opacity: 0.80\"> of proliferation above that achieved by the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> receptor, whereas expression of </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2 (fig. 6a ), verifying that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. </span><span style=\"background-color: hsl(120, 100.00%, 93.15%); opacity: 0.82\" title=\"0.032\">surprisingly</span><span style=\"opacity: 0.80\">, the n549k mutation did not cause </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">constitutive</span><span style=\"opacity: 0.80\"> baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2_ires_neo expression constructs. a. </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> novel </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> identified in melanoma result in a decrease in baf3 proliferation when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">experiments</span><span style=\"opacity: 0.80\"> assayed in triplicate. b. </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> of phospho-p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk on stimulation </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> exogenous rfgf2 shows that, </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k positive control, </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> impair the ability of the receptor to activate this pro-proliferation pathway compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> receptor. c. western blot showing equal expression of fgfr2 by </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> mutants </span><span style=\"background-color: hsl(120, 100.00%, 98.46%); opacity: 0.80\" title=\"0.004\">except</span><span style=\"opacity: 0.80\"> empty vector in the absence () and presence (+) of fgf2. d. graphical representation of p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> receptor. as a further assessment of receptor impairment due to mutation, we </span><span style=\"background-color: hsl(120, 100.00%, 98.22%); opacity: 0.80\" title=\"0.005\">examined</span><span style=\"opacity: 0.80\"> the ability of the receptor to activate the pro-proliferative p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mitogen-activated </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> (mapk) signaling pathway in response to exogenous rfgf2. </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k known activating mutant, </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> resulted in a reduction to varying </span><span style=\"background-color: hsl(120, 100.00%, 95.18%); opacity: 0.81\" title=\"0.019\">levels</span><span style=\"opacity: 0.80\"> in the ability of the receptor to induce phosphorylation of p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk (fig. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> isoform-specific primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, our reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. </span><span style=\"background-color: hsl(120, 100.00%, 93.15%); opacity: 0.82\" title=\"0.032\">surprisingly</span><span style=\"opacity: 0.80\">, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> confirmed by western blot </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> for each cell line (fig. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> assessed using the sulforhodamine b assay. overexpression of fgfr2 or </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> confirmed by western blot </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\">. discussion  in this study, we report novel fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\"> of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary tumors. to determine the ratio of nonsynonymous to synonymous </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in our data set, we went back and </span><span style=\"background-color: hsl(120, 100.00%, 98.22%); opacity: 0.80\" title=\"0.005\">examined</span><span style=\"opacity: 0.80\"> the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in our data set </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> 23:4, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> exceeds the 2:1 ratio one would expect if these were random passenger </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, and supporting a role for these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> were c:g&gt;t:a substitutions </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 suggests that mutation of this gene is possibly an </span><span style=\"background-color: hsl(120, 100.00%, 97.22%); opacity: 0.80\" title=\"0.009\">early</span><span style=\"opacity: 0.80\"> event in transformation, and we anticipate that further sequencing </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> in a greater number of primary tumors and nevi will shed light on when during melanoma development these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> arise. it should be noted that the mutation frequency reported in this study may be an underestimate. this is </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on the </span><span style=\"background-color: hsl(0, 100.00%, 95.27%); opacity: 0.81\" title=\"-0.019\">fact</span><span style=\"opacity: 0.80\"> that </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">40</span><span style=\"opacity: 0.80\"> cell lines were screened by denaturing high-performance liquid chromatography rather </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">than</span><span style=\"opacity: 0.80\"> sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> primary tumors provided high-quality sequence for </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> exons.  insights gained from mapping these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> onto known fgfr2 crystal structure, combined </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional studies, show that these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> lead to loss of receptor activity through different mechanisms, including loss of ligand binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity (a648t, d530n, i642v). future studies confirming the structural predictions </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the loss-of-function nature of our identified melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.002\">published</span><span style=\"opacity: 0.80\"> literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hirschsprung&#x27;s disease, commonly due to loss-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> identified in the germ line of affected patients in two independent families </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our in vitro and in vivo data, a recent study showed that the a648t mutation abrogated the tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> n-lobe toward the c-lobe, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> is predicted to take place when the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> transits from the basal low activity state to the active state. however, precise definition of the mechanism by </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> a648t mutation impairs fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> activity awaits the resolution of crystal structure of the mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain as </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the catalytic loop of the </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain, and the crystal structure of the mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> identified in melanoma to those our lab previously identified in endometrial cancer highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (17 of 20) are identical to fgfr2 and fgfr3 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> previously reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> reported in cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> germ-line </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">than</span><span style=\"opacity: 0.80\"> 20 fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> we identified in melanoma were </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> novel, supporting our findings that the </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> are distinct and different from the many gain-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> highlighting the spectrum difference of somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in melanoma compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endometrial cancer, as well as germ-line </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in craniosynostosis syndromes. a. novel somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 identified in melanoma cell lines and uncultured tumors are presented in red above the schematic representation of the </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> and numbered relative to fgfr2c (np_000132.1). b. somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 identified in primary endometrial </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">cancers</span><span style=\"opacity: 0.80\"> and cell lines are presented in green (top) above the schematic representation of the </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> and are numbered relative to fgfr2b (np_075259.2; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">). bottom, germ-line </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> are underlined. four somatic fgfr2 endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">, although not previously reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> novel. this discovery of loss-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in a receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the presence of activating </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> in fgfr2 in endometrial cancer (</span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">). however, it should not be surprising given that a variety of receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> signaling pathways can induce either proliferation or differentiation depending on the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\">. indeed, cre-lox transgenic mice, </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2b having been deleted in cells </span><span style=\"background-color: hsl(120, 100.00%, 98.20%); opacity: 0.80\" title=\"0.005\">expressing</span><span style=\"opacity: 0.80\"> keratin v, develop spontaneous papillomas. moreover, following 7,</span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">-dimethylbenz(a)anthracene and </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">-o-tetradecanoylphorbol-</span><span style=\"background-color: hsl(120, 100.00%, 98.02%); opacity: 0.80\" title=\"0.005\">13</span><span style=\"opacity: 0.80\">-acetate treatment, these transgenic mice </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">developed</span><span style=\"opacity: 0.80\"> an increased number of papillomas and </span><span style=\"background-color: hsl(0, 100.00%, 91.55%); opacity: 0.82\" title=\"-0.043\">carcinomas</span><span style=\"opacity: 0.80\"> compared </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> mice, providing support that fgfr2 can act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> several </span><span style=\"background-color: hsl(120, 100.00%, 97.79%); opacity: 0.80\" title=\"0.006\">cancers</span><span style=\"opacity: 0.80\"> including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-36), and salivary adenocarcinoma cells (37).  </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> identified in melanoma as loss of function, it </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the previously </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.002\">published</span><span style=\"opacity: 0.80\"> reintroduction results and the current melanoma study is that they involve different fgfr2 splice forms. the previous studies were </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> carried out in cells of epithelial origin </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> are derived from the neural crest. the lack of </span><span style=\"background-color: hsl(120, 100.00%, 94.09%); opacity: 0.81\" title=\"0.026\">suppression</span><span style=\"opacity: 0.80\"> of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an </span><span style=\"background-color: hsl(120, 100.00%, 97.22%); opacity: 0.80\" title=\"0.009\">early</span><span style=\"opacity: 0.80\"> event and those melanoma cell lines have acquired additional genetic aberrations that prevent </span><span style=\"background-color: hsl(120, 100.00%, 94.09%); opacity: 0.81\" title=\"0.026\">suppression</span><span style=\"opacity: 0.80\"> of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> function to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact </span><span style=\"background-color: hsl(120, 100.00%, 98.83%); opacity: 0.80\" title=\"0.003\">inhibition</span><span style=\"opacity: 0.80\">, angiogenesis, cell migration, and metastasis. therefore, the </span><span style=\"background-color: hsl(120, 100.00%, 95.41%); opacity: 0.81\" title=\"0.018\">pathogenic</span><span style=\"opacity: 0.80\"> consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> occur as a consequence of the </span><span style=\"background-color: hsl(120, 100.00%, 98.70%); opacity: 0.80\" title=\"0.003\">oncogenic</span><span style=\"opacity: 0.80\"> state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> they arose. this is consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\">.  elucidation of the mechanism by </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">unknown</span><span style=\"opacity: 0.80\">. our lab is currently investigating the relative expression </span><span style=\"background-color: hsl(120, 100.00%, 95.18%); opacity: 0.81\" title=\"0.019\">levels</span><span style=\"opacity: 0.80\"> of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> signaling. there have been reports that fgfr1 binds cbp and rsk in the </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">nucleus</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 93.94%); opacity: 0.81\" title=\"-0.027\">38</span><span style=\"opacity: 0.80\">), and we and others have observed that fgfr2 localized to the </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">nucleus</span><span style=\"opacity: 0.80\"> in various cell types (39).8 lastly, we and others have shown that fgfr2 can heterodimerize </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr1 at least in vitro (</span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">40</span><span style=\"opacity: 0.80\">)8 and fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology will allow us to identify the mechanism by </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\">), fgfr2 may be the first gene in </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> both loss-of-function and gain-of-function </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> have been reported in different tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by </span><span style=\"background-color: hsl(0, 100.00%, 99.93%); opacity: 0.80\" title=\"-0.000\">which</span><span style=\"opacity: 0.80\"> loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning sites. point </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> or mutant expression constructs and </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> expression </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> induced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant proteins were then sequentially purified by heparin affinity and </span><span style=\"background-color: hsl(120, 100.00%, 98.26%); opacity: 0.80\" title=\"0.005\">size</span><span style=\"opacity: 0.80\"> exclusion chromatography. surface plasmon resonance </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> done as </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> previously (19).  expression and purification of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> domain  the dna fragment encoding residues 458 to 768 of human fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> subcloned into pet bacterial expression vectors </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an nh2-terminal 6his-tag to aid in </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> purification. point </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> or mutant fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> expression constructs, and </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> expression </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> induced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were then lysed using a french press, and the soluble </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> proteins were purified by using sequential ni2+-chelating, anion exchange and </span><span style=\"background-color: hsl(120, 100.00%, 98.26%); opacity: 0.80\" title=\"0.005\">size</span><span style=\"opacity: 0.80\"> exclusion chromatography. the purity of the proteins </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> estimated to be &gt;98% </span><span style=\"background-color: hsl(0, 100.00%, 99.54%); opacity: 0.80\" title=\"-0.001\">based</span><span style=\"opacity: 0.80\"> on sds-page </span><span style=\"background-color: hsl(120, 100.00%, 99.61%); opacity: 0.80\" title=\"0.001\">analysis</span><span style=\"opacity: 0.80\">. purified wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant kinases </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> compared using a continuous spectrophotometric </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\"> assay according to the </span><span style=\"background-color: hsl(120, 100.00%, 98.96%); opacity: 0.80\" title=\"0.002\">published</span><span style=\"opacity: 0.80\"> protocol (43).  site-directed mutagenesis of full-length fgfr2  </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> were introduced to the pcdna3_fgfr2 wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> sequenced for each clone to confirm the presence of the intended mutation and to ensure that no other </span><span style=\"background-color: hsl(120, 100.00%, 99.95%); opacity: 0.80\" title=\"0.000\">mutations</span><span style=\"opacity: 0.80\"> were introduced during the mutagenesis process. plasmid dna </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> then isolated using qiagen endofree plasmid dna isolation kits, and dna </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a260/a280 &gt; 1.8 </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> used in </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> subsequent </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">experiments</span><span style=\"opacity: 0.80\">.  cell lines and transfections  </span><span style=\"background-color: hsl(120, 100.00%, 99.20%); opacity: 0.80\" title=\"0.001\">all</span><span style=\"opacity: 0.80\"> cell lines </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> in this series of </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">experiments</span><span style=\"opacity: 0.80\"> were maintained under standard culturing conditions as </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> by the american </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> or mutant fgfr2 + 750 g empty plasmid) were achieved </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells were washed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5% skim milk/tbst, membranes were probed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:2,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\"> bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as </span><span style=\"background-color: hsl(120, 100.00%, 99.58%); opacity: 0.80\" title=\"0.001\">described</span><span style=\"opacity: 0.80\"> and plated onto eight-well chamber slides. after 36 to 48 h, cells were fixed </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:300 bek c-17 and 1:1,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\"> anti</span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (invitrogen) and then detected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\">; invitrogen), respectively. slides were mounted </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">40</span><span style=\"opacity: 0.80\"> h, pooled, filtered through a 0.</span><span style=\"background-color: hsl(0, 100.00%, 93.92%); opacity: 0.81\" title=\"-0.027\">45</span><span style=\"opacity: 0.80\">-um low </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> binding filter, and stored at 80c until use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crystal violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2 were maintained under selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\"> g/ml geneticin, and supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> removed at day 3 and replaced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an equal volume containing fresh ligand at 2 concentration and no il-3. proliferation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-</span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">15</span><span style=\"opacity: 0.80\"> medium </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 2 mmol/l l-glutamine supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in rpmi supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty vector, fgfr2, or </span><span style=\"background-color: hsl(0, 100.00%, 96.18%); opacity: 0.81\" title=\"-0.014\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\">-4,</span><span style=\"background-color: hsl(120, 100.00%, 95.27%); opacity: 0.81\" title=\"0.019\">000</span><span style=\"opacity: 0.80\">) were plated per well in a 96-well plate in full growth media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were then stimulated </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> phosphorylation  to examine p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in either baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 g/ml heparin only (unstimulated or ) or baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.93%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">protein</span><span style=\"opacity: 0.80\"> were resolved on a 4% to </span><span style=\"background-color: hsl(120, 100.00%, 99.33%); opacity: 0.80\" title=\"0.001\">12</span><span style=\"opacity: 0.80\">% bis tris gel and probed for phosho-p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk. blots were then stripped and reprobed for total p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk (cell signaling technologies).  phosphorylation </span><span style=\"background-color: hsl(120, 100.00%, 99.89%); opacity: 0.80\" title=\"0.000\">was</span><span style=\"opacity: 0.80\"> quantitated using image j9 and presented as the ratio of phosphorylated over total p44/</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">42</span><span style=\"opacity: 0.80\"> mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">type</span><span style=\"opacity: 0.80\"> receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=2\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.009</b>, score <b>-1.992</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 95.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.326\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.059\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__repair\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.29%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__differentiated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__potential\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__non\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__reflected\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__systems\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__suggested\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__germline\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__exceeded\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__57\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__facial\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__endothelial\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__gap\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mass\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__25th\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__95\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fda\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__immune\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__insensitive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sources\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__141\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__demonstrated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__corrected\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__removal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__remaining\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__implications\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__faster\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__statistical\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__condition\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__starved\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__months\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mutational\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__51\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__remains\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__significant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__annotated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.26%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.027\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__agents\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.12%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.035\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__correlations\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.03%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.040\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__health\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.98%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.043\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__unique\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.052\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__female\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.059\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__advanced\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.075\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clinically\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.077\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kras\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.078\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inhibitors\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.21%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.096\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__statistically\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.11%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.104\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__even\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.02%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.111\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__significantly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.134\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__thyroid\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 89.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.200\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there would be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in </span><span style=\"background-color: hsl(0, 100.00%, 97.22%); opacity: 0.80\" title=\"-0.009\">2008</span><span style=\"opacity: 0.80\">. previous studies in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> of growth factors and/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a </span><span style=\"background-color: hsl(120, 100.00%, 93.46%); opacity: 0.82\" title=\"0.030\">prominent</span><span style=\"opacity: 0.80\"> role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends on fgf2, which is </span><span style=\"background-color: hsl(0, 100.00%, 96.94%); opacity: 0.81\" title=\"-0.010\">provided</span><span style=\"opacity: 0.80\"> by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> and metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor growth and induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, </span><span style=\"background-color: hsl(120, 100.00%, 97.88%); opacity: 0.80\" title=\"0.006\">aberrant</span><span style=\"opacity: 0.80\"> splicing, or </span><span style=\"background-color: hsl(120, 100.00%, 93.03%); opacity: 0.82\" title=\"0.033\">missense</span><span style=\"opacity: 0.80\"> mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (12).  we report here the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">. mapping of these mutations </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structures of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.11%); opacity: 0.81\" title=\"-0.014\">together</span><span style=\"opacity: 0.80\"> with in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> and metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> screened for fgfr2 mutations using a combination of denaturing </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\">-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (table 1). no mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> and a panel of 72 vertical growth phase </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> melanomas representing samples from different melanoma subtypes for the presence of mutations in fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.002\">gene</span><span style=\"opacity: 0.80\">. we identified mutations in 3 of 28 metastatic samples and 5 of 72 </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">, we </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> able to extract dna from surrounding normal tissue, and in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator </span><span style=\"background-color: hsl(0, 100.00%, 97.56%); opacity: 0.80\" title=\"-0.007\">phenotype</span><span style=\"opacity: 0.80\"> seen in the breast cancer kinome screen (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, and p708s), which </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> not further analyzed because there was a </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\"> probability that these mutations represented passenger mutations rather than driver mutations. notably, two </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> carried nonsense mutations (table 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras and braf </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> observed in </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> fgfr2 structures  extracellular mutations. to gain insights into how the different fgfr2 mutations might </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> receptor function, we analyzed them in the light of </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> residues in the extracellular </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> of fgfr2, </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> mapped </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; </span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1 ). we intentionally chose this structure instead of the </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> directly and </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> receptor makes contacts with both ligands as well.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 1. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 1. mapping of fgfr2c extracellular mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">/cell lines </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> fgf-fgfr </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> of fgfr2 are mapped </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand binding </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> the mutated receptor residues are located. in </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> image, in </span><span style=\"background-color: hsl(120, 100.00%, 98.40%); opacity: 0.80\" title=\"0.004\">addition</span><span style=\"opacity: 0.80\"> to the mutated residue, </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand and contributes to </span><span style=\"background-color: hsl(0, 100.00%, 95.64%); opacity: 0.81\" title=\"-0.017\">proper</span><span style=\"opacity: 0.80\"> f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds with </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> residues in d3. therefore, mutation of g271 in fgfr2 should negatively </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the tertiary folding of d3. n304, on the </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> hand, maps to the large loop </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> between strand c and c, which protrudes from d3 and does not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is </span><span style=\"background-color: hsl(0, 100.00%, 73.74%); opacity: 0.91\" title=\"-0.218\">highlighted</span><span style=\"opacity: 0.80\"> in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in </span><span style=\"background-color: hsl(0, 100.00%, 97.64%); opacity: 0.80\" title=\"-0.007\">red</span><span style=\"opacity: 0.80\">, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts with the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). by contrast, the e219k mutation maps to the secondary ligand binding </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\"> on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 ligand (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1e). therefore, based on the structure, e219k should not </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> extracellular mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> its side chain is surrounded by </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> hydrophobic residues inside the hydrophobic core of d2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1a). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1c). </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> glycine residue provides hydrogen bonds that contribute to the </span><span style=\"background-color: hsl(0, 100.00%, 95.64%); opacity: 0.81\" title=\"-0.017\">proper</span><span style=\"opacity: 0.80\"> -strand formation in d2 and d3, respectively. therefore, based on the structure, these substitutions should cause local structural perturbations, which </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> ultimately </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on which a key lysine residue involved in heparan sulfate binding resides (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1d), and the bulkier tyrosine side chain at this location </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> cause steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations. more than half of the melanoma mutations </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the conserved tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2, and mapping of these mutations </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> unphosphorylated and phosphorylated fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2 ). notably, m640 and i642 localize </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> either end of the 8 strand (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2), a secondary structure element that precedes the activation loop (a-loop) of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. substitutions of either residues with the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> that should induce local structural perturbations that ultimately </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the conformation of the activation loop. the a648t mutation maps to the activation loop of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(120, 100.00%, 88.00%); opacity: 0.84\" title=\"0.071\">2e</span><span style=\"opacity: 0.80\">) and is also expected to lead to loss of function. this is because in the phosphorylated activated </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 2. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 2. fgfr2c </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">/cell lines mapped </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structures of the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain suggest that these mutations should impair the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. the positions of the mutated residues are mapped </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure of unphosphorylated wild-type fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (pdb entry: 2psq; ref. 15). the coloring of the </span><span style=\"background-color: hsl(120, 100.00%, 95.44%); opacity: 0.81\" title=\"0.018\">intracellular</span><span style=\"opacity: 0.80\"> tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain is as follows: the nh2-terminal lobe of </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> hinge </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> is in green; and the nh2-terminal tail of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the effects of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond with s563 in the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> hinge </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\">. mutation of this residue to lysine </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> influence the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. b. the m640 and i642 side chains (</span><span style=\"background-color: hsl(0, 100.00%, 97.64%); opacity: 0.80\" title=\"-0.007\">red</span><span style=\"opacity: 0.80\"> mesh) point into the inner hydrophobic core of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a reduction in </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are shown. in </span><span style=\"background-color: hsl(120, 100.00%, 98.40%); opacity: 0.80\" title=\"0.004\">addition</span><span style=\"opacity: 0.80\"> to the loss of these hydrogen bonds, introduction of a stop codon at this location </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">will</span><span style=\"opacity: 0.80\"> also truncate the helix i by one helical turn. these structural changes should reduce the stability of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, specifically making the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is expected to lead to loss of function because the larger side chain of threonine </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">will</span><span style=\"opacity: 0.80\"> impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the a-loop conformation. the e636k mutation maps to the loop </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> between the 7 and 8 strands (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> relative to </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\">. based on the structure, the e636k mutation should influence the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> relieve this inhibition and potentially lead to activation of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. alternatively, the e475k mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> introduce structural instability in the n-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> and reduce the half life of the </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2. the g701s mutation maps to the loop between f and g helices (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2f) and facilitates the conformation of this loop </span><span style=\"background-color: hsl(0, 100.00%, 99.16%); opacity: 0.80\" title=\"-0.002\">region</span><span style=\"opacity: 0.80\">, which makes several hydrogen bonds with k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> indirectly lead to reduced </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity. the r759q mutation maps to the cooh-terminal end of the i helix, the last secondary structure element of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. in the structure, r759 makes hydrogen bonds with two </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> residues in its vicinity (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2c), and loss of these hydrogen bonds due to the r759q mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> also induce subtle structural changes leading to a loss in </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\"> of fgfr2 that serves as the docking </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\"> for sh2 domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex with phosphopeptide indicates that the sh2 domain of plc interacts with ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> also reduce y769 phosphorylation because the substrate binding pocket of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain prefers a leucine over a valine next to the tyrosine phosphorylation </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\"> (16). the reduced phosphorylation of y769 </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\">, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different </span><span style=\"background-color: hsl(120, 100.00%, 97.47%); opacity: 0.80\" title=\"0.008\">mechanisms</span><span style=\"opacity: 0.80\"> of loss of function by the melanoma mutations inferred from </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structures. consistent with these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> translation (table 1).  view inlineview popup table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions for the extracellular mutations on fgfr2 function, we expressed recombinant wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> ectodomains in e. coli, and then isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> ectodomain. we have </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> ectodomain folded in vitro with yields similar to those of the wild-type and s252w </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> receptors. we then used surface plasmon resonance to analyze the effect of the r251q mutation on ligand binding affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> fgfs (refs. 17-19; data not shown).  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 3. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-type (</span><span style=\"background-color: hsl(120, 100.00%, 97.41%); opacity: 0.80\" title=\"0.008\">wt</span><span style=\"opacity: 0.80\">; a) or r251q fgfr2c </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> ectodomain (b) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> injected over a cm5 chip </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of </span><span style=\"background-color: hsl(0, 100.00%, 96.41%); opacity: 0.81\" title=\"-0.013\">time</span><span style=\"opacity: 0.80\">. surface plasmon resonance analysis was done as </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> described (19). to test the effect of </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations, we prepared recombinant wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> assays. consistent with structural predictions, the d530n, i642v, and a648t </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domains </span><span style=\"background-color: hsl(120, 100.00%, 97.36%); opacity: 0.80\" title=\"0.008\">exhibited</span><span style=\"opacity: 0.80\"> reduced </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-type </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 4 ). interestingly, the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.36%); opacity: 0.80\" title=\"0.008\">exhibited</span><span style=\"opacity: 0.80\"> a slight increase in </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-type </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. notably, the </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> yield for the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> was severalfold less than that of wild-type </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\">, suggesting that this mutation was partially destabilizing the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 4. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 4. fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domains harboring melanoma mutations have reduced tyrosine autophosphorylation activity. wild-type fgfr2k and four </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2ks, </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2ks </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> determined using a continuous spectrophotometric assay as </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> described (43). analysis of fgfr2 mutations in cultured cells  we next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, and increased signaling from </span><span style=\"background-color: hsl(120, 100.00%, 95.44%); opacity: 0.81\" title=\"0.018\">intracellular</span><span style=\"opacity: 0.80\"> compartments (20). in contrast, the s252w mutation observed in patients with apert syndrome has no effect on receptor folding (19). the k517r </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> receptor served as a </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\">-deficient receptor. k517 coordinates the phosphate </span><span style=\"background-color: hsl(120, 100.00%, 89.11%); opacity: 0.83\" title=\"0.062\">groups</span><span style=\"opacity: 0.80\"> of atp, and its mutation to arginine in the homologous fgfr3 has been </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> shown to abolish </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity (21).  following transfection of wild-type fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the reduction in size of both bands to a single 98-kda band (</span><span style=\"background-color: hsl(120, 100.00%, 97.08%); opacity: 0.80\" title=\"0.009\">open</span><span style=\"opacity: 0.80\"> arrow), confirming that these two bands are differently glycosylated receptor populations (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5a ). endohf digestion, which is </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> to remove n-linked </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\">-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions of no adverse effects on </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent with </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions that these mutations </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">will</span><span style=\"opacity: 0.80\"> adversely </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> the structural integrity of the d2 or d3 domain of fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, bottom), consistent with the decreased </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> yield observed in the in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> assays.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 5. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 5. mutations in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (</span><span style=\"background-color: hsl(120, 100.00%, 97.08%); opacity: 0.80\" title=\"0.009\">open</span><span style=\"opacity: 0.80\"> arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following </span><span style=\"background-color: hsl(0, 100.00%, 97.90%); opacity: 0.80\" title=\"-0.006\">transient</span><span style=\"opacity: 0.80\"> transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 97.64%); opacity: 0.80\" title=\"-0.007\">red</span><span style=\"opacity: 0.80\">, left) and the er-resident marker </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (green, middle) confirms the overall retention of the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 and e219k (mapping </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> extracellular </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5c). in cells transfected with the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to </span><span style=\"background-color: hsl(120, 100.00%, 95.44%); opacity: 0.81\" title=\"0.018\">intracellular</span><span style=\"opacity: 0.80\"> membranes and decreased cell surface </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> shows clear cell surface localization in keeping with the structural predictions and endohf analyses. to confirm the er localization of the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\">, we then performed similar immunofluorescence studies looking for colocalization with the er-resident </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase, which revealed a </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\"> degree of colocalization of the two proteins, confirming impaired receptor localization (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5d). in contrast, marked cell surface localization and limited colocalization with </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> observed for both the wild-type fgfr2 and the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the effects of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it does not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (12) and an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> generated following lentiviral transduction and selection in geneticin. proliferation assays </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> then done in the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">absence</span><span style=\"opacity: 0.80\"> of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> resulted in levels of proliferation above that </span><span style=\"background-color: hsl(120, 100.00%, 91.91%); opacity: 0.82\" title=\"0.041\">achieved</span><span style=\"opacity: 0.80\"> by the wild-type receptor, whereas </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> of all </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">absence</span><span style=\"opacity: 0.80\"> of ligand stimulation.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 6. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2_ires_neo </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">absence</span><span style=\"opacity: 0.80\"> of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on stimulation with exogenous rfgf2 shows that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> of fgfr2 by all mutants except empty vector in the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">absence</span><span style=\"opacity: 0.80\"> () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-type receptor. as a further assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> activating </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\">, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.67%); opacity: 0.80\" title=\"0.000\">could</span><span style=\"opacity: 0.80\"> suppress proliferation of melanoma cells. reverse transcription-</span><span style=\"background-color: hsl(120, 100.00%, 97.64%); opacity: 0.80\" title=\"0.007\">pcr</span><span style=\"opacity: 0.80\"> with isoform-specific primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> by reverse transcription-</span><span style=\"background-color: hsl(120, 100.00%, 97.64%); opacity: 0.80\" title=\"0.007\">pcr</span><span style=\"opacity: 0.80\"> (sbcl2) and in melanoma cell lines that expressed </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 7a ). </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> of fgfr2 following lentiviral transduction was </span><span style=\"background-color: hsl(0, 100.00%, 92.27%); opacity: 0.82\" title=\"-0.038\">confirmed</span><span style=\"opacity: 0.80\"> by western blot analysis for </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> cell line (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 7b). similar results </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> also </span><span style=\"background-color: hsl(120, 100.00%, 91.91%); opacity: 0.82\" title=\"0.041\">achieved</span><span style=\"opacity: 0.80\"> in both the d22 and uacc2534 melanoma cell lines (data not shown).  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 7. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> was </span><span style=\"background-color: hsl(0, 100.00%, 92.27%); opacity: 0.82\" title=\"-0.038\">confirmed</span><span style=\"opacity: 0.80\"> by western blot analysis. discussion  in this </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">study</span><span style=\"opacity: 0.80\">, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\">. to determine the ratio of nonsynonymous to synonymous mutations in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> data set, we went back and examined the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> identified in dbsnp. several additional synonymous variants occurred alongside </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> in multiple individuals, consistent with the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> data set was 23:4, which exceeds the 2:1 ratio one would expect if these </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> c:g&gt;t:a substitutions with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.002\">gene</span><span style=\"opacity: 0.80\"> is possibly an early event in transformation, and we anticipate that further sequencing analysis in a greater number of </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> and nevi </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">will</span><span style=\"opacity: 0.80\"> shed light on when during melanoma development these mutations arise. it should be noted that the mutation frequency reported in this </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">study</span><span style=\"opacity: 0.80\"> may be an underestimate. this is based on the fact that 40 cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> screened by denaturing </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\">-performance liquid chromatography rather than sequencing, exons 2 to 5 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> not screened in this latter panel of cell lines, and not all </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.94%); opacity: 0.81\" title=\"-0.010\">provided</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\">-quality sequence for all exons.  insights gained from mapping these mutations </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.41%); opacity: 0.80\" title=\"-0.001\">known</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure, combined with in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different </span><span style=\"background-color: hsl(120, 100.00%, 97.47%); opacity: 0.80\" title=\"0.008\">mechanisms</span><span style=\"opacity: 0.80\">, including loss of ligand binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the loss-of-function nature of </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> identified melanoma mutations is also consistent with published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated with hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in </span><span style=\"background-color: hsl(120, 100.00%, 98.40%); opacity: 0.80\" title=\"0.004\">addition</span><span style=\"opacity: 0.80\">, while these studies </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> under way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent families with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent with </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> in vitro and in vivo data, a recent </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">study</span><span style=\"opacity: 0.80\"> showed that the a648t mutation abrogated the tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> n-lobe toward the c-lobe, which is predicted to take place when the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> activity awaits the resolution of </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure of the </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> the catalytic loop of the </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, and the </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> structure of the </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> lab </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> identified in endometrial cancer highlights the differences in the mutation spectra (</span><span style=\"background-color: hsl(0, 100.00%, 99.66%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> reported in the germ line, a finding similar to that observed for fgfr3 </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">where</span><span style=\"opacity: 0.80\"> a comparison of fgfr3 mutations reported in cancer with germ-line mutations associated with skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than 20 fgfr2 mutations we identified in melanoma </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> all novel, supporting </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> findings that the mutations are distinct and different from the many gain-of-function mutations </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> identified in the fgfrs.  </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 8. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(0, 100.00%, 98.74%); opacity: 0.80\" title=\"-0.003\">figure</span><span style=\"opacity: 0.80\"> 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured </span><span style=\"background-color: hsl(120, 100.00%, 98.61%); opacity: 0.80\" title=\"0.003\">tumors</span><span style=\"opacity: 0.80\"> are presented in </span><span style=\"background-color: hsl(0, 100.00%, 97.64%); opacity: 0.80\" title=\"-0.007\">red</span><span style=\"opacity: 0.80\"> above the schematic representation of the </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in </span><span style=\"background-color: hsl(120, 100.00%, 98.58%); opacity: 0.80\" title=\"0.003\">primary</span><span style=\"opacity: 0.80\"> endometrial cancers and cell lines are presented in green (top) above the schematic representation of the </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> and are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-line mutations associated with a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although not </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> reported in the germ line, have an identical </span><span style=\"background-color: hsl(120, 100.00%, 93.03%); opacity: 0.82\" title=\"0.033\">missense</span><span style=\"opacity: 0.80\"> change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this </span><span style=\"background-color: hsl(0, 100.00%, 97.86%); opacity: 0.80\" title=\"-0.006\">discovery</span><span style=\"opacity: 0.80\"> of loss-of-function mutations in a receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (12). however, it should not be surprising given that a variety of receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling pathways can induce either proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic mice, with fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene and 12-o-tetradecanoylphorbol-13-acetate </span><span style=\"background-color: hsl(0, 100.00%, 87.55%); opacity: 0.84\" title=\"-0.075\">treatment</span><span style=\"opacity: 0.80\">, these transgenic mice developed an increased number of papillomas and carcinomas compared with wild-type mice, providing support that fgfr2 can act as a tumor </span><span style=\"background-color: hsl(120, 100.00%, 93.54%); opacity: 0.81\" title=\"0.029\">suppressor</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.002\">gene</span><span style=\"opacity: 0.80\"> in some cell types (29). furthermore, loss of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> has been associated with several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (</span><span style=\"background-color: hsl(0, 100.00%, 96.05%); opacity: 0.81\" title=\"-0.015\">33</span><span style=\"opacity: 0.80\">), prostate tumor cells (34-36), and salivary adenocarcinoma cells (37).  based on these data and </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> published reintroduction results and the current melanoma </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">study</span><span style=\"opacity: 0.80\"> is that they involve different fgfr2 splice forms. the previous studies </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> all carried out in cells of epithelial origin with the fgfr2 iiib isoform, whereas </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which function to suppress tumorigenesis not by inhibiting proliferation but by altering </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> aspects of the malignant </span><span style=\"background-color: hsl(0, 100.00%, 97.56%); opacity: 0.80\" title=\"-0.007\">phenotype</span><span style=\"opacity: 0.80\">, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead </span><span style=\"background-color: hsl(0, 100.00%, 96.09%); opacity: 0.81\" title=\"-0.014\">affect</span><span style=\"opacity: 0.80\"> another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\"> frequency and </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">high</span><span style=\"opacity: 0.80\"> nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently unknown. </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> lab is currently investigating the relative </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> levels of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling. there have been reports that fgfr1 binds cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in various cell types (</span><span style=\"background-color: hsl(0, 100.00%, 86.33%); opacity: 0.84\" title=\"-0.086\">39</span><span style=\"opacity: 0.80\">).8 lastly, we and others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (40)8 and fgfrs have also been shown to bind to </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">will</span><span style=\"opacity: 0.80\"> allow </span><span style=\"background-color: hsl(0, 100.00%, 68.74%); opacity: 0.94\" title=\"-0.280\">us</span><span style=\"opacity: 0.80\"> to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first </span><span style=\"background-color: hsl(0, 100.00%, 99.02%); opacity: 0.80\" title=\"-0.002\">gene</span><span style=\"opacity: 0.80\"> in which both loss-of-function and gain-of-function mutations have been reported in different tissue types. ongoing work in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> laboratory is targeted toward understanding the molecular </span><span style=\"background-color: hsl(120, 100.00%, 97.47%); opacity: 0.80\" title=\"0.008\">mechanisms</span><span style=\"opacity: 0.80\"> by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\">, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by </span><span style=\"background-color: hsl(120, 100.00%, 97.64%); opacity: 0.80\" title=\"0.007\">pcr</span><span style=\"opacity: 0.80\"> and subcloned into pet-28a bacterial </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> vector using ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> introduced using quikchange </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\">-directed </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> kit (stratagene). bl21(de3) competent cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> transformed with wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> constructs and </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> lysed using french press and centrifuged; the inclusion bodies containing wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> then sequentially purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described </span><span style=\"background-color: hsl(0, 100.00%, 98.91%); opacity: 0.80\" title=\"-0.002\">previously</span><span style=\"opacity: 0.80\"> (19).  </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> and purification of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 89.13%); opacity: 0.83\" title=\"-0.062\">vectors</span><span style=\"opacity: 0.80\"> with an nh2-terminal 6</span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">his</span><span style=\"opacity: 0.80\">-tag to aid in </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> purification. point mutations (e475k, d530n, i642v, and a648t) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> introduced using quikchange </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\">-directed </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> kit (stratagene). the bacterial strain bl21(de3) cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> transformed with wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> constructs, and </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">expression</span><span style=\"opacity: 0.80\"> was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> then lysed using a french press, and the soluble </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> purified by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. purified wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> concentrated to at least 10 </span><span style=\"background-color: hsl(120, 100.00%, 86.86%); opacity: 0.84\" title=\"0.081\">mg</span><span style=\"opacity: 0.80\">/ml using a centricon-10. the autophosphorylation activity of the wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> kinases was compared using a continuous spectrophotometric </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\"> assay </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.006\">according</span><span style=\"opacity: 0.80\"> to the published protocol (43).  </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\">-directed </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> of full-length fgfr2  all mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\">-directed </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> kit (stratagene) </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.006\">according</span><span style=\"opacity: 0.80\"> to the manufacturer&#x27;s instructions. additionally, </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> primers </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> designed to introduce a novel but silent (noncoding change) restriction enzyme recognition </span><span style=\"background-color: hsl(0, 100.00%, 98.58%); opacity: 0.80\" title=\"-0.003\">site</span><span style=\"opacity: 0.80\"> to allow for fast, preliminary screening of clones. the full list of primer </span><span style=\"background-color: hsl(120, 100.00%, 97.48%); opacity: 0.80\" title=\"0.008\">sequences</span><span style=\"opacity: 0.80\"> can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> clone to confirm the presence of the intended mutation and to ensure that no </span><span style=\"background-color: hsl(0, 100.00%, 99.96%); opacity: 0.80\" title=\"-0.000\">other</span><span style=\"opacity: 0.80\"> mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> introduced during the </span><span style=\"background-color: hsl(120, 100.00%, 98.81%); opacity: 0.80\" title=\"0.003\">mutagenesis</span><span style=\"opacity: 0.80\"> process. plasmid dna was then isolated using qiagen endofree plasmid dna isolation kits, and dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> maintained under standard culturing </span><span style=\"background-color: hsl(0, 100.00%, 91.39%); opacity: 0.82\" title=\"-0.044\">conditions</span><span style=\"opacity: 0.80\"> as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). </span><span style=\"background-color: hsl(0, 100.00%, 97.90%); opacity: 0.80\" title=\"-0.006\">transient</span><span style=\"opacity: 0.80\"> transfections of 1 g total dna (consisting of 250 g wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 + 750 g empty plasmid) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 91.91%); opacity: 0.82\" title=\"0.041\">achieved</span><span style=\"opacity: 0.80\"> with fugene 6 (roche) at a 3:1 ratio of reagent/dna </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.006\">according</span><span style=\"opacity: 0.80\"> to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> washed with ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> subjected to digestion </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.006\">according</span><span style=\"opacity: 0.80\"> to the manufacturer&#x27;s recommendations. </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> was resolved on 3% to 8% tris-acetate gels and transferred </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> transfected as described and plated </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">onto</span><span style=\"opacity: 0.80\"> eight-well chamber slides. after 36 to 48 h, cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated with 1:300 bek c-17 and 1:1,000 anti</span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (invitrogen) and then detected with </span><span style=\"background-color: hsl(0, 100.00%, 85.90%); opacity: 0.85\" title=\"-0.090\">goat</span><span style=\"opacity: 0.80\"> ant-rabbit cy3 (1:200; jackson immunoresearch) and </span><span style=\"background-color: hsl(0, 100.00%, 85.90%); opacity: 0.85\" title=\"-0.090\">goat</span><span style=\"opacity: 0.80\"> anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> mounted with pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> transfected with lentiviral packaging </span><span style=\"background-color: hsl(0, 100.00%, 89.13%); opacity: 0.83\" title=\"-0.062\">vectors</span><span style=\"opacity: 0.80\"> and ptyf.fgfr2.ires.neo (or </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.001\">per</span><span style=\"opacity: 0.80\"> manufacturer&#x27;s instructions. media containing the virus </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> collected at 24 and 40 h, </span><span style=\"background-color: hsl(120, 100.00%, 86.50%); opacity: 0.84\" title=\"0.084\">pooled</span><span style=\"opacity: 0.80\">, filtered through a 0.45-um low </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> binding filter, and stored at 80c until use. viral stocks </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with </span><span style=\"background-color: hsl(0, 100.00%, 95.61%); opacity: 0.81\" title=\"-0.017\">crystal</span><span style=\"opacity: 0.80\"> violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-type and </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> maintained under selection in rpmi/10% fetal bovine </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\"> containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented with 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> counted and washed in the above media containing no il-3. cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> plated at 1  104 </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.001\">per</span><span style=\"opacity: 0.80\"> well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced with an equal volume containing fresh ligand at 2 concentration and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) </span><span style=\"background-color: hsl(0, 100.00%, 97.96%); opacity: 0.80\" title=\"-0.006\">according</span><span style=\"opacity: 0.80\"> to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.</span><span style=\"background-color: hsl(120, 100.00%, 89.88%); opacity: 0.83\" title=\"0.056\">005</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 86.86%); opacity: 0.84\" title=\"0.081\">mg</span><span style=\"opacity: 0.80\">/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\">. ao4, d22, and uacc2534 cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> grown in rpmi supplemented with 10% fetal bovine </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\">. sbcl2, ao4, d22, and uacc2534 melanoma cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> transduced with empty vector, fgfr2, or </span><span style=\"background-color: hsl(120, 100.00%, 99.44%); opacity: 0.80\" title=\"0.001\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> plated </span><span style=\"background-color: hsl(120, 100.00%, 99.34%); opacity: 0.80\" title=\"0.001\">per</span><span style=\"opacity: 0.80\"> well in a 96-well plate in full growth media. the next day, cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> washed in pbs and switched to low-</span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\"> media overnight (0.5% fetal bovine </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\"> for sbcl2 and 1% for ao4, d22, and uacc2534). cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> then stimulated with 10 ng/ml fgf2 and 10 g/ml heparin in low-</span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\"> media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 96.50%); opacity: 0.81\" title=\"0.012\">added</span><span style=\"opacity: 0.80\"> on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> washed in pbs and resuspended in baf media containing no fetal bovine </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">serum</span><span style=\"opacity: 0.80\"> and no il-3 (baf starve media) for 2 h. cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> pelleted and resuspended in either baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> resolved on a 4% to 12% bis tris </span><span style=\"background-color: hsl(0, 100.00%, 96.63%); opacity: 0.81\" title=\"-0.012\">gel</span><span style=\"opacity: 0.80\"> and probed for phosho-p44/42 mapk. blots </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">were</span><span style=\"opacity: 0.80\"> then stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for </span><span style=\"background-color: hsl(0, 100.00%, 99.26%); opacity: 0.80\" title=\"-0.001\">each</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.57%); opacity: 0.80\" title=\"-0.003\">mutant</span><span style=\"opacity: 0.80\"> divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=3\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.002</b>, score <b>-3.452</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__65\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__592\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__kidney\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__passengers\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__truly\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fda\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__avenue\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__argued\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pierce\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__elmer\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__portions\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dysfunction\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sliding\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__p53\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__probe\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__remarkably\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__dataset\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mrna\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__university\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__link\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.53%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recombination\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__043\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.07%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.038\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__siblings\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.068\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__student\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.077\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__listed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.01%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.112\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__3d\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 87.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.492\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 87.54%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -1.533\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the </span><span style=\"background-color: hsl(120, 100.00%, 96.96%); opacity: 0.81\" title=\"0.010\">american</span><span style=\"opacity: 0.80\"> cancer society estimated that there would be 62,480 new cases of melanoma </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic expression of growth factors </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgf16-fgf23), which signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> their tissue-specific alternatively spliced b isoform (epithelial) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c isoform (mesenchymal; ref. 3). previous studies have shown that survival </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> proliferation of melanocytes depends on fgf2, which is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> melanocytes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> metastatic melanoma cell lines resulted in decreased proliferation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> primary </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor growth </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, aberrant splicing, or missense mutations has been reported in </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> bladder cancer </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> to a lesser extent in multiple myeloma </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (12).  we report here the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 8 of 100 (8%) of uncultured melanoma tumors. mapping of these mutations onto the known crystal structures of fgfr2 together with in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uncultured primary </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.003\">genes</span><span style=\"opacity: 0.80\"> by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (table 1). no mutations were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma tumors </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a panel of 72 vertical growth phase primary melanomas representing samples from different melanoma subtypes for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 5 of 72 primary tumors (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding normal tissue, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> p708s), which were not further </span><span style=\"background-color: hsl(0, 100.00%, 82.00%); opacity: 0.86\" title=\"-0.127\">analyzed</span><span style=\"opacity: 0.80\"> because there was a high </span><span style=\"background-color: hsl(0, 100.00%, 94.26%); opacity: 0.81\" title=\"-0.025\">probability</span><span style=\"opacity: 0.80\"> that these mutations represented passenger mutations rather than driver mutations. notably, two primary tumors carried nonsense mutations (table 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> braf were observed in tumors with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 structures  extracellular mutations. to gain insights </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> how the different fgfr2 mutations might affect receptor function, we </span><span style=\"background-color: hsl(0, 100.00%, 82.00%); opacity: 0.86\" title=\"-0.127\">analyzed</span><span style=\"opacity: 0.80\"> them in the light of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g271e mutations, which affect residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; fig. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations on heparin/heparan sulfate </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with each other directly </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> each receptor makes contacts with both ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular mutations identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the extracellular ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> region of fgfr is colored as follows: d2 in green, d3 in cyan, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in addition to the mutated residue, other relevant receptor residues are shown as ball-</span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">-sticks. a. the molecular interactions of g226 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> v247, which correspond to the mutated g227 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> contributes to proper f-g strand pair formation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with other residues in the interior core of d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> neighboring residues are shown in cyan (v247) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf ligand. mutation of r251 to q should substantially reduce the </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> n304 in fgfr1, which correspond to the mutated g271 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g305 in fgfr2. g270 plays a role in formation of turn between a </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> b by making hydrogen bonds with other residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, on the other hand, maps to the large loop region between strand c </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c, which protrudes from d3 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> does not play </span><span style=\"background-color: hsl(0, 100.00%, 89.59%); opacity: 0.83\" title=\"-0.058\">any</span><span style=\"opacity: 0.80\"> apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> t212 correspond to e160 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> h213 in fgfr2, respectively, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> mutation of these residues in melanoma could alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is highlighted in the </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> of the second fgf in the 2:2 fgf-fgfr dimer </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo ligand- </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c denote the nh2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cooh termini of fgfr1c, respectively. ligand interacts with the second </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> third immunoglobulin domains (hereafter referred to as d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its primary fgf2, suggesting that the r251q mutation should diminish the </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> affinity of fgfr2 toward fgf2 (fig. 1b). by </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, the e219k mutation maps to the secondary ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 ligand (fig. 1e). therefore, </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> heparin-induced dimerization.  other extracellular mutations including v248d, g227e, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g271e are predicted to destabilize the tertiary fold of either d2 (v248d </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 where its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (fig. 1a). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> is spatially in close vicinity to v247 (fig. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3, respectively. therefore, </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on the structure, these substitutions should cause local structural perturbations, which could ultimately affect the folding of the whole d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3, respectively. destabilization of d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct processing/maturation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on the structure, the e160a </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> h213y mutations are </span><span style=\"background-color: hsl(0, 100.00%, 93.22%); opacity: 0.82\" title=\"-0.032\">likely</span><span style=\"opacity: 0.80\"> to affect the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on which a key lysine residue involved in heparan sulfate </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> resides (fig. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> thereby attenuate fgf- </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> the heparin </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the bulkier tyrosine side chain at this location could cause steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  kinase domain mutations. more than half of the melanoma mutations affect the conserved tyrosine kinase domain of fgfr2, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> mapping of these mutations onto unphosphorylated </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> phosphorylated fgfr2 kinase domain structures (pdb id: 2psq </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (fig. 2 ). notably, m640 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> i642 localize onto either end of the 8 strand (fig. 2), a secondary structure element that precedes the activation loop (a-loop) of the kinase domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the kinase domain. substitutions of either residues with the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the kinase that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation maps to the activation loop of the kinase domain (fig. 2e) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> is also expected to lead to loss of function. this is because in the phosphorylated activated kinase domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the kinase </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c kinase domain mutations identified in melanoma tumors/cell lines mapped onto the crystal structures of the fgfr2 kinase domain suggest that these mutations should impair the kinase activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 kinase domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular tyrosine kinase domain is as follows: the nh2-terminal lobe of kinase is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the kinase hinge region is in green; </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the nh2-terminal tail of the kinase is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c-lobe of the kinase domain. to assist the viewer, certain  strands </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">  helices, which are relevant for explanation of the effects of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 8 strands, makes hydrogen bond with s563 in the kinase hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> hence alter the tyrosine kinase activity of fgfr2. b. the m640 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> i642 side chains (red mesh) point </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> the inner hydrophobic core of the kinase domain </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> engage c606 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> lead to a reduction in kinase activity. c. the hydrogen bonds between r759 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> side chains of negatively charged e731 (in helix h) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d756 (in helix i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the kinase domain, specifically making the kinase domain temperature sensitive </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> ultimately leading to receptor loss of function. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> backbone atom of t555 (located in the loop between 4 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> l560 from these regions </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> l528, v532, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does not play </span><span style=\"background-color: hsl(0, 100.00%, 89.59%); opacity: 0.83\" title=\"-0.058\">any</span><span style=\"opacity: 0.80\"> role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is expected to lead to loss of function because the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the kinase region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> indirectly affect the a-loop conformation. the e636k mutation maps to the loop region between the 7 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 8 strands (fig. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge region of the kinase domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cooh-terminal lobes of the kinase relative to each other. </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on the structure, the e636k mutation should influence the kinase activity of the fgfr2 kinase domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved kinase domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated kinase domains, however, these hydrogen bonds are weaker </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this inhibition </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> potentially lead to activation of the kinase domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the kinase </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> reduce the half life of the </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2. the g701s mutation maps to the loop between f </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g helices (fig. 2f) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> facilitates the conformation of this loop region, which makes </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> hydrogen bonds with k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced kinase activity. the r759q mutation maps to the cooh-terminal end of the i helix, the last secondary structure element of the kinase domain. in the structure, r759 makes hydrogen bonds with two other residues in its vicinity (fig. 2c), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in kinase activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> thus is required for plc phosphorylation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex with phosphopeptide indicates that the sh2 domain of plc interacts with ptyr </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> pocket of the kinase domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal structures. consistent with these analyses </span><span style=\"background-color: hsl(0, 100.00%, 97.94%); opacity: 0.80\" title=\"-0.006\">predicting</span><span style=\"opacity: 0.80\"> that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a third sample carried a splicing mutation that would result in premature termination of fgfr2 protein translation (table 1).  view inlineview popup table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test our structural predictions for the extracellular mutations on fgfr2 function, we expressed recombinant wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> ectodomains in e. coli, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> then isolated the inclusion bodies enriched in the proteins </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> subjected them to in vitro refolding. as a positive control, we also used the s252w </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> ectodomain. we have previously shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> the g227e, v248d, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> d3 domains. by </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, the r251q </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> ectodomain folded in vitro with yields similar to those of the wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> s252w </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> receptors. we then used surface plasmon resonance to analyze the effect of the r251q mutation on ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgf1, fgf4, fgf7, fgf8, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgf10 (data not shown). on the other hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> other fgfs (refs. 17-19; data not shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\">. a </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of time. surface plasmon resonance analysis was done as previously described (19). to test the effect of kinase domain mutations, we prepared recombinant wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase domain proteins harboring the e475k, d530n, i642v, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648t melanoma mutations </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> subjected them to in vitro kinase assays. consistent with structural predictions, the d530n, i642v, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648t </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase domains exhibited reduced kinase activity relative to the wild-type kinase domain (fig. 4 ). interestingly, the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> exhibited a slight increase in kinase activity relative to the wild-type kinase domain. notably, the protein expression yield for the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase was severalfold less than that of wild-type kinase, suggesting that this mutation was partially destabilizing the kinase domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 kinase domains harboring melanoma mutations have reduced tyrosine autophosphorylation activity. wild-type fgfr2k </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> four </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro kinase assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2ks were determined using a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured cells  we next </span><span style=\"background-color: hsl(0, 100.00%, 82.00%); opacity: 0.86\" title=\"-0.127\">analyzed</span><span style=\"opacity: 0.80\"> the effect of </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> mutations (g227e, r251q, g271e, e475k, d530n, i642v, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648t) on the receptor trafficking </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> increased signaling from intracellular compartments (20). in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, the s252w mutation observed in patients with apert syndrome has no effect on receptor folding (19). the k517r </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> receptor served as a kinase-deficient receptor. k517 coordinates the phosphate groups of atp, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> its mutation to arginine in the homologous fgfr3 has been previously shown to abolish kinase activity (21).  following transfection of wild-type fgfr2 receptor </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> hek293 cells, two major fgfr2-immunoreactive bands (110 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 130 kda) are observed. pngase digestion resulted in the reduction in size of both bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (fig. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (fig. 5a </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> b). the r251q </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with our structural predictions of no adverse effects on protein stability by these mutations. in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, only 30% of g227e, v248d, g271e, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent with our structural predictions that these mutations will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> (fig. 5b, bottom), consistent with the decreased protein yield observed in the in vitro kinase assays.  figure 5. download figureopen in new tabdownload powerpoint figure 5. mutations in fgfr2 impair receptor processing </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c278f mutants. c. localization of wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> mutated fgfr2 following transient transfection </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> hek293 cells </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma </span><span style=\"background-color: hsl(120, 100.00%, 98.10%); opacity: 0.80\" title=\"0.005\">membrane</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> also in the rough er </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> golgi (fig. 5c). in cells transfected with the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> (fig. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear </span><span style=\"background-color: hsl(120, 100.00%, 98.10%); opacity: 0.80\" title=\"0.005\">membrane</span><span style=\"opacity: 0.80\">, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also </span><span style=\"background-color: hsl(0, 100.00%, 96.75%); opacity: 0.81\" title=\"-0.011\">showed</span><span style=\"opacity: 0.80\"> mislocalization to intracellular membranes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> decreased cell surface expression for fgfr2 mutants v248d, g271e, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> c278f. in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, immunofluorescence of the a648t </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> shows clear cell surface localization in keeping with the structural predictions </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> endohf analyses. to confirm the er localization of the g227e </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\">, we then performed similar immunofluorescence studies looking for colocalization with the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, marked cell surface localization </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> limited colocalization with protein disulfide isomerase were observed for both the wild-type fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> p44/42 mitogen-activated protein kinase phosphorylation in response to fgf2 as readout of receptor function  we next </span><span style=\"background-color: hsl(0, 100.00%, 82.00%); opacity: 0.86\" title=\"-0.127\">analyzed</span><span style=\"opacity: 0.80\"> the effects of the melanoma mutations on fgfr2c activity. given that melanocytes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> melanoma cell lines express other fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> to </span><span style=\"background-color: hsl(0, 100.00%, 92.07%); opacity: 0.82\" title=\"-0.040\">determine</span><span style=\"opacity: 0.80\"> receptor activity. the baf3 cell </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> is routinely used as a model system for the evaluation of fgfr function because it does not express endogenous fgf ligands or receptors. although baf3 cell proliferation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> survival is normally dependent on il-3, activated tyrosine kinase signaling can substitute for il-3 to maintain cell viability </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (12) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> selection in geneticin. proliferation assays were then done in the absence of il-3 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in the presence of 1 nmol/l fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 10 g/ml heparin, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cell viability was assayed after 5 days. as anticipated, the constitutively active n549k </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> constructs including g227e, g271e, e475k, i642v, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (fig. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2_ires_neo expression constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on stimulation with exogenous rfgf2 shows that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal expression of fgfr2 by all mutants except empty vector in the absence () </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-type receptor. as a further assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein kinase (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k known activating </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\">, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (fig. 6b-d).  reintroduction of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> melanoma cells  we next sought to </span><span style=\"background-color: hsl(0, 100.00%, 92.07%); opacity: 0.82\" title=\"-0.040\">determine</span><span style=\"opacity: 0.80\"> whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr with isoform-specific primers </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> subsequent sequencing revealed normal human melanocytes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, our reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in melanoma cell lines that expressed </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 (ao4, d22, uacc2534) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> (fig. 7b). similar results were also achieved in both the d22 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uacc2534 melanoma cell lines (data not shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> ao4 were stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or kinase-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression was confirmed by western blot analysis. discussion  in this study, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 8 of 100 uncultured metastases </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> primary tumors. to </span><span style=\"background-color: hsl(0, 100.00%, 92.07%); opacity: 0.82\" title=\"-0.040\">determine</span><span style=\"opacity: 0.80\"> the ratio of nonsynonymous to synonymous mutations in our data set, we went back </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> examined the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been previously identified in dbsnp. </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> additional synonymous variants occurred alongside each other in multiple individuals, consistent with the presence of a rare germ-</span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> we anticipate that further sequencing analysis in a greater number of primary tumors </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> nevi will shed light on when during melanoma development these mutations arise. it should be noted that the mutation frequency reported in this study may be an underestimate. this is </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> not all primary tumors provided high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined with in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> (r251q), incomplete processing of the receptor </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> retention in the er (v248d, g227e, g271e), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> complete or partial loss of tyrosine kinase activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> heparan sulfateinduced receptor dimerization </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> reduced </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> to plc.  the loss-of-function nature of our identified melanoma mutations is also consistent with </span><span style=\"background-color: hsl(120, 100.00%, 92.84%); opacity: 0.82\" title=\"0.034\">published</span><span style=\"opacity: 0.80\"> literature. in support of d530n resulting in fgfr2 loss of function, a germ-</span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> d771n mutation at the equivalent codon in ret has been associated with hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation was identified in the germ </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> of affected patients in two independent families with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent with our in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in vivo data, a recent study </span><span style=\"background-color: hsl(0, 100.00%, 96.75%); opacity: 0.81\" title=\"-0.011\">showed</span><span style=\"opacity: 0.80\"> that the a648t mutation abrogated the tyrosine kinase activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the kinase n-lobe toward the c-lobe, which is predicted to take place when the kinase transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 kinase activity awaits the resolution of crystal structure of the </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase domain as was recently carried out for an a628t ladd mutation. in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\"> to the a648t mutation, the a628t mutation maps onto the catalytic loop of the kinase domain, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the crystal structure of the </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those our lab previously identified in endometrial cancer highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgfr3 mutations previously reported in the germ </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\">, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer with germ-</span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> mutations associated with skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in </span><span style=\"background-color: hsl(120, 100.00%, 98.47%); opacity: 0.80\" title=\"0.004\">contrast</span><span style=\"opacity: 0.80\">, the more than 20 fgfr2 mutations we identified in melanoma were all novel, supporting our findings that the mutations are distinct </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> different from the many gain-of-function mutations previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-</span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uncultured tumors are presented in red above the schematic representation of the protein </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cell lines are presented in green (top) above the schematic representation of the protein </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-</span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\"> mutations associated with a variety of craniosynostosis syndromes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although not previously reported in the germ </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\">, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\">, whereas every melanoma mutation was novel. this discovery of loss-of-function mutations in a receptor tyrosine kinase is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., kit, flt3, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgfr3) drives tumorigenesis </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> our previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (12). however, it should not be surprising given that a variety of receptor tyrosine kinase signaling pathways can induce either proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic mice, with fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 12-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased number of papillomas </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> carcinomas compared with wild-type mice, providing support that fgfr2 can act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated with </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> cancers including prostate </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-36), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> salivary adenocarcinoma cells (37).  </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on these data </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> our structural, biochemical, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are </span><span style=\"background-color: hsl(0, 100.00%, 98.79%); opacity: 0.80\" title=\"-0.003\">several</span><span style=\"opacity: 0.80\"> possible explanations for these apparent disparate results. one difference between the previously </span><span style=\"background-color: hsl(120, 100.00%, 92.84%); opacity: 0.82\" title=\"0.034\">published</span><span style=\"opacity: 0.80\"> reintroduction results </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the current melanoma study is that they involve different fgfr2 splice forms. the previous studies were all carried out in cells of epithelial origin with the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.003\">genes</span><span style=\"opacity: 0.80\"> exist, which function to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively high frequency </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.001\">understanding</span><span style=\"opacity: 0.80\"> about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> cooh-terminal domain of fgfr2 (c1, c3), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the role of these splice variants in normal melanocyte function </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> melanoma pathogenesis is currently </span><span style=\"background-color: hsl(0, 100.00%, 82.16%); opacity: 0.86\" title=\"-0.126\">unknown</span><span style=\"opacity: 0.80\">. our lab is currently investigating the relative expression levels of these various isoforms </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> their functional significance in melanocytes </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine kinase signaling. there have been reports that fgfr1 binds cbp </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> rsk in the nucleus (38), </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> we </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> others have observed that fgfr2 localized to the nucleus in various cell types (39).8 lastly, we </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (40)8 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.001\">understanding</span><span style=\"opacity: 0.80\"> of fgfr biology will allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\">/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.003\">genes</span><span style=\"opacity: 0.80\"> that play context-dependent opposing roles in tumorigenesis. although many </span><span style=\"background-color: hsl(120, 100.00%, 98.66%); opacity: 0.80\" title=\"0.003\">genes</span><span style=\"opacity: 0.80\"> have been shown to have context-dependent opposing roles in cancer (e.g., notch </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> tgf; ref. 42), fgfr2 may be the first gene in which both loss-of-function </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> gain-of-function mutations have been reported in different tissue types. ongoing work in our laboratory is targeted toward </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.001\">understanding</span><span style=\"opacity: 0.80\"> the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.37%); opacity: 0.80\" title=\"0.008\">methods</span><span style=\"opacity: 0.80\">  see supplementary </span><span style=\"background-color: hsl(120, 100.00%, 97.37%); opacity: 0.80\" title=\"0.008\">methods</span><span style=\"opacity: 0.80\"> for details on mutation detection.  expression, refolding, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> purification of ectodomain of fgfr2  dna fragment </span><span style=\"background-color: hsl(0, 100.00%, 99.88%); opacity: 0.80\" title=\"-0.000\">encoding</span><span style=\"opacity: 0.80\"> residues 147 to 366 of fgfr2c was amplified by pcr </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> subcloned </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> pet-28a bacterial expression vector using ncoi </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed with wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> expression constructs </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> centrifuged; the inclusion bodies </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> 150 mmol/l nacl, 10% glycerol, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> l-cysteine. the correctly refolded wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> proteins were then sequentially purified by heparin affinity </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  expression </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> purification of fgfr2 kinase domain  the dna fragment </span><span style=\"background-color: hsl(0, 100.00%, 99.88%); opacity: 0.80\" title=\"-0.000\">encoding</span><span style=\"opacity: 0.80\"> residues 458 to 768 of human fgfr2 was subcloned </span><span style=\"background-color: hsl(0, 100.00%, 98.48%); opacity: 0.80\" title=\"-0.004\">into</span><span style=\"opacity: 0.80\"> pet bacterial expression vectors with an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed with wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 kinase expression constructs, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were then lysed using a french press, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> the soluble kinase proteins were purified by using sequential ni2+-chelating, anion exchange </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% </span><span style=\"background-color: hsl(120, 100.00%, 96.65%); opacity: 0.81\" title=\"0.012\">based</span><span style=\"opacity: 0.80\"> on sds-page analysis. purified wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinase proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> kinases was compared using a continuous spectrophotometric kinase assay according to the </span><span style=\"background-color: hsl(120, 100.00%, 92.84%); opacity: 0.82\" title=\"0.034\">published</span><span style=\"opacity: 0.80\"> protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> to ensure that no other mutations were introduced during the mutagenesis process. plasmid dna was then isolated using qiagen endofree plasmid dna isolation kits, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> transfections  all cell lines described in this series of experiments were maintained under standard culturing </span><span style=\"background-color: hsl(120, 100.00%, 97.28%); opacity: 0.80\" title=\"0.009\">conditions</span><span style=\"opacity: 0.80\"> as described by the </span><span style=\"background-color: hsl(120, 100.00%, 96.96%); opacity: 0.81\" title=\"0.010\">american</span><span style=\"opacity: 0.80\"> type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell </span><span style=\"background-color: hsl(120, 100.00%, 99.48%); opacity: 0.80\" title=\"0.001\">line</span><span style=\"opacity: 0.80\">). transient transfections of 1 g total dna (consisting of 250 g wild-type or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 + 750 g empty plasmid) were achieved with fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> western blotting  at 36 to </span><span style=\"background-color: hsl(0, 100.00%, 97.39%); opacity: 0.80\" title=\"-0.008\">48</span><span style=\"opacity: 0.80\"> h posttransfection, cells were washed with ice-cold pbs </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved on 3% to 8% tris-acetate gels </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes were probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> plated onto eight-well chamber slides. after 36 to </span><span style=\"background-color: hsl(0, 100.00%, 97.39%); opacity: 0.80\" title=\"-0.008\">48</span><span style=\"opacity: 0.80\"> h, cells were fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> incubated with 1:300 bek c-17 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 1:1,000 antiprotein disulfide isomerase (invitrogen) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> then detected with goat ant-rabbit cy3 (1:200; jackson immunoresearch) </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted with pro long gold antifade reagent </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected with lentiviral packaging vectors </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> ptyf.fgfr2.ires.neo (or </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> the virus were collected at 24 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 40 h, pooled, filtered through a 0.45-um low protein </span><span style=\"background-color: hsl(0, 100.00%, 98.73%); opacity: 0.80\" title=\"-0.003\">binding</span><span style=\"opacity: 0.80\"> filter, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> stored at 80c until use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with crystal violet, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> fgfr2 were maintained under selection in rpmi/10% fetal bovine serum </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> supplemented with 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> washed in the above media </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> 16.7 ng/ml (1 nmol/l) fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 10 g/ml heparin. half the volume of media was removed at day 3 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> replaced with an equal volume </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> fresh ligand at 2 concentration </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 2% fetal bovine serum. ao4, d22, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uacc2534 cells were grown in rpmi supplemented with 10% fetal bovine serum. sbcl2, ao4, d22, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uacc2534 melanoma cells were transduced with empty vector, fgfr2, or kinase-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full growth media. the next day, cells were washed in pbs </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 1% for ao4, d22, </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> uacc2534). cells were then stimulated with 10 ng/ml fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 10 g/ml heparin in low-serum media </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> proliferation was assessed using the sulforhodamine b assay. fresh fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> heparin were added on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells were washed in pbs </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> resuspended in baf media </span><span style=\"background-color: hsl(0, 100.00%, 94.07%); opacity: 0.81\" title=\"-0.026\">containing</span><span style=\"opacity: 0.80\"> no fetal bovine serum </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> no il-3 (baf starve media) for 2 h. cells were pelleted </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> resuspended in either baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved on a 4% to 12% </span><span style=\"background-color: hsl(0, 100.00%, 95.79%); opacity: 0.81\" title=\"-0.016\">bis</span><span style=\"opacity: 0.80\"> tris gel </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> probed for phosho-p44/42 mapk. blots were then stripped </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">and</span><span style=\"opacity: 0.80\"> presented as the ratio of phosphorylated over total p44/42 mapk for each </span><span style=\"background-color: hsl(0, 100.00%, 98.49%); opacity: 0.80\" title=\"-0.004\">mutant</span><span style=\"opacity: 0.80\"> divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=4\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.011</b>, score <b>-1.855</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 92.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.727\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.051\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__potential\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.032\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__efficacy\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.28%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.026\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anchorage\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.38%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__digested\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__demonstrate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.43%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__codons\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__respond\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__therapeutics\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__smad2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__useful\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stem\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__humans\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__s2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__insensitive\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__f3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__otherwise\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__taq\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stromal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__identifying\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__code\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__modeling\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lacking\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__smad3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__follow\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__subcellular\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__3h\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__assessing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hmsh2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__p70\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__arrest\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__orthologs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inactivated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bona\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__throughout\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__investigated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__field\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__heterozygous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cl\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__trypsinized\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__00\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__implying\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pi3k\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.58%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__106\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__checked\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.54%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__e3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__iv\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__genomes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.49%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hela\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.47%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__msh2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__germline\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__affects\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__downstream\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.43%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.019\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__seems\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.36%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__retardation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.20%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.030\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__defective\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.16%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.032\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sense\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.14%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.034\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__yielded\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.10%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__led\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.09%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.037\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mucinous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.050\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__null\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.055\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__non\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 83.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -2.246\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> society estimated that there would be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes and melanoma cells </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> revealed that one of the hallmarks of melanocytic </span><span style=\"background-color: hsl(0, 100.00%, 94.25%); opacity: 0.81\" title=\"-0.025\">transformation</span><span style=\"opacity: 0.80\"> is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic expression of growth factors and/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf </span><span style=\"background-color: hsl(0, 100.00%, 98.23%); opacity: 0.80\" title=\"-0.005\">family</span><span style=\"opacity: 0.80\"> comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> that survival and proliferation of melanocytes depends on fgf2, which is provided by the surrounding keratinocytes in the skin. a </span><span style=\"background-color: hsl(0, 100.00%, 97.16%); opacity: 0.80\" title=\"-0.009\">number</span><span style=\"opacity: 0.80\"> of functional studies </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> implicated both fgf2 and fgfr1 signaling in melanoma </span><span style=\"background-color: hsl(0, 100.00%, 98.50%); opacity: 0.80\" title=\"-0.004\">progression</span><span style=\"opacity: 0.80\">. adenoviral transduction of melanocytes </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf2 resulted in phenotypic changes consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> to result in inhibition of tumor growth and induction of apoptosis (10, 11).  constitutive </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> of fgfrs through chromosomal translocation, aberrant splicing, or </span><span style=\"background-color: hsl(0, 100.00%, 97.30%); opacity: 0.80\" title=\"-0.008\">missense</span><span style=\"opacity: 0.80\"> mutations has been reported in several cancers. the catalog of somatic mutations in </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> (cosmic)7 provides a repository of all somatic changes reported to date in this receptor </span><span style=\"background-color: hsl(0, 100.00%, 98.23%); opacity: 0.80\" title=\"-0.005\">family</span><span style=\"opacity: 0.80\">. to date, fgfr3 is </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> at highest frequency in benign seborrheic keratoses and bladder </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> and to a lesser extent in multiple myeloma and cervical </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> recently reported the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(120, 100.00%, 97.44%); opacity: 0.80\" title=\"0.008\">activating</span><span style=\"opacity: 0.80\"> mutations in fgfr2 in 16% of endometrioid endometrial cancers (12).  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> report here the identification of </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">. mapping of these mutations onto the known crystal structures of fgfr2 together </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> screened an initial panel of 47 melanoma cell lines for the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations were </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> in a panel of lymphocyte dna from 150 caucasian controls (data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (table 1). no mutations were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 mutations next, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> screened a panel of 28 metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> and a panel of 72 vertical growth phase primary melanomas representing samples from different melanoma </span><span style=\"background-color: hsl(0, 100.00%, 73.19%); opacity: 0.91\" title=\"-0.225\">subtypes</span><span style=\"opacity: 0.80\"> for the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of mutations in fgfr2 gene. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> identified mutations in 3 of 28 metastatic samples and 5 of 72 primary </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> were able to extract dna from surrounding normal tissue, and in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> kinome </span><span style=\"background-color: hsl(0, 100.00%, 97.67%); opacity: 0.80\" title=\"-0.007\">screen</span><span style=\"opacity: 0.80\"> (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, and p708s), which were not </span><span style=\"background-color: hsl(0, 100.00%, 96.83%); opacity: 0.81\" title=\"-0.011\">further</span><span style=\"opacity: 0.80\"> analyzed because there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> carried </span><span style=\"background-color: hsl(120, 100.00%, 77.05%); opacity: 0.89\" title=\"0.180\">nonsense</span><span style=\"opacity: 0.80\"> mutations (table 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">). mutations in both nras and braf were </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 structures  extracellular mutations. to gain insights into how the different fgfr2 mutations might </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> receptor function, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> analyzed them in the </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">light</span><span style=\"opacity: 0.80\"> of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.001\">entry</span><span style=\"opacity: 0.80\"> 1fq9; ref. 14; fig. 1 ). </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> each other directly and each receptor makes contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> both ligands as well.  figure 1. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 1. mapping of fgfr2c extracellular mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.010\">impair</span><span style=\"opacity: 0.80\"> fgfr2c activity. the locations of </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand binding region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> receptor residues are located. in each image, in addition to the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> residue, other relevant receptor residues are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in d3. therefore, mutation of g271 in fgfr2 should negatively </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the tertiary folding of d3. n304, on the other hand, maps to the large loop region between strand c and c, which protrudes from d3 and does not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> fgfr2s </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> its primary fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (fig. 1b). by contrast, the e219k mutation maps to the secondary ligand binding site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k35 of the secondary fgf2 ligand (fig. 1e). therefore, based on the structure, e219k should not </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other extracellular mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (fig. 1a). substitution of v248 </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (fig. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based on the structure, these substitutions should cause local structural perturbations, which could ultimately </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.010\">impair</span><span style=\"opacity: 0.80\"> the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the correct processing/maturation and trafficking of the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the interaction of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> of ga helix in d2, on which a key lysine residue involved in heparan sulfate binding resides (fig. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfate</span><span style=\"background-color: hsl(0, 100.00%, 71.45%); opacity: 0.92\" title=\"-0.246\">mediated</span><span style=\"opacity: 0.80\"> fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), and the bulkier tyrosine side chain at this location could cause steric clashes </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations. </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">more</span><span style=\"opacity: 0.80\"> than half of the melanoma mutations </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the conserved tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (fig. 2 ). notably, m640 and i642 localize onto either end of the 8 strand (fig. 2), a secondary structure element that precedes the </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> loop (a-loop) of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. the hydrophobic side chains of these residues are in contact </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the e helix in the innermost core of the c-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. substitutions of either residues </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> that should induce local structural perturbations that ultimately </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> loop. the a648t mutation maps to the </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> loop of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (fig. 2e) and is also expected to lead to loss of function. this is because in the phosphorylated activated </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 2. fgfr2c </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">/cell lines mapped onto the crystal structures of the fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain </span><span style=\"background-color: hsl(0, 100.00%, 78.51%); opacity: 0.88\" title=\"-0.164\">suggest</span><span style=\"opacity: 0.80\"> that these mutations should </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.010\">impair</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. the positions of the </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> residues are mapped onto the crystal structure of unphosphorylated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (pdb </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.001\">entry</span><span style=\"opacity: 0.80\">: 2psq; ref. 15). the coloring of the intracellular tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain is as follows: the nh2-terminal lobe of </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> is in </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">light</span><span style=\"opacity: 0.80\"> blue; the cooh-terminal lobe is in bright blue; the </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> loop is in magenta; the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> hinge region is in green; and the nh2-terminal tail of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> is colored wheat. note that atp (not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">) binds in the cleft between the n-lobe and c-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the </span><span style=\"background-color: hsl(0, 100.00%, 97.59%); opacity: 0.80\" title=\"-0.007\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s563 in the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a reduction in </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, specifically making the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the region </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> e475 is located. the side chain of e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> as colored mesh. e. close-up view of the region </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> d530 and a648 are located. in unphosphorylated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2k structure, d530 does not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb </span><span style=\"background-color: hsl(0, 100.00%, 99.27%); opacity: 0.80\" title=\"-0.001\">entry</span><span style=\"opacity: 0.80\">: 2pvf; ref. 15), the side chain of d530 engages in salt bridge </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> r664 in the a-loop and contributes to a-loop </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> in the active state (not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> threonine is expected to lead to loss of function because the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> region </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> g701 is located. g701 plays a role in stabilizing the </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> of the loop between the helices f and g, which is in hydrogen-bonding contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the a-loop </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\">. the e636k mutation maps to the loop region between the 7 and 8 strands (fig. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s568, which is located in the hinge region of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> relative to each other. based on the structure, the e636k mutation should influence the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of the fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. the e475 is </span><span style=\"background-color: hsl(0, 100.00%, 93.54%); opacity: 0.81\" title=\"-0.029\">part</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> consensus motif that defines approximately the nh2-terminal boundary of the conserved </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> t555 whose methyl group is in hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this inhibition and potentially lead to </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (fig. 2f) and facilitates the </span><span style=\"background-color: hsl(0, 100.00%, 96.98%); opacity: 0.80\" title=\"-0.010\">conformation</span><span style=\"opacity: 0.80\"> of this loop region, which makes several hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity. the r759q mutation maps to the cooh-terminal end of the i helix, the last secondary structure element of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. in the structure, r759 makes hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> two other residues in its vicinity (fig. 2c), and loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> phosphopeptide indicates that the sh2 domain of plc interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate binding pocket of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal structures. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored </span><span style=\"background-color: hsl(120, 100.00%, 77.05%); opacity: 0.89\" title=\"0.180\">nonsense</span><span style=\"opacity: 0.80\"> mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 protein translation (table 1).  view inlineview popup table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test our structural predictions for the extracellular mutations on fgfr2 function, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> expressed recombinant wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant ectodomains in e. coli, and then isolated the inclusion bodies </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">enriched</span><span style=\"opacity: 0.80\"> in the proteins and subjected them to in vitro refolding. as a positive control, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> also used the s252w mutant ectodomain. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> previously </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> yields similar to those of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and s252w mutant receptors. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> then used surface plasmon resonance to analyze the effect of the r251q mutation on ligand binding affinity of fgfr2c. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the r251q mutation </span><span style=\"background-color: hsl(0, 100.00%, 96.81%); opacity: 0.81\" title=\"-0.011\">severely</span><span style=\"opacity: 0.80\"> impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">). on the other hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">).  figure 3. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> is plotted as a function of time. surface plasmon resonance analysis was done as previously described (19). to test the effect of </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain mutations, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> prepared recombinant wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> assays. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the d530n, i642v, and a648t mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domains exhibited </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain (fig. 4 ). </span><span style=\"background-color: hsl(120, 100.00%, 97.53%); opacity: 0.80\" title=\"0.007\">interestingly</span><span style=\"opacity: 0.80\">, the e475k mutant exhibited a slight increase in </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain. notably, the protein expression yield for the e475k mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> was severalfold less than that of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\">, suggesting that this mutation was partially destabilizing the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain.  figure 4. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 4. fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domains harboring melanoma mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> tyrosine autophosphorylation activity. wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2k and four mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2ks were determined using a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured cells  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r </span><span style=\"background-color: hsl(0, 100.00%, 98.04%); opacity: 0.80\" title=\"-0.005\">served</span><span style=\"opacity: 0.80\"> as controls. the c278f </span><span style=\"background-color: hsl(120, 100.00%, 97.44%); opacity: 0.80\" title=\"0.008\">activating</span><span style=\"opacity: 0.80\"> mutation associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation and </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\">. this mutation has been </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> to result in impaired receptor processing, constitutive dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> in patients </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> apert syndrome has no effect on receptor folding (19). the k517r mutant receptor </span><span style=\"background-color: hsl(0, 100.00%, 98.04%); opacity: 0.80\" title=\"-0.005\">served</span><span style=\"opacity: 0.80\"> as a </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\">-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> to abolish </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity (21).  following transfection of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\">. pngase digestion resulted in the reduction in size of both bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (fig. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 (fig. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our structural predictions of no adverse </span><span style=\"background-color: hsl(0, 100.00%, 97.59%); opacity: 0.80\" title=\"-0.007\">effects</span><span style=\"opacity: 0.80\"> on protein stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our structural predictions that these mutations will adversely </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (fig. 5b, bottom), consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the decreased protein yield </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> in the in vitro </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> assays.  figure 5. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 5. mutations in fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.010\">impair</span><span style=\"opacity: 0.80\"> receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. </span><span style=\"background-color: hsl(120, 100.00%, 82.30%); opacity: 0.86\" title=\"0.124\">sensitivity</span><span style=\"opacity: 0.80\"> to digestion </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and </span><span style=\"background-color: hsl(120, 100.00%, 98.76%); opacity: 0.80\" title=\"0.003\">mutated</span><span style=\"opacity: 0.80\"> fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> trafficking of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the cell surface localization of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 and e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> no effect on misfolding). to confirm altered cellular localization of the mutants, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (fig. 5c). in cells transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the g227e mutant (fig. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also </span><span style=\"background-color: hsl(120, 100.00%, 96.67%); opacity: 0.81\" title=\"0.011\">showed</span><span style=\"opacity: 0.80\"> mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> then performed similar immunofluorescence studies looking for colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in contrast, marked cell surface localization and limited colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> protein disulfide isomerase were </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> for both the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 and the e219k mutation, predicted to </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> no detrimental effect on receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated protein </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> phosphorylation in </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> to fgf2 as readout of receptor function  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> next analyzed the </span><span style=\"background-color: hsl(0, 100.00%, 97.59%); opacity: 0.80\" title=\"-0.007\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express other fgfrs whose </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> may mask the ability to specifically evaluate fgfr2 signaling and function, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it does not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling can substitute for il-3 to maintain cell </span><span style=\"background-color: hsl(0, 100.00%, 85.29%); opacity: 0.85\" title=\"-0.095\">viability</span><span style=\"opacity: 0.80\"> and proliferation. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> included the n549k fgfr2 receptor as a constitutively activated positive control because this </span><span style=\"background-color: hsl(120, 100.00%, 97.44%); opacity: 0.80\" title=\"0.008\">activating</span><span style=\"opacity: 0.80\"> mutation has been reported in endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> (12) and an identical mutation at the paralogous positions in fgfr3 has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were then done in the absence of il-3 and in the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of 1 nmol/l fgf2 and 10 g/ml heparin, and cell </span><span style=\"background-color: hsl(0, 100.00%, 85.29%); opacity: 0.85\" title=\"-0.095\">viability</span><span style=\"opacity: 0.80\"> was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in </span><span style=\"background-color: hsl(0, 100.00%, 97.00%); opacity: 0.80\" title=\"-0.010\">levels</span><span style=\"opacity: 0.80\"> of proliferation above that achieved by the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> receptor, whereas expression of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> cell proliferation compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2 (fig. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> to ligand in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 6. cell proliferation in </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> to fgf2 in baf3 cell lines stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2_ires_neo expression constructs. a. all </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> mutations identified in melanoma result in a decrease in baf3 proliferation when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on stimulation </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> exogenous rfgf2 shows that, </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k positive control, mutations </span><span style=\"background-color: hsl(120, 100.00%, 96.93%); opacity: 0.81\" title=\"0.010\">impair</span><span style=\"opacity: 0.80\"> the ability of the receptor to activate this pro-proliferation pathway compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> receptor. c. western blot showing equal expression of fgfr2 by all mutants except empty vector in the absence () and </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> receptor. as a </span><span style=\"background-color: hsl(0, 100.00%, 96.83%); opacity: 0.81\" title=\"-0.011\">further</span><span style=\"opacity: 0.80\"> assessment of receptor impairment due to mutation, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.003\">examined</span><span style=\"opacity: 0.80\"> the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> (mapk) signaling pathway in </span><span style=\"background-color: hsl(0, 100.00%, 93.21%); opacity: 0.82\" title=\"-0.032\">response</span><span style=\"opacity: 0.80\"> to exogenous rfgf2. </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k known </span><span style=\"background-color: hsl(120, 100.00%, 97.44%); opacity: 0.80\" title=\"0.008\">activating</span><span style=\"opacity: 0.80\"> mutant, all mutations resulted in a reduction to varying </span><span style=\"background-color: hsl(0, 100.00%, 97.00%); opacity: 0.80\" title=\"-0.010\">levels</span><span style=\"opacity: 0.80\"> in the ability of the receptor to induce phosphorylation of p44/42 mapk (fig. 6b-d).  reintroduction of fgfr2 into melanoma cells  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> isoform-specific </span><span style=\"background-color: hsl(120, 100.00%, 97.27%); opacity: 0.80\" title=\"0.009\">primers</span><span style=\"opacity: 0.80\"> and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">). for this </span><span style=\"background-color: hsl(0, 100.00%, 64.59%); opacity: 0.97\" title=\"-0.335\">reason</span><span style=\"opacity: 0.80\">, our reintroduction studies used the fgfr2c isoform. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction was </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">confirmed</span><span style=\"opacity: 0.80\"> by western blot analysis for each cell line (fig. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\">).  figure 7. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression was </span><span style=\"background-color: hsl(120, 100.00%, 98.98%); opacity: 0.80\" title=\"0.002\">confirmed</span><span style=\"opacity: 0.80\"> by western blot analysis. discussion  in this </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">study</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> report </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">. to determine the ratio of nonsynonymous to synonymous mutations in our data set, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> went back and </span><span style=\"background-color: hsl(0, 100.00%, 98.84%); opacity: 0.80\" title=\"-0.003\">examined</span><span style=\"opacity: 0.80\"> the synonymous variants </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> identified. because </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> did not </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> constitutional dna for these samples, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> excluded them as probable somatic mutations if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> one cc:gg&gt;tt:aa tandem mutation). the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in </span><span style=\"background-color: hsl(0, 100.00%, 94.25%); opacity: 0.81\" title=\"-0.025\">transformation</span><span style=\"opacity: 0.80\">, and </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> anticipate that </span><span style=\"background-color: hsl(0, 100.00%, 96.83%); opacity: 0.81\" title=\"-0.011\">further</span><span style=\"opacity: 0.80\"> sequencing analysis in a greater </span><span style=\"background-color: hsl(0, 100.00%, 97.16%); opacity: 0.80\" title=\"-0.009\">number</span><span style=\"opacity: 0.80\"> of primary </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> and nevi will shed </span><span style=\"background-color: hsl(120, 100.00%, 99.16%); opacity: 0.80\" title=\"0.002\">light</span><span style=\"opacity: 0.80\"> on when during melanoma development these mutations arise. it should be noted that the mutation frequency reported in this </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">study</span><span style=\"opacity: 0.80\"> may be an underestimate. this is based on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not all primary </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> provided high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of ligand binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> binding to plc.  the loss-of-function nature of our identified melanoma mutations is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation was identified in the germ line of </span><span style=\"background-color: hsl(0, 100.00%, 87.30%); opacity: 0.84\" title=\"-0.077\">affected</span><span style=\"opacity: 0.80\"> patients in two independent families </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> our in vitro and in vivo data, a recent </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">study</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 96.67%); opacity: 0.81\" title=\"0.011\">showed</span><span style=\"opacity: 0.80\"> that the a648t mutation abrogated the tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2 (25). as discussed earlier, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> propose that this mutation creates a steric barrier for the movement of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> n-lobe toward the c-lobe, which is predicted to take place when the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> activity awaits the resolution of crystal structure of the mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the catalytic loop of the </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain, and the crystal structure of the mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those our lab previously identified in endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations previously reported in the germ line, a </span><span style=\"background-color: hsl(0, 100.00%, 84.68%); opacity: 0.85\" title=\"-0.101\">finding</span><span style=\"opacity: 0.80\"> similar to that </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> for fgfr3 </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">where</span><span style=\"opacity: 0.80\"> a comparison of fgfr3 mutations reported in </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> germ-line mutations associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">more</span><span style=\"opacity: 0.80\"> than 20 fgfr2 mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> identified in melanoma were all </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\">, supporting our findings that the mutations are distinct and different from the many gain-of-function mutations previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload </span><span style=\"background-color: hsl(0, 100.00%, 98.38%); opacity: 0.80\" title=\"-0.004\">powerpoint</span><span style=\"opacity: 0.80\"> figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\">, as well as germ-line mutations in craniosynostosis syndromes. a. </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> somatic mutations in fgfr2 identified in melanoma cell lines and uncultured </span><span style=\"background-color: hsl(120, 100.00%, 99.41%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> are presented in red above the schematic representation of the protein and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (top) above the schematic representation of the protein and are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-line mutations associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> mutations are underlined. four somatic fgfr2 endometrial mutations, although not previously reported in the germ line, </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> an identical </span><span style=\"background-color: hsl(0, 100.00%, 97.30%); opacity: 0.80\" title=\"-0.008\">missense</span><span style=\"opacity: 0.80\"> change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> parallel those identified in the germ line, whereas every melanoma mutation was </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\">. this discovery of loss-of-function mutations in a receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> is </span><span style=\"background-color: hsl(0, 100.00%, 80.03%); opacity: 0.87\" title=\"-0.148\">unexpected</span><span style=\"opacity: 0.80\">, given the conventional wisdom that </span><span style=\"background-color: hsl(0, 100.00%, 97.31%); opacity: 0.80\" title=\"-0.008\">activation</span><span style=\"opacity: 0.80\"> of receptor tyrosine kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(120, 100.00%, 97.44%); opacity: 0.80\" title=\"0.008\">activating</span><span style=\"opacity: 0.80\"> mutations in fgfr2 in endometrial </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> (12). however, it should not be surprising given that a variety of receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling pathways can induce either proliferation or differentiation depending on the cell </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">indeed</span><span style=\"opacity: 0.80\">, cre-lox transgenic mice, </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene and 12-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased </span><span style=\"background-color: hsl(0, 100.00%, 97.16%); opacity: 0.80\" title=\"-0.009\">number</span><span style=\"opacity: 0.80\"> of papillomas and carcinomas compared </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> mice, providing support that fgfr2 can act as a tumor </span><span style=\"background-color: hsl(0, 100.00%, 71.16%); opacity: 0.93\" title=\"-0.250\">suppressor</span><span style=\"opacity: 0.80\"> gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro and </span><span style=\"background-color: hsl(120, 100.00%, 98.86%); opacity: 0.80\" title=\"0.002\">reduced</span><span style=\"opacity: 0.80\"> tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-36), and salivary adenocarcinoma cells (37).  based on these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">indeed</span><span style=\"opacity: 0.80\"> surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the previously published reintroduction results and the current melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.64%); opacity: 0.80\" title=\"0.000\">study</span><span style=\"opacity: 0.80\"> is that they involve different fgfr2 splice forms. the previous studies were all carried out in cells of epithelial origin </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.002\">indeed</span><span style=\"opacity: 0.80\">, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which function to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">affect</span><span style=\"opacity: 0.80\"> another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell </span><span style=\"background-color: hsl(0, 100.00%, 94.25%); opacity: 0.81\" title=\"-0.025\">transformation</span><span style=\"opacity: 0.80\"> but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently </span><span style=\"background-color: hsl(0, 100.00%, 80.52%); opacity: 0.87\" title=\"-0.143\">unknown</span><span style=\"opacity: 0.80\">. our lab is currently investigating the relative expression </span><span style=\"background-color: hsl(0, 100.00%, 97.00%); opacity: 0.80\" title=\"-0.010\">levels</span><span style=\"opacity: 0.80\"> of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> cellular functions distinct from typical receptor tyrosine </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> signaling. there </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> been reports that fgfr1 binds cbp and rsk in the nucleus (38), and </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> and others </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 95.55%); opacity: 0.81\" title=\"-0.017\">observed</span><span style=\"opacity: 0.80\"> that fgfr2 localized to the nucleus in various cell types (39).8 lastly, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> and others </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> that fgfr2 can heterodimerize </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr1 at least in vitro (40)8 and fgfrs </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> also been </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> postulate that a </span><span style=\"background-color: hsl(0, 100.00%, 96.68%); opacity: 0.81\" title=\"-0.011\">more</span><span style=\"opacity: 0.80\"> complete understanding of fgfr biology will allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or </span><span style=\"background-color: hsl(0, 100.00%, 98.50%); opacity: 0.80\" title=\"-0.004\">progression</span><span style=\"opacity: 0.80\">.  in conclusion, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> yet to uncover the biological consequence of loss of fgfr2 function in melanoma, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">we</span><span style=\"opacity: 0.80\"> propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> been </span><span style=\"background-color: hsl(0, 100.00%, 98.70%); opacity: 0.80\" title=\"-0.003\">shown</span><span style=\"opacity: 0.80\"> to </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> context-dependent opposing roles in </span><span style=\"background-color: hsl(120, 100.00%, 99.79%); opacity: 0.80\" title=\"0.000\">cancer</span><span style=\"opacity: 0.80\"> (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which both loss-of-function and gain-of-function mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.88%); opacity: 0.80\" title=\"0.000\">have</span><span style=\"opacity: 0.80\"> been reported in different tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning </span><span style=\"background-color: hsl(0, 100.00%, 95.44%); opacity: 0.81\" title=\"-0.018\">sites</span><span style=\"opacity: 0.80\">. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> or mutant expression constructs and protein expression was induced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant proteins were then sequentially </span><span style=\"background-color: hsl(120, 100.00%, 95.68%); opacity: 0.81\" title=\"0.017\">purified</span><span style=\"opacity: 0.80\"> by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  expression and purification of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial expression vectors </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> or mutant fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> expression constructs, and protein expression was induced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were then lysed using a french press, and the soluble </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> proteins were </span><span style=\"background-color: hsl(120, 100.00%, 95.68%); opacity: 0.81\" title=\"0.017\">purified</span><span style=\"opacity: 0.80\"> by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. </span><span style=\"background-color: hsl(120, 100.00%, 95.68%); opacity: 0.81\" title=\"0.017\">purified</span><span style=\"opacity: 0.80\"> wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant kinases was compared using a continuous spectrophotometric </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\"> assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis </span><span style=\"background-color: hsl(120, 100.00%, 97.27%); opacity: 0.80\" title=\"0.009\">primers</span><span style=\"opacity: 0.80\"> were designed to introduce a </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.006\">novel</span><span style=\"opacity: 0.80\"> but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of </span><span style=\"background-color: hsl(120, 100.00%, 97.47%); opacity: 0.80\" title=\"0.008\">clones</span><span style=\"opacity: 0.80\">. the full list of primer sequences can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of the intended mutation and to ensure that no other mutations were introduced during the mutagenesis process. plasmid dna was then isolated using qiagen endofree plasmid dna isolation kits, and dna </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained under standard culturing conditions as described by the american </span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> or mutant fgfr2 + 750 g empty plasmid) were achieved </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells were washed </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5% skim milk/tbst, membranes were probed </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described and plated onto eight-well chamber slides. after 36 to 48 h, cells were fixed </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:300 bek c-17 and 1:1,000 antiprotein disulfide isomerase (invitrogen) and then detected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and 40 h, pooled, filtered through a 0.45-um low protein binding filter, and stored at 80c </span><span style=\"background-color: hsl(0, 100.00%, 98.13%); opacity: 0.80\" title=\"-0.005\">until</span><span style=\"opacity: 0.80\"> use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crystal violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> and mutant fgfr2 were maintained under selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an equal volume containing fresh ligand at 2 </span><span style=\"background-color: hsl(0, 100.00%, 95.57%); opacity: 0.81\" title=\"-0.017\">concentration</span><span style=\"opacity: 0.80\"> and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 2 mmol/l l-glutamine supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 0.</span><span style=\"background-color: hsl(0, 100.00%, 92.17%); opacity: 0.82\" title=\"-0.039\">005</span><span style=\"opacity: 0.80\"> mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in rpmi supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty vector, fgfr2, or </span><span style=\"background-color: hsl(0, 100.00%, 99.56%); opacity: 0.80\" title=\"-0.001\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the </span><span style=\"background-color: hsl(0, 100.00%, 98.83%); opacity: 0.80\" title=\"-0.003\">presence</span><span style=\"opacity: 0.80\"> of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full growth media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were then stimulated </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/42 phosphorylation  to </span><span style=\"background-color: hsl(0, 100.00%, 92.80%); opacity: 0.82\" title=\"-0.034\">examine</span><span style=\"opacity: 0.80\"> p44/42 mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in either baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 g/ml heparin only (unstimulated or ) or baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.96%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved on a 4% to 12% bis tris gel and probed for phosho-p44/42 mapk. blots were then stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-</span><span style=\"background-color: hsl(120, 100.00%, 99.46%); opacity: 0.80\" title=\"0.001\">type</span><span style=\"opacity: 0.80\"> receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=5\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.005</b>, score <b>-2.692</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.31%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__statistical\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__incidence\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__s3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__67\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__msh2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__death\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__upfront\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__t1720a\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__331\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__followed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__coordination\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__e545k\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pi3\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__c0\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__speculation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__transfectant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__undertaken\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__calculate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__methanol\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.68%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__volumes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__establishment\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__imparts\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nevertheless\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__version\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.63%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__attributed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.55%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__preference\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__loading\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__years\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__calculated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__intervening\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recognizing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.38%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__existing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.37%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__improvements\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.36%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.022\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__besides\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.29%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.025\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__uncertain\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.19%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stringent\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.048\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__autoinhibitory\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.086\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__listed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 96.38%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.263\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 84.80%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -2.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there would be 62,480 new </span><span style=\"background-color: hsl(120, 100.00%, 88.81%); opacity: 0.83\" title=\"0.065\">cases</span><span style=\"opacity: 0.80\"> of melanoma and about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> factor stimulation by the surrounding keratinocytes in the skin to autocrine </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> factor stimulation via ectopic expression of </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> factors and/or receptors by melanoma cells (1, 2). fibroblast </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> fgf2, which is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> and induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, aberrant splicing, or missense mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser </span><span style=\"background-color: hsl(0, 100.00%, 96.11%); opacity: 0.81\" title=\"-0.014\">extent</span><span style=\"opacity: 0.80\"> in multiple myeloma and cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in </span><span style=\"background-color: hsl(0, 100.00%, 96.03%); opacity: 0.81\" title=\"-0.015\">16</span><span style=\"opacity: 0.80\">% of endometrioid endometrial cancers (12).  we report </span><span style=\"background-color: hsl(0, 100.00%, 92.51%); opacity: 0.82\" title=\"-0.036\">here</span><span style=\"opacity: 0.80\"> the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma tumors. mapping of these mutations onto the known crystal structures of fgfr2 together with in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (</span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1). no mutations were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved </span><span style=\"background-color: hsl(0, 100.00%, 99.01%); opacity: 0.80\" title=\"-0.002\">codon</span><span style=\"opacity: 0.80\"> was found in fgfr4, p716r.  view inlineview popup </span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma tumors and a panel of 72 vertical </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> phase primary melanomas representing samples from different melanoma subtypes for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples and 5 of 72 primary tumors (</span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding </span><span style=\"background-color: hsl(120, 100.00%, 97.92%); opacity: 0.80\" title=\"0.006\">normal</span><span style=\"opacity: 0.80\"> tissue, and in all </span><span style=\"background-color: hsl(120, 100.00%, 88.81%); opacity: 0.83\" title=\"0.065\">cases</span><span style=\"opacity: 0.80\"> the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, and p708s), which were </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 93.78%); opacity: 0.81\" title=\"-0.028\">further</span><span style=\"opacity: 0.80\"> analyzed because there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary tumors carried nonsense mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown). mutations in </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> nras and braf were observed in tumors with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 structures  extracellular mutations. to gain insights into how the different fgfr2 mutations might affect receptor function, we analyzed them in the light of our crystallographic data </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which affect residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; fig. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with each </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> directly and each receptor makes contacts with </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular mutations identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand binding region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in addition to the mutated residue, </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds with </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> hand, maps to the large loop region between strand c and c, which protrudes from d3 and does </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts with the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its primary fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (fig. 1b). by contrast, the e219k mutation maps to the secondary ligand binding site </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 ligand (fig. 1e). therefore, based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the structure, e219k should </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> affect the ability of fgfr2c to bind fgf2, but </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> extracellular mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal </span><span style=\"background-color: hsl(0, 100.00%, 97.13%); opacity: 0.80\" title=\"-0.009\">end</span><span style=\"opacity: 0.80\"> of the g strand in d2 where its side chain is surrounded by </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> hydrophobic residues inside the hydrophobic core of d2 (fig. 1a). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (fig. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the structure, these substitutions should cause local structural perturbations, which could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the structure, the e160a and h213y mutations are likely to affect the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> which a key lysine residue involved in heparan sulfate binding resides (fig. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), and the bulkier </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> side chain at this location could cause steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  kinase domain mutations. more than half of the melanoma mutations affect the conserved </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 kinase domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (fig. 2 ). notably, m640 and i642 localize onto either </span><span style=\"background-color: hsl(0, 100.00%, 97.13%); opacity: 0.80\" title=\"-0.009\">end</span><span style=\"opacity: 0.80\"> of the 8 strand (fig. 2), a secondary structure element that precedes the activation loop (a-loop) of the kinase domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the kinase domain. substitutions of either residues with the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the kinase that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation maps to the activation loop of the kinase domain (fig. 2e) and is also expected to lead to loss of function. this is because in the phosphorylated activated kinase domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the kinase and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and </span><span style=\"background-color: hsl(0, 100.00%, 88.79%); opacity: 0.83\" title=\"-0.065\">consequently</span><span style=\"opacity: 0.80\">, the catalytic residues would </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c kinase domain mutations identified in melanoma tumors/cell lines mapped onto the crystal structures of the fgfr2 kinase domain suggest that these mutations should impair the kinase activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 kinase domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase domain is as follows: the nh2-terminal lobe of kinase is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the kinase hinge region is in green; and the nh2-terminal tail of the kinase is colored wheat. note that atp (</span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown) </span><span style=\"background-color: hsl(0, 100.00%, 96.27%); opacity: 0.81\" title=\"-0.013\">binds</span><span style=\"opacity: 0.80\"> in the cleft between the n-lobe and c-lobe of the kinase domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the effects of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond with s563 in the kinase hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the kinase domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the </span><span style=\"background-color: hsl(0, 100.00%, 96.11%); opacity: 0.81\" title=\"-0.014\">extent</span><span style=\"opacity: 0.80\"> of these core hydrophobic contacts and lead to a reduction in kinase activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop </span><span style=\"background-color: hsl(0, 100.00%, 99.01%); opacity: 0.80\" title=\"-0.002\">codon</span><span style=\"opacity: 0.80\"> at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the kinase domain, specifically making the kinase domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop and contributes to a-loop conformation in the active state (</span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is expected to lead to loss of function because the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the kinase region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation maps to the loop region between the 7 and 8 strands (fig. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge region of the kinase domain, suggesting that </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the kinase relative to each </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\">. based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the structure, the e636k mutation should influence the kinase activity of the fgfr2 kinase domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved kinase domain in many receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated kinase domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this inhibition and potentially lead to activation of the kinase domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the kinase and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (fig. 2f) and facilitates the conformation of this loop region, which makes several hydrogen bonds with k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced kinase activity. the r759q mutation maps to the cooh-terminal </span><span style=\"background-color: hsl(0, 100.00%, 97.13%); opacity: 0.80\" title=\"-0.009\">end</span><span style=\"opacity: 0.80\"> of the i helix, the last secondary structure element of the kinase domain. in the structure, r759 makes hydrogen bonds with two </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> residues in its vicinity (fig. 2c), and loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in kinase activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex with phosphopeptide indicates that the sh2 domain of plc interacts with ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate binding pocket of the kinase domain prefers a leucine over a valine next to the </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> phosphorylation site (</span><span style=\"background-color: hsl(0, 100.00%, 96.03%); opacity: 0.81\" title=\"-0.015\">16</span><span style=\"opacity: 0.80\">). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. </span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal structures. consistent with these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 protein translation (</span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 1).  view inlineview popup </span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test our structural predictions for the extracellular mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> fgfr2 function, we </span><span style=\"background-color: hsl(0, 100.00%, 99.05%); opacity: 0.80\" title=\"-0.002\">expressed</span><span style=\"opacity: 0.80\"> recombinant wild-type and mutant ectodomains in e. coli, and </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> shown that the s252w mutation has no adverse effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro with yields similar to those of the wild-type and s252w mutant receptors. we </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> used surface plasmon resonance to analyze the effect of the r251q mutation </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> ligand binding affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown). </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> fgfs (refs. 17-19; data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of time. surface plasmon resonance analysis was done as </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> described (19). to test the effect of kinase domain mutations, we prepared recombinant wild-type and mutant kinase domain proteins </span><span style=\"background-color: hsl(0, 100.00%, 82.85%); opacity: 0.86\" title=\"-0.119\">harboring</span><span style=\"opacity: 0.80\"> the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro kinase </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\">. consistent with structural predictions, the d530n, i642v, and a648t mutant kinase domains exhibited reduced kinase activity relative to the wild-type kinase domain (fig. 4 ). interestingly, the e475k mutant exhibited a slight increase in kinase activity relative to the wild-type kinase domain. notably, the protein expression yield for the e475k mutant kinase was severalfold less than that of wild-type kinase, suggesting that this mutation was partially destabilizing the kinase domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 kinase domains </span><span style=\"background-color: hsl(0, 100.00%, 82.85%); opacity: 0.86\" title=\"-0.119\">harboring</span><span style=\"opacity: 0.80\"> melanoma mutations have reduced </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> autophosphorylation activity. wild-type fgfr2k and four mutant fgfr2ks, each </span><span style=\"background-color: hsl(0, 100.00%, 82.85%); opacity: 0.86\" title=\"-0.119\">harboring</span><span style=\"opacity: 0.80\"> e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro kinase </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\">. columns, average of three independent experiments; bars, sd. the </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> autophosphorylation activities of these wild-type and mutant fgfr2ks were determined using a continuous spectrophotometric assay as </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> described (43). analysis of fgfr2 mutations in cultured cells  we next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants </span><span style=\"background-color: hsl(0, 100.00%, 82.85%); opacity: 0.86\" title=\"-0.119\">harboring</span><span style=\"opacity: 0.80\"> c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation observed in patients with apert syndrome has no effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> receptor folding (19). the k517r mutant receptor served as a kinase-deficient receptor. k517 coordinates the phosphate </span><span style=\"background-color: hsl(0, 100.00%, 94.96%); opacity: 0.81\" title=\"-0.021\">groups</span><span style=\"opacity: 0.80\"> of atp, and its mutation to arginine in the homologous fgfr3 has been </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> shown to abolish kinase activity (21).  following transfection of wild-type fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the reduction in size of </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (fig. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. </span><span style=\"background-color: hsl(0, 100.00%, 97.74%); opacity: 0.80\" title=\"-0.007\">densitometric</span><span style=\"opacity: 0.80\"> analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (fig. 5a and b). the r251q and s252w mutations did </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with our structural predictions of no adverse effects </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> protein stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent with our structural predictions that these mutations will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, </span><span style=\"background-color: hsl(120, 100.00%, 93.60%); opacity: 0.81\" title=\"0.029\">top</span><span style=\"opacity: 0.80\">). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (fig. 5b, bottom), consistent with the decreased protein yield observed in the in vitro kinase </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\">.  figure 5. download figureopen in new tabdownload powerpoint figure 5. mutations in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 and e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to have no effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (fig. 5c). in cells transfected with the g227e mutant (fig. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping with the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> performed similar immunofluorescence studies looking for colocalization with the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in contrast, marked cell surface localization and limited colocalization with protein disulfide isomerase were observed for </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> the wild-type fgfr2 and the e219k mutation, predicted to have no detrimental effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated protein kinase phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the effects of the melanoma mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> fgfr2c activity. given that melanocytes and melanoma cell lines express </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> does </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> il-3, activated </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (12) and an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\"> were </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (fig. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> cause constitutive baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type and mutant fgfr2_ires_neo expression constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> stimulation with exogenous rfgf2 shows that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal expression of fgfr2 by all mutants </span><span style=\"background-color: hsl(120, 100.00%, 92.76%); opacity: 0.82\" title=\"0.035\">except</span><span style=\"opacity: 0.80\"> empty vector in the absence () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation </span><span style=\"background-color: hsl(0, 100.00%, 99.05%); opacity: 0.80\" title=\"-0.002\">expressed</span><span style=\"opacity: 0.80\"> as a percentage of phosphorylation (</span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> ligand stimulation) of that of wild-type receptor. as a </span><span style=\"background-color: hsl(0, 100.00%, 93.78%); opacity: 0.81\" title=\"-0.028\">further</span><span style=\"opacity: 0.80\"> assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein kinase (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k known activating mutant, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (fig. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr with isoform-specific primers and subsequent sequencing revealed </span><span style=\"background-color: hsl(120, 100.00%, 97.92%); opacity: 0.80\" title=\"0.006\">normal</span><span style=\"opacity: 0.80\"> human melanocytes and the majority of melanoma cell lines </span><span style=\"background-color: hsl(0, 100.00%, 99.05%); opacity: 0.80\" title=\"-0.002\">expressed</span><span style=\"opacity: 0.80\"> the mesenchymal fgfr2c isoform (data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown). for this </span><span style=\"background-color: hsl(120, 100.00%, 94.90%); opacity: 0.81\" title=\"0.021\">reason</span><span style=\"opacity: 0.80\">, our reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that </span><span style=\"background-color: hsl(0, 100.00%, 99.05%); opacity: 0.80\" title=\"-0.002\">expressed</span><span style=\"opacity: 0.80\"> mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell line (fig. 7b). similar results were also achieved in </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> the d22 and uacc2534 melanoma cell lines (data </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or kinase-dead fgfr2 k517r had no effect </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the proliferation of these melanoma cell lines. b. fgfr2 expression was confirmed by western blot analysis. discussion  in this study, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary tumors. to determine the ratio of nonsynonymous to synonymous mutations in our data set, we went back and examined the synonymous variants we identified. because we did </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> identified in dbsnp. several additional synonymous variants occurred alongside each </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> in multiple individuals, consistent with the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, and we anticipate that </span><span style=\"background-color: hsl(0, 100.00%, 93.78%); opacity: 0.81\" title=\"-0.028\">further</span><span style=\"opacity: 0.80\"> sequencing analysis in a greater number of primary tumors and nevi will shed light </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> when during melanoma development these mutations arise. </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> should be noted that the mutation frequency reported in this study may be an underestimate. this is based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the fact that </span><span style=\"background-color: hsl(0, 100.00%, 99.49%); opacity: 0.80\" title=\"-0.001\">40</span><span style=\"opacity: 0.80\"> cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> screened in this latter panel of cell lines, and </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> all primary tumors provided high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined with in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of ligand binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the loss-of-function nature of our identified melanoma mutations is also consistent with published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent </span><span style=\"background-color: hsl(0, 100.00%, 99.01%); opacity: 0.80\" title=\"-0.002\">codon</span><span style=\"opacity: 0.80\"> in ret has been associated with hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in addition, while these studies were </span><span style=\"background-color: hsl(120, 100.00%, 97.33%); opacity: 0.80\" title=\"0.008\">under</span><span style=\"opacity: 0.80\"> way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent families with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent with our in vitro and in vivo data, a recent study showed that the a648t mutation abrogated the </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase activity of fgfr2 (25). as discussed earlier, we </span><span style=\"background-color: hsl(120, 100.00%, 96.03%); opacity: 0.81\" title=\"0.015\">propose</span><span style=\"opacity: 0.80\"> that this mutation creates a steric barrier for the movement of the kinase n-lobe toward the c-lobe, which is predicted to take place when the kinase transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 kinase activity awaits the resolution of crystal structure of the mutant kinase domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the catalytic loop of the kinase domain, and the crystal structure of the mutant kinase shows that this mutation </span><span style=\"background-color: hsl(0, 100.00%, 91.04%); opacity: 0.82\" title=\"-0.047\">acts</span><span style=\"opacity: 0.80\"> by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those our lab </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> identified in endometrial cancer highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer with germ-line mutations associated with skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than 20 fgfr2 mutations we identified in melanoma were all novel, supporting our findings that the mutations are distinct and different from the many gain-of-function mutations </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured tumors are presented in red above the schematic representation of the protein and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (</span><span style=\"background-color: hsl(120, 100.00%, 93.60%); opacity: 0.81\" title=\"0.029\">top</span><span style=\"opacity: 0.80\">) above the schematic representation of the protein and are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-line mutations associated with a </span><span style=\"background-color: hsl(0, 100.00%, 95.71%); opacity: 0.81\" title=\"-0.016\">variety</span><span style=\"opacity: 0.80\"> of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this discovery of loss-of-function mutations in a receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase is unexpected, given the conventional wisdom that activation of receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (12). however, </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> should </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> be surprising given that a </span><span style=\"background-color: hsl(0, 100.00%, 95.71%); opacity: 0.81\" title=\"-0.016\">variety</span><span style=\"opacity: 0.80\"> of receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase signaling pathways can induce either proliferation or differentiation depending </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> the cell type. indeed, cre-lox transgenic mice, with fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene and 12-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased number of papillomas and carcinomas compared with wild-type mice, providing support that fgfr2 can act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated with several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (</span><span style=\"background-color: hsl(0, 100.00%, 98.43%); opacity: 0.80\" title=\"-0.004\">34</span><span style=\"opacity: 0.80\">-36), and salivary adenocarcinoma cells (37).  based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, </span><span style=\"background-color: hsl(0, 100.00%, 99.06%); opacity: 0.80\" title=\"-0.002\">it</span><span style=\"opacity: 0.80\"> was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> published reintroduction results and the current melanoma study is that they involve different fgfr2 splice forms. the previous studies were all carried out in cells of epithelial origin with the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.05%); opacity: 0.80\" title=\"-0.002\">expressed</span><span style=\"opacity: 0.80\"> in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> contribute to melanoma pathogenesis by removing a </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> inhibitory signal. indeed, although tumor suppressors classically function to </span><span style=\"background-color: hsl(0, 100.00%, 94.72%); opacity: 0.81\" title=\"-0.022\">regulate</span><span style=\"opacity: 0.80\"> cell cycle, accumulating evidence suggests that additional subsets of genes </span><span style=\"background-color: hsl(0, 100.00%, 84.68%); opacity: 0.85\" title=\"-0.101\">exist</span><span style=\"opacity: 0.80\">, which function to suppress tumorigenesis </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> by inhibiting proliferation but by altering </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might </span><span style=\"background-color: hsl(0, 100.00%, 99.82%); opacity: 0.80\" title=\"-0.000\">not</span><span style=\"opacity: 0.80\"> be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in </span><span style=\"background-color: hsl(120, 100.00%, 97.92%); opacity: 0.80\" title=\"0.006\">normal</span><span style=\"opacity: 0.80\"> melanocyte function and melanoma pathogenesis is currently unknown. our lab is currently investigating the relative expression levels of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.10%); opacity: 0.81\" title=\"-0.014\">tyrosine</span><span style=\"opacity: 0.80\"> kinase signaling. there have been reports that fgfr1 </span><span style=\"background-color: hsl(0, 100.00%, 96.27%); opacity: 0.81\" title=\"-0.013\">binds</span><span style=\"opacity: 0.80\"> cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in various cell types (39).8 lastly, we and others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (</span><span style=\"background-color: hsl(0, 100.00%, 99.49%); opacity: 0.80\" title=\"-0.001\">40</span><span style=\"opacity: 0.80\">)8 and fgfrs have also been shown to bind to </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology will allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we </span><span style=\"background-color: hsl(120, 100.00%, 96.03%); opacity: 0.81\" title=\"0.015\">propose</span><span style=\"opacity: 0.80\"> that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which </span><span style=\"background-color: hsl(120, 100.00%, 98.38%); opacity: 0.80\" title=\"0.004\">both</span><span style=\"opacity: 0.80\"> loss-of-function and gain-of-function mutations have been reported in different tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment </span><span style=\"background-color: hsl(120, 100.00%, 94.85%); opacity: 0.81\" title=\"0.021\">encoding</span><span style=\"opacity: 0.80\"> residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed with wild-type or mutant expression constructs and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-type or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-type and mutant proteins were </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> sequentially purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described </span><span style=\"background-color: hsl(120, 100.00%, 99.31%); opacity: 0.80\" title=\"0.001\">previously</span><span style=\"opacity: 0.80\"> (19).  expression and purification of fgfr2 kinase domain  the dna fragment </span><span style=\"background-color: hsl(120, 100.00%, 94.85%); opacity: 0.81\" title=\"0.021\">encoding</span><span style=\"opacity: 0.80\"> residues 458 to 768 of human fgfr2 was subcloned into pet bacterial expression vectors with an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed with wild-type or mutant fgfr2 kinase expression constructs, and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> lysed using a french press, and the soluble kinase proteins were purified by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> sds-page analysis. purified wild-type and mutant kinase proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-type and mutant kinases was compared using a continuous spectrophotometric kinase assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary </span><span style=\"background-color: hsl(0, 100.00%, 99.76%); opacity: 0.80\" title=\"-0.000\">table</span><span style=\"opacity: 0.80\"> s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation and to ensure that no </span><span style=\"background-color: hsl(120, 100.00%, 97.77%); opacity: 0.80\" title=\"0.006\">other</span><span style=\"opacity: 0.80\"> mutations were introduced during the mutagenesis process. plasmid dna was </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> isolated using </span><span style=\"background-color: hsl(0, 100.00%, 73.76%); opacity: 0.91\" title=\"-0.218\">qiagen</span><span style=\"opacity: 0.80\"> endofree plasmid dna isolation kits, and dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained </span><span style=\"background-color: hsl(120, 100.00%, 97.33%); opacity: 0.80\" title=\"0.008\">under</span><span style=\"opacity: 0.80\"> standard culturing conditions as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-type or mutant fgfr2 + 750 g empty plasmid) were achieved with fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells were washed with ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease </span><span style=\"background-color: hsl(0, 100.00%, 69.79%); opacity: 0.93\" title=\"-0.267\">inhibitor</span><span style=\"opacity: 0.80\"> cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes were probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described and plated onto eight-well chamber slides. after 36 to 48 h, cells were fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated with 1:300 bek c-17 and 1:1,000 antiprotein disulfide isomerase (invitrogen) and </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> detected with goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted with pro long gold antifade reagent and imaged </span><span style=\"background-color: hsl(120, 100.00%, 97.33%); opacity: 0.80\" title=\"0.008\">under</span><span style=\"opacity: 0.80\"> a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> poly-d-lysinecoated plates were transfected with lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (</span><span style=\"background-color: hsl(0, 100.00%, 73.76%); opacity: 0.91\" title=\"-0.218\">qiagen</span><span style=\"opacity: 0.80\">) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and </span><span style=\"background-color: hsl(0, 100.00%, 99.49%); opacity: 0.80\" title=\"-0.001\">40</span><span style=\"opacity: 0.80\"> h, pooled, filtered through a 0.45-um low protein binding filter, and stored at 80c until </span><span style=\"background-color: hsl(0, 100.00%, 74.08%); opacity: 0.91\" title=\"-0.214\">use</span><span style=\"opacity: 0.80\">. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with crystal violet, and colony counting.  baf3 proliferation </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\">  baf3 cells (5  105) were infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> placed </span><span style=\"background-color: hsl(120, 100.00%, 97.33%); opacity: 0.80\" title=\"0.008\">under</span><span style=\"opacity: 0.80\"> selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-type and mutant fgfr2 were maintained </span><span style=\"background-color: hsl(120, 100.00%, 97.33%); opacity: 0.80\" title=\"0.008\">under</span><span style=\"opacity: 0.80\"> selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented with 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media containing </span><span style=\"background-color: hsl(0, 100.00%, 96.03%); opacity: 0.81\" title=\"-0.015\">16</span><span style=\"opacity: 0.80\">.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced with an equal volume containing fresh ligand at 2 concentration and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation </span><span style=\"background-color: hsl(0, 100.00%, 90.70%); opacity: 0.82\" title=\"-0.050\">assays</span><span style=\"opacity: 0.80\">  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in rpmi supplemented with 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced with empty vector, fgfr2, or kinase-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full </span><span style=\"background-color: hsl(0, 100.00%, 96.13%); opacity: 0.81\" title=\"-0.014\">growth</span><span style=\"opacity: 0.80\"> media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> stimulated with 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in either baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved </span><span style=\"background-color: hsl(120, 100.00%, 99.19%); opacity: 0.80\" title=\"0.002\">on</span><span style=\"opacity: 0.80\"> a 4% to 12% bis tris gel and probed for phosho-p44/42 mapk. blots were </span><span style=\"background-color: hsl(120, 100.00%, 99.39%); opacity: 0.80\" title=\"0.001\">then</span><span style=\"opacity: 0.80\"> stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=6\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.474</b>, score <b>1.947</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 84.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +2.048\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.56%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.070\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 98.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.057\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__respect\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.17%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.032\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__very\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.18%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.031\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__significant\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.22%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.029\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__recurrent\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.32%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.024\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__versus\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.54%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.014\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__plasmids\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.59%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__frequently\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__harbor\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.67%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__targets\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__induces\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.70%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__often\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__listed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__erbb2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__61\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__establish\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__approaches\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__age\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__regulatory\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.78%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__anomalous\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__thought\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__parameters\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.012\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lower\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__algorithm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mutagenized\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.46%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__classes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.23%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.029\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__surgery\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.056\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__genetics\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 96.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.239\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"background-color: hsl(120, 100.00%, 60.00%); opacity: 1.00\" title=\"0.070\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes and melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> have revealed that </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> of growth factors and/or receptors by melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> of parenchymal organs including the skin. the fgf family comprises </span><span style=\"background-color: hsl(120, 100.00%, 74.57%); opacity: 0.90\" title=\"0.209\">18</span><span style=\"opacity: 0.80\"> ligands (fgf1-fgf10 and fgf16-fgf23), which signal through </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends on fgf2, which is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf2 resulted in phenotypic changes consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides </span><span style=\"background-color: hsl(0, 100.00%, 97.36%); opacity: 0.80\" title=\"-0.008\">targeted</span><span style=\"opacity: 0.80\"> toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in </span><span style=\"background-color: hsl(120, 100.00%, 92.31%); opacity: 0.82\" title=\"0.038\">decreased</span><span style=\"opacity: 0.80\"> proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor growth and induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, aberrant splicing, or missense mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (</span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">).  we report here the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma tumors. mapping of these mutations onto the known crystal </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\"> of fgfr2 together </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (table 1). </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> mutations were identified in either fgfr1 or fgfr3, whereas </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> mutation at a nonconserved codon was found in fgfr4, p716r.  view </span><span style=\"background-color: hsl(120, 100.00%, 76.99%); opacity: 0.89\" title=\"0.181\">inlineview</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 90.80%); opacity: 0.82\" title=\"0.049\">popup</span><span style=\"opacity: 0.80\"> table 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma tumors and a panel of 72 vertical growth </span><span style=\"background-color: hsl(120, 100.00%, 91.09%); opacity: 0.82\" title=\"0.047\">phase</span><span style=\"opacity: 0.80\"> primary melanomas representing samples from different melanoma subtypes for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples and 5 of 72 primary tumors (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (</span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\">%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding normal tissue, and in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype </span><span style=\"background-color: hsl(120, 100.00%, 90.13%); opacity: 0.83\" title=\"0.054\">seen</span><span style=\"opacity: 0.80\"> in the </span><span style=\"background-color: hsl(120, 100.00%, 91.24%); opacity: 0.82\" title=\"0.046\">breast</span><span style=\"opacity: 0.80\"> cancer kinome screen (13), </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> metastatic tumor was found to carry </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> different fgfr2 mutations (v77m, e574k, s688f, and p708s), which were not further analyzed because there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary tumors carried nonsense mutations (table 1). the majority (</span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\"> of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras and braf were observed in tumors </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\">  extracellular mutations. to gain insights into how the different fgfr2 mutations might affect receptor function, we analyzed them in the light of </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which affect residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; </span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> each other directly and each receptor makes contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> both ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular mutations identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\"> suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in addition to the mutated residue, other relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgf ligand. mutation of r251 to q should substantially reduce the </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> other residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, on the other hand, maps to the large loop region between strand c and c, which protrudes from d3 and </span><span style=\"background-color: hsl(0, 100.00%, 94.17%); opacity: 0.81\" title=\"-0.025\">does</span><span style=\"opacity: 0.80\"> not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin/heparan sulfate. e. the important role of d218 is highlighted in the </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> its primary fgf2, suggesting that the r251q mutation should diminish the </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> affinity of fgfr2 toward fgf2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). by contrast, the e219k mutation maps to the secondary ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k35 of the secondary fgf2 ligand (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1e). therefore, based on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other extracellular mutations including v248d, g227e, and g271e are </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 where its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1a). substitution of v248 </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based on the structure, these substitutions should cause local structural perturbations, which could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> also interfere </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to affect the interaction of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> in d2, on which a key lysine residue involved in heparan sulfate </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> resides (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1d). therefore, destabilization of the ga </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> due to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 1d), and the </span><span style=\"background-color: hsl(120, 100.00%, 69.23%); opacity: 0.94\" title=\"0.274\">bulkier</span><span style=\"opacity: 0.80\"> tyrosine side chain at this location could cause steric clashes </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  kinase domain mutations. more than half of the melanoma mutations affect the conserved tyrosine kinase domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 kinase domain </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\"> (pdb id: 2psq and 2pvf; ref. 15) suggests that they also </span><span style=\"background-color: hsl(120, 100.00%, 87.15%); opacity: 0.84\" title=\"0.079\">represent</span><span style=\"opacity: 0.80\"> loss-of-function mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2 ). notably, m640 and i642 localize onto either end of the 8 strand (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2), a secondary structure element that precedes the activation loop (a-loop) of the kinase domain. the hydrophobic side chains of these residues are in contact </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the e </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> in the innermost core of the c-lobe of the kinase domain. substitutions of either residues </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the smaller hydrophobic residues isoleucine or valine, respectively, </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> cause a cavity within the core of the kinase that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation maps to the activation loop of the kinase domain (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2e) and is also </span><span style=\"background-color: hsl(0, 100.00%, 82.89%); opacity: 0.86\" title=\"-0.118\">expected</span><span style=\"opacity: 0.80\"> to lead to loss of function. this is because in the phosphorylated activated kinase domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> m537 in the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\">, which contributes to the correct positioning/orientation of the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> relative to the cooh-terminal lobe of the kinase and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> to sterically hinder the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c kinase domain mutations identified in melanoma tumors/cell lines mapped onto the crystal </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\"> of the fgfr2 kinase domain </span><span style=\"background-color: hsl(120, 100.00%, 89.88%); opacity: 0.83\" title=\"0.056\">suggest</span><span style=\"opacity: 0.80\"> that these mutations should impair the kinase activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 kinase domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular tyrosine kinase domain is as follows: the nh2-terminal lobe of kinase is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the kinase hinge region is in green; and the nh2-terminal </span><span style=\"background-color: hsl(120, 100.00%, 90.40%); opacity: 0.83\" title=\"0.052\">tail</span><span style=\"opacity: 0.80\"> of the kinase is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the kinase domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the </span><span style=\"background-color: hsl(120, 100.00%, 92.63%); opacity: 0.82\" title=\"0.036\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s563 in the kinase hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine kinase activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the kinase domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a reduction in kinase activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> h) and d756 (in </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location </span><span style=\"background-color: hsl(120, 100.00%, 95.81%); opacity: 0.81\" title=\"0.016\">will</span><span style=\"opacity: 0.80\"> also truncate the </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> i by </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> helical turn. these structural changes should reduce the stability of the kinase domain, specifically making the kinase domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\">. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 </span><span style=\"background-color: hsl(0, 100.00%, 94.17%); opacity: 0.81\" title=\"-0.025\">does</span><span style=\"opacity: 0.80\"> not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\">. replacement of a648 </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> threonine is </span><span style=\"background-color: hsl(0, 100.00%, 82.89%); opacity: 0.86\" title=\"-0.118\">expected</span><span style=\"opacity: 0.80\"> to lead to loss of function because the larger side chain of threonine </span><span style=\"background-color: hsl(120, 100.00%, 95.81%); opacity: 0.81\" title=\"0.016\">will</span><span style=\"opacity: 0.80\"> impede c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the kinase region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation maps to the loop region between the 7 and 8 strands (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> s568, which is located in the hinge region of the kinase domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the kinase relative to each other. based on the structure, the e636k mutation should influence the kinase activity of the fgfr2 kinase domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved kinase domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> t555 whose methyl group is in hydrophobic contact </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> residues in the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2d). hence, these interactions hold the c </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated kinase domains, however, these hydrogen bonds are weaker and, concomitantly, the c-</span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\"> moves </span><span style=\"background-color: hsl(120, 100.00%, 89.39%); opacity: 0.83\" title=\"0.060\">closer</span><span style=\"opacity: 0.80\"> toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this inhibition and potentially lead to activation of the kinase domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the kinase and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2f) and facilitates the conformation of this loop region, which makes several hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced kinase activity. the r759q mutation maps to the cooh-terminal end of the i </span><span style=\"background-color: hsl(120, 100.00%, 99.32%); opacity: 0.80\" title=\"0.001\">helix</span><span style=\"opacity: 0.80\">, the last secondary structure element of the kinase domain. in the structure, r759 makes hydrogen bonds </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> two other residues in its vicinity (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 2c), and loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in kinase activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> phosphopeptide indicates that the sh2 domain of plc interacts </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> pocket of the kinase domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal </span><span style=\"background-color: hsl(120, 100.00%, 96.76%); opacity: 0.81\" title=\"0.011\">structures</span><span style=\"opacity: 0.80\">. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations and a third sample carried a splicing mutation that </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> result in premature termination of fgfr2 protein translation (table 1).  view </span><span style=\"background-color: hsl(120, 100.00%, 76.99%); opacity: 0.89\" title=\"0.181\">inlineview</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 90.80%); opacity: 0.82\" title=\"0.049\">popup</span><span style=\"opacity: 0.80\"> table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions for the extracellular mutations on fgfr2 function, we expressed recombinant wild-type and mutant ectodomains in e. coli, and then isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have previously shown that the s252w mutation has </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> adverse </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> yields similar to those of the wild-type and s252w mutant receptors. we then used surface plasmon resonance to analyze the </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> of the r251q mutation on ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> affinity of fgfr2c. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the other hand, as </span><span style=\"background-color: hsl(0, 100.00%, 82.89%); opacity: 0.86\" title=\"-0.118\">expected</span><span style=\"opacity: 0.80\">, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\">. a and b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of </span><span style=\"background-color: hsl(0, 100.00%, 94.87%); opacity: 0.81\" title=\"-0.021\">time</span><span style=\"opacity: 0.80\">. surface plasmon resonance analysis was done as previously described (19). to test the </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> of kinase domain mutations, we prepared recombinant wild-type and mutant kinase domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro kinase assays. consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> structural predictions, the d530n, i642v, and a648t mutant kinase domains exhibited reduced kinase activity relative to the wild-type kinase domain (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 4 ). interestingly, the e475k mutant exhibited a slight increase in kinase activity relative to the wild-type kinase domain. notably, the protein </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> yield for the e475k mutant kinase was severalfold less than that of wild-type kinase, suggesting that this mutation was partially destabilizing the kinase domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 kinase domains harboring melanoma mutations have reduced tyrosine autophosphorylation activity. wild-type fgfr2k and </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro kinase assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-type and mutant fgfr2ks were determined </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">  we next analyzed the </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crouzon syndrome was used to assess the </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> of destabilizing d3 on both receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, and increased signaling from intracellular compartments (</span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\">). in contrast, the s252w mutation observed in patients </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> apert syndrome has </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on receptor folding (19). the k517r mutant receptor served as a kinase-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously shown to abolish kinase activity (21).  following transfection of wild-type fgfr2 receptor into hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the reduction in size of both bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions of </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> adverse </span><span style=\"background-color: hsl(120, 100.00%, 92.63%); opacity: 0.82\" title=\"0.036\">effects</span><span style=\"opacity: 0.80\"> on protein stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural predictions that these mutations </span><span style=\"background-color: hsl(120, 100.00%, 95.81%); opacity: 0.81\" title=\"0.016\">will</span><span style=\"opacity: 0.80\"> adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, bottom), consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 92.31%); opacity: 0.82\" title=\"0.038\">decreased</span><span style=\"opacity: 0.80\"> protein yield observed in the in vitro kinase assays.  figure 5. download figureopen in new tabdownload powerpoint figure 5. mutations in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">. sensitivity to digestion </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following transient transfection into hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (yellow, right) when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the cell surface localization of wild-type fgfr2 and e219k (mapping onto the known extracellular crystal structure </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> the latter mutation to have </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">. in </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5c). in </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the g227e mutant (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also </span><span style=\"background-color: hsl(0, 100.00%, 89.24%); opacity: 0.83\" title=\"-0.061\">showed</span><span style=\"opacity: 0.80\"> mislocalization to intracellular membranes and </span><span style=\"background-color: hsl(120, 100.00%, 92.31%); opacity: 0.82\" title=\"0.038\">decreased</span><span style=\"opacity: 0.80\"> cell surface </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we then performed similar immunofluorescence studies </span><span style=\"background-color: hsl(120, 100.00%, 81.64%); opacity: 0.87\" title=\"0.131\">looking</span><span style=\"opacity: 0.80\"> for colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 5d). in contrast, </span><span style=\"background-color: hsl(120, 100.00%, 75.98%); opacity: 0.90\" title=\"0.192\">marked</span><span style=\"opacity: 0.80\"> cell surface localization and limited colocalization </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> protein disulfide isomerase were observed for both the wild-type fgfr2 and the e219k mutation, </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> to have </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> detrimental </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated protein kinase phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the </span><span style=\"background-color: hsl(120, 100.00%, 92.63%); opacity: 0.82\" title=\"0.036\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express other fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it </span><span style=\"background-color: hsl(0, 100.00%, 94.17%); opacity: 0.81\" title=\"-0.025\">does</span><span style=\"opacity: 0.80\"> not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine kinase signaling </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (</span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">) and an identical mutation at the paralogous positions in fgfr3 has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were then done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type fgfr2 (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type and mutant fgfr2_ires_neo </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on stimulation </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> exogenous rfgf2 shows that, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type receptor. c. western blot showing equal </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> of fgfr2 by all mutants except empty </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.010\">vector</span><span style=\"opacity: 0.80\"> in the absence () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-type receptor. as a further assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein kinase (mapk) signaling pathway in response to exogenous rfgf2. </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the exception of the n549k known activating mutant, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 6b-d).  reintroduction of fgfr2 into melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">  we next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">. reverse transcription-pcr </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> isoform-specific primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 7a ). </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell line (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\">. a. melanoma cell lines sbcl2 and ao4 were stably transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.010\">vector</span><span style=\"opacity: 0.80\">, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> the sulforhodamine b assay. overexpression of fgfr2 or kinase-dead fgfr2 k517r had </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.78%); opacity: 0.80\" title=\"0.000\">effect</span><span style=\"opacity: 0.80\"> on the proliferation of these melanoma cell lines. b. fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> was confirmed by western blot analysis. discussion  in this </span><span style=\"background-color: hsl(0, 100.00%, 96.87%); opacity: 0.81\" title=\"-0.010\">study</span><span style=\"opacity: 0.80\">, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary tumors. to determine the ratio of nonsynonymous to synonymous mutations in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> data set, we went back and examined the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> data set was 23:4, which exceeds the 2:1 ratio </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, and we anticipate that further sequencing analysis in a greater number of primary tumors and nevi </span><span style=\"background-color: hsl(120, 100.00%, 95.81%); opacity: 0.81\" title=\"0.016\">will</span><span style=\"opacity: 0.80\"> shed light on when during melanoma </span><span style=\"background-color: hsl(0, 100.00%, 96.36%); opacity: 0.81\" title=\"-0.013\">development</span><span style=\"opacity: 0.80\"> these mutations arise. it should be noted that the mutation frequency reported in this </span><span style=\"background-color: hsl(0, 100.00%, 96.87%); opacity: 0.81\" title=\"-0.010\">study</span><span style=\"opacity: 0.80\"> may be an underestimate. this is based on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not all primary tumors provided high-</span><span style=\"background-color: hsl(120, 100.00%, 88.52%); opacity: 0.83\" title=\"0.067\">quality</span><span style=\"opacity: 0.80\"> sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine kinase activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> to plc.  the loss-of-function nature of </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> identified melanoma mutations is also consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent families </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> in vitro and in vivo data, a recent </span><span style=\"background-color: hsl(0, 100.00%, 96.87%); opacity: 0.81\" title=\"-0.010\">study</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 89.24%); opacity: 0.83\" title=\"-0.061\">showed</span><span style=\"opacity: 0.80\"> that the a648t mutation abrogated the tyrosine kinase activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the kinase n-lobe toward the c-lobe, which is </span><span style=\"background-color: hsl(120, 100.00%, 96.38%); opacity: 0.81\" title=\"0.013\">predicted</span><span style=\"opacity: 0.80\"> to take place when the kinase transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 kinase activity awaits the resolution of crystal structure of the mutant kinase domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the catalytic loop of the kinase domain, and the crystal structure of the mutant kinase shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> lab previously identified in endometrial cancer </span><span style=\"background-color: hsl(120, 100.00%, 82.73%); opacity: 0.86\" title=\"0.120\">highlights</span><span style=\"opacity: 0.80\"> the differences in the mutation spectra (</span><span style=\"background-color: hsl(0, 100.00%, 99.74%); opacity: 0.80\" title=\"-0.000\">fig</span><span style=\"opacity: 0.80\">. 8 ). the majority of the endometrial mutations (17 of </span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\">) are identical to fgfr2 and fgfr3 mutations previously reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> germ-line mutations associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than </span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\"> fgfr2 mutations we identified in melanoma were all novel, supporting </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> findings that the mutations are distinct and different from the many gain-of-function mutations previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured tumors are presented in red above the schematic representation of the protein and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (top) above the schematic representation of the protein and are numbered relative to fgfr2b (np_075259.2; ref. </span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">). bottom, germ-line mutations associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. </span><span style=\"background-color: hsl(120, 100.00%, 96.84%); opacity: 0.81\" title=\"0.011\">four</span><span style=\"opacity: 0.80\"> somatic fgfr2 endometrial mutations, although not previously reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (</span><span style=\"background-color: hsl(120, 100.00%, 97.35%); opacity: 0.80\" title=\"0.008\">indicated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this discovery of loss-of-function mutations in a receptor tyrosine kinase is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">kit</span><span style=\"opacity: 0.80\">, flt3, and fgfr3) drives tumorigenesis and </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (</span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">). however, it should not be surprising given that a variety of receptor tyrosine kinase signaling pathways </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> induce either proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic mice, </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr2b having been deleted in </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> expressing keratin v, develop spontaneous papillomas. moreover, following 7,</span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">-dimethylbenz(a)anthracene and </span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased number of papillomas and carcinomas compared </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type mice, providing support that fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> has been associated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in </span><span style=\"background-color: hsl(120, 100.00%, 92.31%); opacity: 0.82\" title=\"0.038\">decreased</span><span style=\"opacity: 0.80\"> growth in vitro and reduced tumorigenicity in vivo of bladder carcinoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (33), prostate tumor </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (34-36), and salivary adenocarcinoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (37).  based on these data and </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several </span><span style=\"background-color: hsl(120, 100.00%, 97.68%); opacity: 0.80\" title=\"0.007\">possible</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 91.98%); opacity: 0.82\" title=\"0.040\">explanations</span><span style=\"opacity: 0.80\"> for these apparent disparate results. </span><span style=\"background-color: hsl(120, 100.00%, 99.25%); opacity: 0.80\" title=\"0.001\">one</span><span style=\"opacity: 0.80\"> difference between the previously published reintroduction results and the current melanoma </span><span style=\"background-color: hsl(0, 100.00%, 96.87%); opacity: 0.81\" title=\"-0.010\">study</span><span style=\"opacity: 0.80\"> is that they involve different fgfr2 splice forms. the previous studies were all carried out in </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> of epithelial origin </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> the fgfr2 iiib isoform, whereas </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. </span><span style=\"background-color: hsl(0, 100.00%, 98.94%); opacity: 0.80\" title=\"-0.002\">another</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.68%); opacity: 0.80\" title=\"0.007\">possible</span><span style=\"opacity: 0.80\"> explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which function to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead affect </span><span style=\"background-color: hsl(0, 100.00%, 98.94%); opacity: 0.80\" title=\"-0.002\">another</span><span style=\"opacity: 0.80\"> aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples </span><span style=\"background-color: hsl(120, 100.00%, 99.09%); opacity: 0.80\" title=\"0.002\">would</span><span style=\"opacity: 0.80\"> argue that they must provide a selective advantage to the </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> in which they arose. this is consistent </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently unknown. </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> lab is currently investigating the relative </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> levels of these </span><span style=\"background-color: hsl(120, 100.00%, 93.65%); opacity: 0.81\" title=\"0.029\">various</span><span style=\"opacity: 0.80\"> isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine kinase signaling. there have been reports that fgfr1 binds cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in </span><span style=\"background-color: hsl(120, 100.00%, 93.65%); opacity: 0.81\" title=\"0.029\">various</span><span style=\"opacity: 0.80\"> cell types (39).8 lastly, we and others have shown that fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> heterodimerize </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fgfr1 at least in vitro (40)8 and fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology </span><span style=\"background-color: hsl(120, 100.00%, 95.81%); opacity: 0.81\" title=\"0.016\">will</span><span style=\"opacity: 0.80\"> allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which both loss-of-function and gain-of-function mutations have been reported in different tissue types. ongoing work in </span><span style=\"background-color: hsl(120, 100.00%, 99.68%); opacity: 0.80\" title=\"0.000\">our</span><span style=\"opacity: 0.80\"> laboratory is </span><span style=\"background-color: hsl(0, 100.00%, 97.36%); opacity: 0.80\" title=\"-0.008\">targeted</span><span style=\"opacity: 0.80\"> toward understanding the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\">, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.010\">vector</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> quikchange site-directed mutagenesis </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">kit</span><span style=\"opacity: 0.80\"> (stratagene). bl21(de3) competent </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type or mutant </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> constructs and protein </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> was induced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were lysed </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> french press and centrifuged; the inclusion bodies containing wild-type or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-type and mutant proteins were then </span><span style=\"background-color: hsl(120, 100.00%, 60.00%); opacity: 1.00\" title=\"0.399\">sequentially</span><span style=\"opacity: 0.80\"> purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> and purification of fgfr2 kinase domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> vectors </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, and a648t) were introduced </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> quikchange site-directed mutagenesis </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">kit</span><span style=\"opacity: 0.80\"> (stratagene). the bacterial strain bl21(de3) </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were transformed </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> wild-type or mutant fgfr2 kinase </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> constructs, and protein </span><span style=\"background-color: hsl(0, 100.00%, 99.35%); opacity: 0.80\" title=\"-0.001\">expression</span><span style=\"opacity: 0.80\"> was induced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were then lysed </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> a french press, and the soluble kinase proteins were purified by </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. purified wild-type and mutant kinase proteins were concentrated to at least 10 mg/ml </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> a centricon-10. the autophosphorylation activity of the wild-type and mutant kinases was compared </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> a continuous spectrophotometric kinase assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> the quikchange ii xl site-directed mutagenesis </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">kit</span><span style=\"opacity: 0.80\"> (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences </span><span style=\"background-color: hsl(120, 100.00%, 98.92%); opacity: 0.80\" title=\"0.002\">can</span><span style=\"opacity: 0.80\"> be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation and to ensure that </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> other mutations were introduced during the mutagenesis process. plasmid dna was then isolated </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> qiagen endofree plasmid dna isolation kits, and dna </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained under standard culturing conditions as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\"> total dna (consisting of 250 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\"> wild-type or mutant fgfr2 + 750 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\"> empty plasmid) were achieved </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were washed </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> ice-cold pbs and lysed in ice-cold lysis buffer [</span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\"> mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, </span><span style=\"background-color: hsl(120, 100.00%, 99.65%); opacity: 0.80\" title=\"0.000\">20</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\"> of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5% skim milk/tbst, membranes were probed </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> as described and plated onto eight-well chamber slides. after 36 to 48 h, </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were fixed </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1:300 bek c-17 and 1:1,000 antiprotein disulfide isomerase (invitrogen) and then detected </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> on poly-d-lysinecoated plates were </span><span style=\"background-color: hsl(0, 100.00%, 98.24%); opacity: 0.80\" title=\"-0.005\">transfected</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and 40 h, pooled, filtered through a 0.45-um low protein </span><span style=\"background-color: hsl(120, 100.00%, 98.90%); opacity: 0.80\" title=\"0.002\">binding</span><span style=\"opacity: 0.80\"> filter, and stored at </span><span style=\"background-color: hsl(0, 100.00%, 88.34%); opacity: 0.83\" title=\"-0.068\">80</span><span style=\"opacity: 0.80\">c until use. viral stocks were titered in hek293 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> following serial dilution in a six-well plate, selection for 14 d in 800 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml geneticin, staining </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> crystal violet, and colony counting.  baf3 proliferation assays  baf3 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (5  105) were infected </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> containing wild-type and mutant fgfr2 were maintained under selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml streptomycin sulfate, 1,000 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml geneticin, and supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were counted and washed in the above media containing </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> il-3. </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were plated at 1  104 per well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml heparin. half the volume of media was removed at day 3 and replaced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> an equal volume containing fresh ligand at 2 concentration and </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> il-3. proliferation was assessed </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> the vialight plus cell proliferation/cytotoxity </span><span style=\"background-color: hsl(120, 100.00%, 97.55%); opacity: 0.80\" title=\"0.007\">kit</span><span style=\"opacity: 0.80\"> (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 2 mmol/l l-glutamine supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were grown in rpmi supplemented </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were transduced </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> empty </span><span style=\"background-color: hsl(0, 100.00%, 96.92%); opacity: 0.81\" title=\"-0.010\">vector</span><span style=\"opacity: 0.80\">, fgfr2, or kinase-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml polybrene. </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were selected for at least 14 d in 500 to 800 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml g418. </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> (2,000-4,000) were plated per well in a 96-well plate in full growth media. the next day, </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were then stimulated </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 ng/ml fgf2 and 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml heparin in low-serum media and proliferation was assessed </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were washed in pbs and resuspended in baf media containing </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> fetal bovine serum and </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">no</span><span style=\"opacity: 0.80\"> il-3 (baf starve media) for 2 h. </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were pelleted and resuspended in either baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> 10 </span><span style=\"background-color: hsl(120, 100.00%, 99.29%); opacity: 0.80\" title=\"0.001\">g</span><span style=\"opacity: 0.80\">/ml heparin only (unstimulated or ) or baf starve media </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">with</span><span style=\"opacity: 0.80\"> heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the </span><span style=\"background-color: hsl(0, 100.00%, 97.54%); opacity: 0.80\" title=\"-0.007\">cells</span><span style=\"opacity: 0.80\"> were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved on a 4% to </span><span style=\"background-color: hsl(0, 100.00%, 96.37%); opacity: 0.81\" title=\"-0.013\">12</span><span style=\"opacity: 0.80\">% bis tris gel and probed for phosho-p44/42 mapk. blots were then stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated </span><span style=\"background-color: hsl(120, 100.00%, 97.74%); opacity: 0.80\" title=\"0.007\">using</span><span style=\"opacity: 0.80\"> image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=7\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.012</b>, score <b>-1.727</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 90.17%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +1.093\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.10%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.036\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__association\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.35%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.023\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__disruption\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.51%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mutational\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__possibility\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__141\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mb\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.69%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.008\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__taken\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__global\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.76%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__inter\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__seven\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ovarian\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(120, 100.00%, 99.81%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        +0.004\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__repair\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__49\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.72%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__systemic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.007\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__minutes\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__since\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__existing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.65%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.009\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__106\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__puromycin\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__adult\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__separated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.62%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.010\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__structurally\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.61%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__correlated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__erk\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__therapeutics\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.60%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.011\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__verify\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.57%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.013\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__triphosphate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.52%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__reactions\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.45%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.018\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__egfp\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.41%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__detectable\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.020\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__maximal\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.38%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.021\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__diagnosis\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.15%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.033\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__unique\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.13%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.034\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__digested\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.07%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.038\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__often\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.06%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.038\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__nm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.99%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.042\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__investigation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.047\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__4b\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.71%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.060\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__foci\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.64%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.065\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__elucidate\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.10%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.105\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__like\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 83.66%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -2.258\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there would be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in 2008. previous </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> factor </span><span style=\"background-color: hsl(0, 100.00%, 97.82%); opacity: 0.80\" title=\"-0.006\">stimulation</span><span style=\"opacity: 0.80\"> by the surrounding keratinocytes in the skin to autocrine </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> factor </span><span style=\"background-color: hsl(0, 100.00%, 97.82%); opacity: 0.80\" title=\"-0.006\">stimulation</span><span style=\"opacity: 0.80\"> via ectopic expression of </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> factors and/or receptors by melanoma cells (1, 2). fibroblast </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (fgfr1-fgfr4) and their tissue-</span><span style=\"background-color: hsl(120, 100.00%, 97.93%); opacity: 0.80\" title=\"0.006\">specific</span><span style=\"opacity: 0.80\"> alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> have shown that survival and proliferation of melanocytes depends on fgf2, which is </span><span style=\"background-color: hsl(120, 100.00%, 98.19%); opacity: 0.80\" title=\"0.005\">provided</span><span style=\"opacity: 0.80\"> by the surrounding keratinocytes in the skin. a number of functional </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude </span><span style=\"background-color: hsl(0, 100.00%, 99.19%); opacity: 0.80\" title=\"-0.002\">mice</span><span style=\"opacity: 0.80\"> has also been shown to result in inhibition of tumor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> and induction of apoptosis (10, 11).  constitutive </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> of fgfrs through chromosomal </span><span style=\"background-color: hsl(0, 100.00%, 91.08%); opacity: 0.82\" title=\"-0.047\">translocation</span><span style=\"opacity: 0.80\">, aberrant splicing, or missense mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have </span><span style=\"background-color: hsl(0, 100.00%, 98.44%); opacity: 0.80\" title=\"-0.004\">recently</span><span style=\"opacity: 0.80\"> reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (12).  we report here the identification of novel fgfr2 mutations in 15 of 113 (</span><span style=\"background-color: hsl(0, 100.00%, 97.30%); opacity: 0.80\" title=\"-0.008\">13</span><span style=\"opacity: 0.80\">%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">. mapping of these mutations onto the known crystal structures of fgfr2 together with in vitro and in vivo functional analyses </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> that these mutations result in receptor </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> fgfr2 mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (</span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1). no mutations were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup </span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> and a panel of 72 vertical </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> phase primary melanomas representing samples from </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> melanoma subtypes for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples and 5 of 72 primary </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> (</span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">, we were able to extract dna from surrounding normal tissue, and in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (</span><span style=\"background-color: hsl(0, 100.00%, 97.30%); opacity: 0.80\" title=\"-0.008\">13</span><span style=\"opacity: 0.80\">), one metastatic tumor was found to carry four </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> fgfr2 mutations (v77m, e574k, s688f, and p708s), which were not further analyzed </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> carried </span><span style=\"background-color: hsl(0, 100.00%, 78.26%); opacity: 0.88\" title=\"-0.167\">nonsense</span><span style=\"opacity: 0.80\"> mutations (</span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras and braf were observed in </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations </span><span style=\"background-color: hsl(0, 100.00%, 96.72%); opacity: 0.81\" title=\"-0.011\">within</span><span style=\"opacity: 0.80\"> known fgfr2 structures  </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> mutations. to </span><span style=\"background-color: hsl(0, 100.00%, 93.28%); opacity: 0.82\" title=\"-0.031\">gain</span><span style=\"opacity: 0.80\"> insights into how the </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> fgfr2 mutations might affect receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">, we analyzed them in the light of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which affect residues in the </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; </span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.003\">evaluate</span><span style=\"opacity: 0.80\"> the effect of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with each other directly and each receptor makes contacts with both ligands as well.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 1. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 1. mapping of fgfr2c </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> ligand binding region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in </span><span style=\"background-color: hsl(0, 100.00%, 97.61%); opacity: 0.80\" title=\"-0.007\">addition</span><span style=\"opacity: 0.80\"> to the mutated residue, other relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with other residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by </span><span style=\"background-color: hsl(0, 100.00%, 89.99%); opacity: 0.83\" title=\"-0.055\">making</span><span style=\"opacity: 0.80\"> hydrogen bonds with other residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, on the other hand, maps to the </span><span style=\"background-color: hsl(0, 100.00%, 96.59%); opacity: 0.81\" title=\"-0.012\">large</span><span style=\"opacity: 0.80\"> loop region between strand c and c, which protrudes from d3 and does not play </span><span style=\"background-color: hsl(0, 100.00%, 96.99%); opacity: 0.80\" title=\"-0.010\">any</span><span style=\"opacity: 0.80\"> apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts with the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its primary fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1b). by contrast, the e219k mutation maps to the secondary ligand binding site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 ligand (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1e). therefore, based on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> end of the g strand in d2 where its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(0, 100.00%, 92.52%); opacity: 0.82\" title=\"-0.036\">1a</span><span style=\"opacity: 0.80\">). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. </span><span style=\"background-color: hsl(0, 100.00%, 92.52%); opacity: 0.82\" title=\"-0.036\">1a</span><span style=\"opacity: 0.80\">). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based on the structure, these </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">substitutions</span><span style=\"opacity: 0.80\"> should </span><span style=\"background-color: hsl(0, 100.00%, 94.04%); opacity: 0.81\" title=\"-0.026\">cause</span><span style=\"opacity: 0.80\"> local structural perturbations, which could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct </span><span style=\"background-color: hsl(0, 100.00%, 96.50%); opacity: 0.81\" title=\"-0.012\">processing</span><span style=\"opacity: 0.80\">/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to affect the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on which a </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">key</span><span style=\"opacity: 0.80\"> lysine residue involved in heparan sulfate binding resides (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 1d), and the bulkier </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> side chain at this location could </span><span style=\"background-color: hsl(0, 100.00%, 94.04%); opacity: 0.81\" title=\"-0.026\">cause</span><span style=\"opacity: 0.80\"> steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain mutations. more than half of the melanoma mutations affect the conserved </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2 ). notably, m640 and i642 localize onto either end of the 8 strand (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2), a secondary structure element that precedes the </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> loop (a-loop) of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">substitutions</span><span style=\"opacity: 0.80\"> of either residues with the smaller hydrophobic residues isoleucine or valine, respectively, would </span><span style=\"background-color: hsl(0, 100.00%, 94.04%); opacity: 0.81\" title=\"-0.026\">cause</span><span style=\"opacity: 0.80\"> a cavity </span><span style=\"background-color: hsl(0, 100.00%, 96.72%); opacity: 0.81\" title=\"-0.011\">within</span><span style=\"opacity: 0.80\"> the core of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> that should induce local structural perturbations that ultimately affect the conformation of the </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> loop. the a648t mutation maps to the </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> loop of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2e) and is also </span><span style=\"background-color: hsl(0, 100.00%, 97.52%); opacity: 0.80\" title=\"-0.007\">expected</span><span style=\"opacity: 0.80\"> to lead to </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">. this is </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> in the phosphorylated </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> and, hence, to the productive alignment of the </span><span style=\"background-color: hsl(120, 100.00%, 97.97%); opacity: 0.80\" title=\"0.006\">catalytic</span><span style=\"opacity: 0.80\"> residues for the phosphotransfer </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.007\">reaction</span><span style=\"opacity: 0.80\"> (15). a threonine is unfavorable at this location </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> lobe, and consequently, the </span><span style=\"background-color: hsl(120, 100.00%, 97.97%); opacity: 0.80\" title=\"0.006\">catalytic</span><span style=\"opacity: 0.80\"> residues would not be aligned for the phosphotransfer </span><span style=\"background-color: hsl(0, 100.00%, 97.76%); opacity: 0.80\" title=\"-0.007\">reaction</span><span style=\"opacity: 0.80\">.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 2. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 2. fgfr2c </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain mutations identified in melanoma </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">/cell lines mapped onto the crystal structures of the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain suggest that these mutations should impair the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain is as follows: the nh2-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> lobe of </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> is in light blue; the cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> lobe is in bright blue; the </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> loop is in magenta; the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> hinge region is in green; and the nh2-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> tail of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the </span><span style=\"background-color: hsl(120, 100.00%, 98.12%); opacity: 0.80\" title=\"0.005\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond with s563 in the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a </span><span style=\"background-color: hsl(0, 100.00%, 97.53%); opacity: 0.80\" title=\"-0.007\">reduction</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are shown. in </span><span style=\"background-color: hsl(0, 100.00%, 97.61%); opacity: 0.80\" title=\"-0.007\">addition</span><span style=\"opacity: 0.80\"> to the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain, specifically </span><span style=\"background-color: hsl(0, 100.00%, 89.99%); opacity: 0.83\" title=\"-0.055\">making</span><span style=\"opacity: 0.80\"> the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain temperature sensitive and ultimately leading to receptor </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does not play </span><span style=\"background-color: hsl(0, 100.00%, 96.99%); opacity: 0.80\" title=\"-0.010\">any</span><span style=\"opacity: 0.80\"> role. however, in the a-loop phosphorylated </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is </span><span style=\"background-color: hsl(0, 100.00%, 97.52%); opacity: 0.80\" title=\"-0.007\">expected</span><span style=\"opacity: 0.80\"> to lead to </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation maps to the loop region between the 7 and 8 strands (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge region of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain, suggesting that </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> may play a role in regulation of the spatial positioning of the nh2-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> and cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> lobes of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> relative to each other. based on the structure, the e636k mutation should influence the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity of the fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. the e475 is part of the we consensus motif that defines approximately the nh2-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> boundary of the conserved </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain in many receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves </span><span style=\"background-color: hsl(0, 100.00%, 81.12%); opacity: 0.87\" title=\"-0.136\">closer</span><span style=\"opacity: 0.80\"> toward the c-lobe. </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of these hydrogen bonds due to the e475k mutation could relieve this inhibition and potentially lead to </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2f) and facilitates the conformation of this loop region, which makes several hydrogen bonds with k668 in the </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity. the r759q mutation maps to the cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> end of the i helix, the last secondary structure element of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. in the structure, r759 makes hydrogen bonds with two other residues in its vicinity (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 2c), and </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> in </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> by the </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex with phosphopeptide indicates that the sh2 domain of plc interacts with ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> fgfr2. the l770v mutation could also reduce y769 phosphorylation </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> the substrate binding pocket of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain prefers a leucine over a valine next to the </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> phosphorylation site (16). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. </span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 2 summarizes the </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> mechanisms of </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> by the melanoma mutations inferred from crystal structures. consistent with these analyses predicting that the fgfr2 mutations are </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations, two melanoma samples harbored </span><span style=\"background-color: hsl(0, 100.00%, 78.26%); opacity: 0.88\" title=\"-0.167\">nonsense</span><span style=\"opacity: 0.80\"> mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> translation (</span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 1).  view inlineview popup </span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to </span><span style=\"background-color: hsl(120, 100.00%, 96.77%); opacity: 0.81\" title=\"0.011\">test</span><span style=\"opacity: 0.80\"> our structural predictions for the </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> mutations on fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">, we expressed recombinant wild-type and mutant ectodomains in e. </span><span style=\"background-color: hsl(120, 100.00%, 98.50%); opacity: 0.80\" title=\"0.004\">coli</span><span style=\"opacity: 0.80\">, and then isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have previously shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro with yields similar to those of the wild-type and s252w mutant receptors. we then used surface plasmon resonance to analyze the effect of the r251q mutation on ligand binding affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the other hand, as </span><span style=\"background-color: hsl(0, 100.00%, 97.52%); opacity: 0.80\" title=\"-0.007\">expected</span><span style=\"opacity: 0.80\">, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not shown).  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 3. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 3. surface plasmon resonance analysis </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> of time. surface plasmon resonance analysis was done as previously described (19). to </span><span style=\"background-color: hsl(120, 100.00%, 96.77%); opacity: 0.81\" title=\"0.011\">test</span><span style=\"opacity: 0.80\"> the effect of </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain mutations, we prepared recombinant wild-type and mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> assays. consistent with structural predictions, the d530n, i642v, and a648t mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domains exhibited reduced </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-type </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 4 ). interestingly, the e475k mutant exhibited a slight increase in </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity relative to the wild-type </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain. notably, the </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> expression yield for the e475k mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> was severalfold less than that of wild-type </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\">, suggesting that this mutation was partially destabilizing the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 4. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 4. fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domains harboring melanoma mutations have reduced </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> autophosphorylation activity. wild-type fgfr2k and four mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> assays. columns, average of three independent experiments; </span><span style=\"background-color: hsl(0, 100.00%, 97.14%); opacity: 0.80\" title=\"-0.009\">bars</span><span style=\"opacity: 0.80\">, sd. the </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> autophosphorylation activities of these wild-type and mutant fgfr2ks were determined using a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured cells  we next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation and </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\">. this mutation has been shown to result in impaired receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.50%); opacity: 0.81\" title=\"-0.012\">processing</span><span style=\"opacity: 0.80\">, constitutive dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation observed in patients with apert syndrome has no effect on receptor folding (19). the k517r mutant receptor served as a </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\">-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously shown to abolish </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity (21).  following transfection of wild-type fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the </span><span style=\"background-color: hsl(0, 100.00%, 97.53%); opacity: 0.80\" title=\"-0.007\">reduction</span><span style=\"opacity: 0.80\"> in size of both bands to a single 98-kda band (</span><span style=\"background-color: hsl(120, 100.00%, 95.23%); opacity: 0.81\" title=\"0.019\">open</span><span style=\"opacity: 0.80\"> arrow), confirming that these two bands are differently glycosylated receptor populations (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the </span><span style=\"background-color: hsl(0, 100.00%, 97.53%); opacity: 0.80\" title=\"-0.007\">reduction</span><span style=\"opacity: 0.80\"> of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with our structural predictions of no adverse </span><span style=\"background-color: hsl(120, 100.00%, 98.12%); opacity: 0.80\" title=\"0.005\">effects</span><span style=\"opacity: 0.80\"> on </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.50%); opacity: 0.81\" title=\"-0.012\">processing</span><span style=\"opacity: 0.80\"> of these fgfr2 mutants. this is also consistent with our structural predictions that these mutations will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5b, bottom), consistent with the decreased </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> yield observed in the in vitro </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> assays.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 5. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 5. mutations in fgfr2 impair receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.50%); opacity: 0.81\" title=\"-0.012\">processing</span><span style=\"opacity: 0.80\"> and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (</span><span style=\"background-color: hsl(120, 100.00%, 95.23%); opacity: 0.81\" title=\"0.019\">open</span><span style=\"opacity: 0.80\"> arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 and e219k (mapping onto the known </span><span style=\"background-color: hsl(120, 100.00%, 96.71%); opacity: 0.81\" title=\"0.011\">extracellular</span><span style=\"opacity: 0.80\"> crystal structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5c). in cells transfected with the g227e mutant (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> clear cell surface localization in keeping with the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we then performed similar immunofluorescence </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> looking for colocalization with the er-resident </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 5d). in contrast, marked cell surface localization and limited colocalization with </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase were observed for both the wild-type fgfr2 and the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay and p44/42 mitogen-</span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> phosphorylation in response to fgf2 as readout of receptor </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">  we next analyzed the </span><span style=\"background-color: hsl(120, 100.00%, 98.12%); opacity: 0.80\" title=\"0.005\">effects</span><span style=\"opacity: 0.80\"> of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express other fgfrs whose </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> may mask the ability to specifically </span><span style=\"background-color: hsl(0, 100.00%, 98.85%); opacity: 0.80\" title=\"-0.003\">evaluate</span><span style=\"opacity: 0.80\"> fgfr2 signaling and </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">, we used the </span><span style=\"background-color: hsl(120, 100.00%, 94.44%); opacity: 0.81\" title=\"0.024\">murine</span><span style=\"opacity: 0.80\"> interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> does not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively </span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> positive control </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> this activating mutation has been reported in endometrial cancer (12) and an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were then done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did not </span><span style=\"background-color: hsl(0, 100.00%, 94.04%); opacity: 0.81\" title=\"-0.026\">cause</span><span style=\"opacity: 0.80\"> constitutive baf3 proliferation in the absence of ligand </span><span style=\"background-color: hsl(0, 100.00%, 97.82%); opacity: 0.80\" title=\"-0.006\">stimulation</span><span style=\"opacity: 0.80\">.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 6. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type and mutant fgfr2_ires_neo expression constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on </span><span style=\"background-color: hsl(0, 100.00%, 97.82%); opacity: 0.80\" title=\"-0.006\">stimulation</span><span style=\"opacity: 0.80\"> with exogenous rfgf2 </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal expression of fgfr2 by all mutants except empty vector in the absence () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand </span><span style=\"background-color: hsl(0, 100.00%, 97.82%); opacity: 0.80\" title=\"-0.006\">stimulation</span><span style=\"opacity: 0.80\">) of that of wild-type receptor. as a further assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-</span><span style=\"background-color: hsl(120, 100.00%, 96.91%); opacity: 0.81\" title=\"0.010\">activated</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k known activating mutant, all mutations resulted in a </span><span style=\"background-color: hsl(0, 100.00%, 97.53%); opacity: 0.80\" title=\"-0.007\">reduction</span><span style=\"opacity: 0.80\"> to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr with isoform-</span><span style=\"background-color: hsl(120, 100.00%, 97.93%); opacity: 0.80\" title=\"0.006\">specific</span><span style=\"opacity: 0.80\"> primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, our reintroduction </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 7a ). expression of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell line (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not shown).  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 7. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. </span><span style=\"background-color: hsl(120, 100.00%, 88.44%); opacity: 0.83\" title=\"0.068\">overexpression</span><span style=\"opacity: 0.80\"> of fgfr2 or </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression was confirmed by western blot analysis. discussion  in this study, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\">. to determine the ratio of nonsynonymous to synonymous mutations in our data set, we went back and examined the synonymous variants we identified. </span><span style=\"background-color: hsl(0, 100.00%, 99.34%); opacity: 0.80\" title=\"-0.001\">because</span><span style=\"opacity: 0.80\"> we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent with the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a </span><span style=\"background-color: hsl(120, 100.00%, 97.87%); opacity: 0.80\" title=\"0.006\">substitutions</span><span style=\"opacity: 0.80\"> with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, and we anticipate that further sequencing analysis in a greater number of primary </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> and nevi will shed light on when during melanoma development these mutations arise. </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> should be noted that the mutation frequency reported in this study may be an underestimate. this is based on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not all primary </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 98.19%); opacity: 0.80\" title=\"0.005\">provided</span><span style=\"opacity: 0.80\"> high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined with in vitro and in vivo functional </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\">, show that these mutations lead to </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of receptor activity through </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> mechanisms, including </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of ligand binding (r251q), incomplete </span><span style=\"background-color: hsl(0, 100.00%, 96.50%); opacity: 0.81\" title=\"-0.012\">processing</span><span style=\"opacity: 0.80\"> of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity (a648t, d530n, i642v). future </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> confirming the structural predictions described herein may also reveal that fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> nature of our identified melanoma mutations is also consistent with published literature. in support of d530n resulting in fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">, a germ-line d771n mutation at the equivalent codon in ret has been associated with hirschsprung&#x27;s disease, commonly due to </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations in ret (23). in </span><span style=\"background-color: hsl(0, 100.00%, 97.61%); opacity: 0.80\" title=\"-0.007\">addition</span><span style=\"opacity: 0.80\">, while these </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> were under way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent </span><span style=\"background-color: hsl(0, 100.00%, 91.75%); opacity: 0.82\" title=\"-0.042\">families</span><span style=\"opacity: 0.80\"> with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. </span><span style=\"background-color: hsl(0, 100.00%, 95.40%); opacity: 0.81\" title=\"-0.018\">24</span><span style=\"opacity: 0.80\">). consistent with our in vitro and in vivo data, a recent study showed that the a648t mutation abrogated the </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> n-lobe toward the c-lobe, which is predicted to take place when the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> activity awaits the resolution of crystal structure of the mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain as was </span><span style=\"background-color: hsl(0, 100.00%, 98.44%); opacity: 0.80\" title=\"-0.004\">recently</span><span style=\"opacity: 0.80\"> carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the </span><span style=\"background-color: hsl(120, 100.00%, 97.97%); opacity: 0.80\" title=\"0.006\">catalytic</span><span style=\"opacity: 0.80\"> loop of the </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain, and the crystal structure of the mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring </span><span style=\"background-color: hsl(0, 100.00%, 97.35%); opacity: 0.80\" title=\"-0.008\">key</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.97%); opacity: 0.80\" title=\"0.006\">catalytic</span><span style=\"opacity: 0.80\"> residues (26).  a comparison of the mutations identified in melanoma to those our lab previously identified in endometrial cancer highlights the differences in the mutation spectra (</span><span style=\"background-color: hsl(120, 100.00%, 99.86%); opacity: 0.80\" title=\"0.000\">fig</span><span style=\"opacity: 0.80\">. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations previously reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer with germ-line mutations associated with skeletal disorders </span><span style=\"background-color: hsl(0, 100.00%, 95.88%); opacity: 0.81\" title=\"-0.015\">shows</span><span style=\"opacity: 0.80\"> a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than 20 fgfr2 mutations we identified in melanoma were all novel, supporting our findings that the mutations are distinct and </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> from the many </span><span style=\"background-color: hsl(0, 100.00%, 93.28%); opacity: 0.82\" title=\"-0.031\">gain</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations previously identified in the fgfrs.  </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 8. download figureopen in new tabdownload powerpoint </span><span style=\"background-color: hsl(120, 100.00%, 99.71%); opacity: 0.80\" title=\"0.000\">figure</span><span style=\"opacity: 0.80\"> 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured </span><span style=\"background-color: hsl(120, 100.00%, 99.30%); opacity: 0.80\" title=\"0.001\">tumors</span><span style=\"opacity: 0.80\"> are presented in red above the schematic representation of the </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (top) above the schematic representation of the </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> and are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-line mutations associated with a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although not previously reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this discovery of </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations in a receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> is unexpected, given the conventional wisdom that </span><span style=\"background-color: hsl(120, 100.00%, 95.72%); opacity: 0.81\" title=\"0.016\">activation</span><span style=\"opacity: 0.80\"> of receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (12). however, </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> should not be surprising given that a variety of receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> signaling pathways can induce either proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic </span><span style=\"background-color: hsl(0, 100.00%, 99.19%); opacity: 0.80\" title=\"-0.002\">mice</span><span style=\"opacity: 0.80\">, with fgfr2b having been deleted in cells </span><span style=\"background-color: hsl(0, 100.00%, 78.04%); opacity: 0.88\" title=\"-0.169\">expressing</span><span style=\"opacity: 0.80\"> keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene and 12-o-tetradecanoylphorbol-</span><span style=\"background-color: hsl(0, 100.00%, 97.30%); opacity: 0.80\" title=\"-0.008\">13</span><span style=\"opacity: 0.80\">-acetate </span><span style=\"background-color: hsl(0, 100.00%, 95.31%); opacity: 0.81\" title=\"-0.019\">treatment</span><span style=\"opacity: 0.80\">, these transgenic </span><span style=\"background-color: hsl(0, 100.00%, 99.19%); opacity: 0.80\" title=\"-0.002\">mice</span><span style=\"opacity: 0.80\"> developed an increased number of papillomas and carcinomas compared with wild-type </span><span style=\"background-color: hsl(0, 100.00%, 99.19%); opacity: 0.80\" title=\"-0.002\">mice</span><span style=\"opacity: 0.80\">, providing support that fgfr2 can act as a tumor </span><span style=\"background-color: hsl(120, 100.00%, 90.96%); opacity: 0.82\" title=\"0.048\">suppressor</span><span style=\"opacity: 0.80\"> gene in </span><span style=\"background-color: hsl(120, 100.00%, 97.64%); opacity: 0.80\" title=\"0.007\">some</span><span style=\"opacity: 0.80\"> cell types (29). furthermore, </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 expression has been associated with several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-</span><span style=\"background-color: hsl(0, 100.00%, 97.28%); opacity: 0.80\" title=\"-0.009\">36</span><span style=\"opacity: 0.80\">), and salivary adenocarcinoma cells (37).  based on these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\">, </span><span style=\"background-color: hsl(0, 100.00%, 99.33%); opacity: 0.80\" title=\"-0.001\">it</span><span style=\"opacity: 0.80\"> was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the previously published reintroduction results and the current melanoma study is that they involve </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> fgfr2 splice forms. the previous </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> were all carried out in cells of epithelial origin with the fgfr2 iiib isoform, whereas our </span><span style=\"background-color: hsl(0, 100.00%, 99.23%); opacity: 0.80\" title=\"-0.001\">studies</span><span style=\"opacity: 0.80\"> evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following </span><span style=\"background-color: hsl(120, 100.00%, 88.44%); opacity: 0.83\" title=\"0.068\">overexpression</span><span style=\"opacity: 0.80\"> of fgfr2 may therefore reflect a cell lineage</span><span style=\"background-color: hsl(120, 100.00%, 97.93%); opacity: 0.80\" title=\"0.006\">specific</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> of fgfr2. another possible explanation is that </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> inhibitory signal. indeed, although tumor suppressors classically </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> may not directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of understanding about the differing cellular </span><span style=\"background-color: hsl(0, 100.00%, 89.47%); opacity: 0.83\" title=\"-0.059\">functions</span><span style=\"opacity: 0.80\"> of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> and melanoma pathogenesis is currently unknown. our lab is currently investigating the relative expression levels of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular </span><span style=\"background-color: hsl(0, 100.00%, 89.47%); opacity: 0.83\" title=\"-0.059\">functions</span><span style=\"opacity: 0.80\"> distinct from typical receptor </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">tyrosine</span><span style=\"opacity: 0.80\"> </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> signaling. there have been reports that fgfr1 binds cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in various cell types (39).8 lastly, we and others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (40)8 and fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology will allow us to </span><span style=\"background-color: hsl(120, 100.00%, 94.12%); opacity: 0.81\" title=\"0.026\">identify</span><span style=\"opacity: 0.80\"> the mechanism by which </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 </span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which both </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> and </span><span style=\"background-color: hsl(0, 100.00%, 93.28%); opacity: 0.82\" title=\"-0.031\">gain</span><span style=\"opacity: 0.80\">-of-</span><span style=\"background-color: hsl(0, 100.00%, 96.34%); opacity: 0.81\" title=\"-0.013\">function</span><span style=\"opacity: 0.80\"> mutations have been reported in </span><span style=\"background-color: hsl(0, 100.00%, 99.70%); opacity: 0.80\" title=\"-0.000\">different</span><span style=\"opacity: 0.80\"> tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by which </span><span style=\"background-color: hsl(0, 100.00%, 97.15%); opacity: 0.80\" title=\"-0.009\">loss</span><span style=\"opacity: 0.80\"> of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed with wild-type or mutant expression constructs and </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-type or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-type and mutant proteins were then sequentially purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  expression and purification of fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial expression vectors with an nh2-</span><span style=\"background-color: hsl(0, 100.00%, 85.01%); opacity: 0.85\" title=\"-0.098\">terminal</span><span style=\"opacity: 0.80\"> 6his-tag to aid in </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> purification. point mutations (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed with wild-type or mutant fgfr2 </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> expression constructs, and </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were then lysed using a french press, and the soluble </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> proteins were purified by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. purified wild-type and mutant </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-type and mutant kinases was compared using a continuous spectrophotometric </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\"> assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary </span><span style=\"background-color: hsl(0, 100.00%, 99.58%); opacity: 0.80\" title=\"-0.001\">table</span><span style=\"opacity: 0.80\"> s1. after restriction enzyme screening, the entire </span><span style=\"background-color: hsl(0, 100.00%, 92.60%); opacity: 0.82\" title=\"-0.036\">coding</span><span style=\"opacity: 0.80\"> sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation and to ensure that no other mutations were introduced during the mutagenesis process. plasmid dna was then isolated using qiagen endofree plasmid dna isolation kits, and dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained under standard culturing conditions as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-type or mutant fgfr2 + 750 g empty plasmid) were achieved with fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at </span><span style=\"background-color: hsl(0, 100.00%, 97.28%); opacity: 0.80\" title=\"-0.009\">36</span><span style=\"opacity: 0.80\"> to 48 h posttransfection, cells were washed with ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> was resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes were probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described and plated onto eight-well chamber slides. after </span><span style=\"background-color: hsl(0, 100.00%, 97.28%); opacity: 0.80\" title=\"-0.009\">36</span><span style=\"opacity: 0.80\"> to 48 h, cells were fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated with 1:300 bek c-17 and 1:1,000 anti</span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> disulfide isomerase (invitrogen) and then detected with goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted with pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected with lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at </span><span style=\"background-color: hsl(0, 100.00%, 95.40%); opacity: 0.81\" title=\"-0.018\">24</span><span style=\"opacity: 0.80\"> and 40 h, pooled, filtered through a 0.45-um low </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> binding filter, and stored at 80c until use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with crystal violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-type and mutant fgfr2 were maintained under selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented with 5 ng/ml </span><span style=\"background-color: hsl(120, 100.00%, 94.44%); opacity: 0.81\" title=\"0.024\">murine</span><span style=\"opacity: 0.80\"> il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced with an equal volume containing fresh ligand at 2 concentration and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in rpmi supplemented with 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced with empty vector, fgfr2, or </span><span style=\"background-color: hsl(120, 100.00%, 97.96%); opacity: 0.80\" title=\"0.006\">kinase</span><span style=\"opacity: 0.80\">-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full </span><span style=\"background-color: hsl(120, 100.00%, 98.41%); opacity: 0.80\" title=\"0.004\">growth</span><span style=\"opacity: 0.80\"> media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were then stimulated with 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in either baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total </span><span style=\"background-color: hsl(120, 100.00%, 99.06%); opacity: 0.80\" title=\"0.002\">protein</span><span style=\"opacity: 0.80\"> were resolved on a 4% to 12% bis tris gel and probed for phosho-p44/42 mapk. blots were then stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=8\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.001</b>, score <b>-3.920</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__50mm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__10mm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.94%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__150mm\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__n6\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__10th\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__houston\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__klenow\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pymol\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__keep\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fkbp12\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cord\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pip2\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__administered\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__operon\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__731\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cos\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__featured\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bringing\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__blocked\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.82%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__c9\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.77%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__imager\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.75%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mt\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__followed\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.016\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__magnification\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__separated\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__images\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.48%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.017\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__methionine\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.047\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__leads\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.54%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.071\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__retain\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.34%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.086\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__loaded\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.13%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.102\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__platelet\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.73%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.134\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__400\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.44%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.160\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__clontech\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.21%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.181\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 80.00%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -3.015\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there would be 62,480 new cases of melanoma and about 8,420 deaths due to melanoma in 2008. previous studies in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic expression of growth factors and/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and epithelial cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-</span><span style=\"background-color: hsl(0, 100.00%, 98.19%); opacity: 0.80\" title=\"-0.005\">specific</span><span style=\"opacity: 0.80\"> alternatively spliced b isoform (epithelial) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends on fgf2, which is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor growth and induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, aberrant splicing, or missense mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (12).  we report here the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma tumors. mapping of these mutations onto the known crystal structures of fgfr2 together with in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> braf or nras (table 1). no mutations were identified in </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma tumors and a panel of 72 vertical growth phase primary melanomas representing samples from different melanoma subtypes for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples and 5 of 72 primary tumors (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding normal tissue, and in all cases the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, and p708s), which were not further analyzed because there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary tumors carried nonsense mutations (table 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras and braf were observed in tumors with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 structures  extracellular mutations. to gain insights into how the different fgfr2 mutations might affect receptor function, we analyzed them in the light of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which affect residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; fig. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with each other directly and each receptor makes contacts with both ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular mutations identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in addition to the mutated residue, other relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with other residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\">. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\">. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds with other residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, on the other hand, maps to the large loop region between strand c and c, which protrudes from d3 and does not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\">- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> interacts with the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation maps to the highly conserved d2-d3 linker region (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its primary fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (fig. 1b). by contrast, the e219k mutation maps to the secondary </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> (fig. 1e). therefore, based on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other extracellular mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 where its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (fig. 1a). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, maps to the start of the f strand in d2 and is spatially in close vicinity to v247 (fig. 1a). g270, which corresponds to g271 in fgfr2, maps to the loop between the a and b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based on the structure, these substitutions should cause local structural perturbations, which could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to affect the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on which a key lysine residue involved in heparan sulfate binding resides (fig. 1d). therefore, destabilization of the ga helix due to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), and the bulkier tyrosine side chain at this location could cause steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  kinase domain mutations. more than half of the melanoma mutations affect the conserved tyrosine kinase domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 kinase domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (fig. 2 ). notably, m640 and i642 localize onto </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> end of the 8 strand (fig. 2), a secondary structure element that precedes the activation loop (a-loop) of the kinase domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the kinase domain. substitutions of </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> residues with the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the kinase that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation maps to the activation loop of the kinase domain (fig. 2e) and is also expected to lead to loss of function. this is because in the phosphorylated activated kinase domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the kinase and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c kinase domain mutations identified in melanoma tumors/cell lines mapped onto the crystal structures of the fgfr2 kinase domain suggest that these mutations should impair the kinase activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 kinase domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular tyrosine kinase domain is as follows: the nh2-terminal lobe of kinase is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the kinase hinge region is in green; and the nh2-terminal tail of the kinase is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the kinase domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the effects of the melanoma mutations, are labeled. a. e636, which maps to the sharp turn between 7 and 8 strands, makes hydrogen bond with s563 in the kinase hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine kinase activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the kinase domain and engage c606 and m614 (cyan mesh). mutation of </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a reduction in kinase activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the kinase domain, specifically making the kinase domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is expected to lead to loss of function because the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the kinase region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation maps to the loop region between the 7 and 8 strands (fig. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge region of the kinase domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the kinase relative to each other. based on the structure, the e636k mutation should influence the kinase activity of the fgfr2 kinase domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved kinase domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated kinase domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds due to the e475k mutation could relieve this inhibition and potentially lead to activation of the kinase domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the kinase and reduce the half life of the mutant fgfr2. the g701s mutation maps to the loop between f and g helices (fig. 2f) and facilitates the conformation of this loop region, which makes several hydrogen bonds with k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced kinase activity. the r759q mutation maps to the cooh-terminal end of the i helix, the last secondary structure element of the kinase domain. in the structure, r759 makes hydrogen bonds with two other residues in its vicinity (fig. </span><span style=\"background-color: hsl(0, 100.00%, 95.70%); opacity: 0.81\" title=\"-0.016\">2c</span><span style=\"opacity: 0.80\">), and loss of these hydrogen bonds due to the r759q mutation could also induce subtle structural changes leading to a loss in kinase activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for </span><span style=\"background-color: hsl(0, 100.00%, 99.65%); opacity: 0.80\" title=\"-0.000\">sh2</span><span style=\"opacity: 0.80\"> domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc </span><span style=\"background-color: hsl(0, 100.00%, 99.65%); opacity: 0.80\" title=\"-0.000\">sh2</span><span style=\"opacity: 0.80\"> domain in complex with phosphopeptide indicates that the </span><span style=\"background-color: hsl(0, 100.00%, 99.65%); opacity: 0.80\" title=\"-0.000\">sh2</span><span style=\"opacity: 0.80\"> domain of plc interacts with ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of </span><span style=\"background-color: hsl(0, 100.00%, 99.65%); opacity: 0.80\" title=\"-0.000\">sh2</span><span style=\"opacity: 0.80\"> domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate binding pocket of the kinase domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal structures. consistent with these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 protein translation (table 1).  view inlineview popup table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test our structural predictions for the extracellular mutations on fgfr2 function, we expressed recombinant wild-type and mutant ectodomains in e. coli, and then isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have previously shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro with yields similar to those of the wild-type and s252w mutant receptors. we then used surface plasmon resonance to analyze the effect of the r251q mutation on </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the other hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding. a and b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of time. surface plasmon resonance analysis was done as previously described (19). to test the effect of kinase domain mutations, we prepared recombinant wild-type and mutant kinase domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro kinase assays. consistent with structural predictions, the d530n, i642v, and a648t mutant kinase domains exhibited reduced kinase activity relative to the wild-type kinase domain (fig. 4 ). interestingly, the e475k mutant exhibited a slight increase in kinase activity relative to the wild-type kinase domain. notably, the protein expression yield for the e475k mutant kinase was severalfold less than that of wild-type kinase, suggesting that this mutation was partially destabilizing the kinase domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 kinase domains harboring melanoma mutations have reduced tyrosine autophosphorylation activity. wild-type fgfr2k and four mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro kinase assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-type and mutant fgfr2ks were determined using a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured cells  we next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation observed in patients with apert syndrome has no effect on receptor folding (19). the k517r mutant receptor served as a kinase-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously shown to abolish kinase activity (21).  following transfection of wild-type fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive </span><span style=\"background-color: hsl(120, 100.00%, 97.81%); opacity: 0.80\" title=\"0.006\">bands</span><span style=\"opacity: 0.80\"> (110 and 130 kda) are observed. pngase digestion resulted in the reduction in size of both </span><span style=\"background-color: hsl(120, 100.00%, 97.81%); opacity: 0.80\" title=\"0.006\">bands</span><span style=\"opacity: 0.80\"> to a single 98-kda band (open arrow), confirming that these two </span><span style=\"background-color: hsl(120, 100.00%, 97.81%); opacity: 0.80\" title=\"0.006\">bands</span><span style=\"opacity: 0.80\"> are differently glycosylated receptor populations (fig. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (fig. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with our structural predictions of no adverse effects on protein stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent with our structural predictions that these mutations will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (fig. 5b, bottom), consistent with the decreased protein yield observed in the in vitro kinase assays.  figure 5. download figureopen in new tabdownload powerpoint figure 5. mutations in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 and e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (fig. 5c). in cells transfected with the g227e mutant (fig. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping with the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we then performed similar immunofluorescence studies looking for colocalization with the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in contrast, marked cell surface localization and limited colocalization with protein disulfide isomerase were observed for both the wild-type fgfr2 and the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated protein kinase phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the effects of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express other fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it does not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine kinase signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (12) and an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were then done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (fig. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the absence of </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type and mutant fgfr2_ires_neo expression constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the absence of </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\">; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in </span><span style=\"background-color: hsl(0, 100.00%, 92.48%); opacity: 0.82\" title=\"-0.037\">triplicate</span><span style=\"opacity: 0.80\">. b. analysis of phospho-p44/42 mapk on stimulation with exogenous rfgf2 shows that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal expression of fgfr2 by all mutants except empty vector in the absence () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> stimulation) of that of wild-type receptor. as a further assessment of receptor impairment due to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein kinase (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k known activating mutant, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (fig. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 could </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">suppress</span><span style=\"opacity: 0.80\"> proliferation of melanoma cells. reverse transcription-pcr with isoform-</span><span style=\"background-color: hsl(0, 100.00%, 98.19%); opacity: 0.80\" title=\"-0.005\">specific</span><span style=\"opacity: 0.80\"> primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, our reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">suppress</span><span style=\"opacity: 0.80\"> the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell line (fig. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">suppress</span><span style=\"opacity: 0.80\"> proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or kinase-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression was confirmed by western blot analysis. discussion  in this study, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary tumors. to determine the ratio of nonsynonymous to synonymous mutations in our data set, we went back and examined the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent with the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, and we anticipate that further sequencing analysis in a greater number of primary tumors and nevi will shed light on when during melanoma development these mutations arise. it should be noted that the mutation frequency reported in this study may be an underestimate. this is based on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not all primary tumors provided high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined with in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine kinase activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the loss-of-function nature of our identified melanoma mutations is also consistent with published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated with hirschsprung&#x27;s disease, commonly due to loss-of-function mutations in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent families with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent with our in vitro and in vivo data, a recent study showed that the a648t mutation abrogated the tyrosine kinase activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the kinase n-lobe toward the c-lobe, which is predicted to take place when the kinase transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 kinase activity awaits the resolution of crystal structure of the mutant kinase domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation maps onto the catalytic loop of the kinase domain, and the crystal structure of the mutant kinase shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those our lab previously identified in endometrial cancer highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations previously reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer with germ-line mutations associated with skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than 20 fgfr2 mutations we identified in melanoma were all novel, supporting our findings that the mutations are distinct and different from the many gain-of-function mutations previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured tumors are presented in red above the schematic representation of the protein and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (top) above the schematic representation of the protein and are numbered relative to fgfr2b (np_075259.2; ref. 12). bottom, germ-line mutations associated with a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although not previously reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (</span><span style=\"background-color: hsl(0, 100.00%, 93.09%); opacity: 0.82\" title=\"-0.032\">indicated</span><span style=\"opacity: 0.80\"> with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this discovery of loss-of-function mutations in a receptor tyrosine kinase is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (12). however, it should not be surprising given that a variety of receptor tyrosine kinase signaling pathways can induce </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic mice, with fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,12-dimethylbenz(a)anthracene and 12-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased number of papillomas and carcinomas compared with wild-type mice, providing support that fgfr2 can act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated with several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-36), and salivary adenocarcinoma cells (37).  based on these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was indeed surprising that reintroduction of fgfr2 failed to </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">suppress</span><span style=\"opacity: 0.80\"> proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the previously published reintroduction results and the </span><span style=\"background-color: hsl(0, 100.00%, 97.38%); opacity: 0.80\" title=\"-0.008\">current</span><span style=\"opacity: 0.80\"> melanoma study is that they involve different fgfr2 splice forms. the previous studies were all carried out in cells of epithelial origin with the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineage</span><span style=\"background-color: hsl(0, 100.00%, 98.19%); opacity: 0.80\" title=\"-0.005\">specific</span><span style=\"opacity: 0.80\"> function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which function to </span><span style=\"background-color: hsl(0, 100.00%, 98.60%); opacity: 0.80\" title=\"-0.003\">suppress</span><span style=\"opacity: 0.80\"> tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic consequence of loss of fgfr2 function may not directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a consequence of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our </span><span style=\"background-color: hsl(0, 100.00%, 97.38%); opacity: 0.80\" title=\"-0.008\">current</span><span style=\"opacity: 0.80\"> lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently unknown. our lab is currently investigating the relative expression levels of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine kinase signaling. there have been reports that fgfr1 binds cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in various cell types (39).8 lastly, we and others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (40)8 and fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology will allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological consequence of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which both loss-of-function and gain-of-function mutations have been reported in different tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed with wild-type or mutant expression constructs and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-type or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l </span><span style=\"background-color: hsl(0, 100.00%, 91.02%); opacity: 0.82\" title=\"-0.047\">nacl</span><span style=\"opacity: 0.80\">, 10% glycerol, and l-cysteine. the correctly refolded wild-type and mutant proteins were then sequentially purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  expression and purification of fgfr2 kinase domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial expression vectors with an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed with wild-type or mutant fgfr2 kinase expression constructs, and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were then lysed using a french press, and the soluble kinase proteins were purified by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. purified wild-type and mutant kinase proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-type and mutant kinases was compared using a continuous spectrophotometric kinase assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation and to ensure that no other mutations were introduced during the mutagenesis process. plasmid dna was then isolated using qiagen endofree plasmid dna isolation kits, and dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained under standard culturing conditions as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-type or mutant fgfr2 + 750 g empty plasmid) were achieved with fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells were washed with ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes were probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described and plated onto eight-well chamber slides. after 36 to 48 h, cells were fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated with 1:300 bek c-17 and 1:1,000 antiprotein disulfide isomerase (invitrogen) and then detected with goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted with pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected with lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and 40 h, pooled, filtered through a 0.45-um low protein binding filter, and stored at 80c until use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with crystal violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, then placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-type and mutant fgfr2 were maintained under selection in </span><span style=\"background-color: hsl(120, 100.00%, 96.99%); opacity: 0.80\" title=\"0.010\">rpmi</span><span style=\"opacity: 0.80\">/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented with 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in </span><span style=\"background-color: hsl(0, 100.00%, 92.48%); opacity: 0.82\" title=\"-0.037\">triplicate</span><span style=\"opacity: 0.80\"> in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced with an equal volume containing fresh </span><span style=\"background-color: hsl(0, 100.00%, 99.38%); opacity: 0.80\" title=\"-0.001\">ligand</span><span style=\"opacity: 0.80\"> at 2 concentration and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in </span><span style=\"background-color: hsl(120, 100.00%, 96.99%); opacity: 0.80\" title=\"0.010\">rpmi</span><span style=\"opacity: 0.80\"> supplemented with 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced with empty vector, fgfr2, or kinase-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full growth media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were then stimulated with 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in </span><span style=\"background-color: hsl(0, 100.00%, 99.77%); opacity: 0.80\" title=\"-0.000\">either</span><span style=\"opacity: 0.80\"> baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief </span><span style=\"background-color: hsl(120, 100.00%, 95.55%); opacity: 0.81\" title=\"0.017\">centrifugation</span><span style=\"opacity: 0.80\"> and immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved on a 4% to 12% bis tris gel and probed for phosho-p44/42 mapk. blots were then stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "        \n",
       "    \n",
       "        \n",
       "        \n",
       "    \n",
       "        <p style=\"margin-bottom: 0.5em; margin-top: 0em\">\n",
       "            <b>\n",
       "    \n",
       "        y=9\n",
       "    \n",
       "</b>\n",
       "\n",
       "    \n",
       "    (probability <b>0.001</b>, score <b>-3.972</b>)\n",
       "\n",
       "top features\n",
       "        </p>\n",
       "    \n",
       "    <table class=\"eli5-weights\"\n",
       "           style=\"border-collapse: collapse; border: none; margin-top: 0em; table-layout: auto; margin-bottom: 2em;\">\n",
       "        <thead>\n",
       "        <tr style=\"border: none;\">\n",
       "            \n",
       "                <th style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\" title=\"Feature contribution already accounts for the feature value (for linear models, contribution = weight * feature value), and the sum of feature contributions is equal to the score or, for some classifiers, to the probability. Feature values are shown if &quot;show_feature_values&quot; is True.\">\n",
       "                    Contribution<sup>?</sup>\n",
       "                </th>\n",
       "            \n",
       "            <th style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">Feature</th>\n",
       "            \n",
       "        </tr>\n",
       "        </thead>\n",
       "        <tbody>\n",
       "        \n",
       "        \n",
       "\n",
       "        \n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__p73\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__htert\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.93%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__invasion\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__sp1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__upregulating\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.92%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pcrs\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__vimentin\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.001\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__fivefold\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__ranking\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__hepatic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.89%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__r248q\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.88%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__programming\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__irrespective\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__modulation\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.002\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__machinery\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__bioinformatics\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.86%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__lymphocytic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mds\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__besides\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__differential\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__cytopenia\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.85%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__spliceosome\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__downregulates\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__branch\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__refractory\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.84%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__excess\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.83%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.003\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__who\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.79%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.005\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__neoplasms\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.74%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.006\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__myelodysplastic\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.05%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.039\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__pre\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 99.02%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.041\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__stem\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.90%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.048\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__4i\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 98.21%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.096\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Gene__sf3b1\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 97.91%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.120\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__qpcr\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 96.87%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.213\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text: Highlighted in text (sum)\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 93.33%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -0.629\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        Text__mrna\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "            <tr style=\"background-color: hsl(0, 100.00%, 81.39%); border: none;\">\n",
       "    <td style=\"padding: 0 1em 0 0.5em; text-align: right; border: none;\">\n",
       "        -2.720\n",
       "    </td>\n",
       "    <td style=\"padding: 0 0.5em 0 0.5em; text-align: left; border: none;\">\n",
       "        &lt;BIAS&gt;\n",
       "    </td>\n",
       "    \n",
       "</tr>\n",
       "        \n",
       "\n",
       "        </tbody>\n",
       "    </table>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n",
       "    <p style=\"margin-bottom: 2.5em; margin-top:-0.5em;\">\n",
       "        <b>Variation:</b> <span style=\"opacity: 0.80\">e219k</span><br/><b>Gene:</b> <span style=\"opacity: 0.80\">fgfr2</span><br/><b>Text:</b> <span style=\"opacity: 0.80\">introduction  melanoma is the most lethal of all skin cancers. the american cancer society estimated that there would be 62,480 new </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.001\">cases</span><span style=\"opacity: 0.80\"> of melanoma and about 8,420 deaths </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to melanoma in 2008. previous studies in human melanocytes and melanoma cells have revealed that one of the hallmarks of melanocytic transformation is a change from paracrine growth factor stimulation by the surrounding keratinocytes in the skin to autocrine growth factor stimulation via ectopic expression of growth factors and/or receptors by melanoma cells (1, 2). fibroblast growth factor (fgf) signaling plays a prominent role in tissue homeostasis by providing bidirectional paracrine communication between mesenchymal and </span><span style=\"background-color: hsl(0, 100.00%, 99.64%); opacity: 0.80\" title=\"-0.000\">epithelial</span><span style=\"opacity: 0.80\"> cells of parenchymal organs including the skin. the fgf family comprises 18 ligands (fgf1-fgf10 and fgf16-fgf23), which signal through four transmembrane receptor tyrosine kinases (fgfr1-fgfr4) and their tissue-specific alternatively spliced b isoform (</span><span style=\"background-color: hsl(0, 100.00%, 99.64%); opacity: 0.80\" title=\"-0.000\">epithelial</span><span style=\"opacity: 0.80\">) and c isoform (mesenchymal; ref. 3). previous studies have shown that survival and proliferation of melanocytes depends on fgf2, which is provided by the surrounding keratinocytes in the skin. a number of functional studies have implicated both fgf2 and fgfr1 signaling in melanoma progression. adenoviral transduction of melanocytes with fgf2 resulted in phenotypic changes consistent with increased tumorigenicity (4, 5), and down-regulation of fgf2 via antisense oligonucleotides in metastatic melanoma inhibited proliferation and colony formation in soft agar (6, 7). introduction of antisense oligonucleotides targeted toward fgfr1 into melanocytes and metastatic melanoma cell lines resulted in decreased proliferation and signs of differentiation (8, 9). furthermore, injection of an antisense fgfr1 construct into primary and metastatic melanomas grown in nude mice has also been shown to result in inhibition of tumor growth and induction of apoptosis (10, 11).  constitutive activation of fgfrs through chromosomal translocation, aberrant splicing, or missense mutations has been reported in several cancers. the catalog of somatic mutations in cancer (cosmic)7 provides a repository of all somatic changes reported to date in this receptor family. to date, fgfr3 is mutated at highest frequency in benign seborrheic keratoses and bladder cancer and to a lesser extent in multiple myeloma and cervical cancer. we have recently reported the presence of activating mutations in fgfr2 in 16% of endometrioid endometrial cancers (</span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">).  we report here the identification of novel fgfr2 mutations in 15 of 113 (13%) of melanoma cell lines and 8 of 100 (8%) of uncultured melanoma tumors. mapping of these mutations onto the known crystal structures of fgfr2 together with in vitro and in vivo functional analyses shows that these mutations result in receptor loss of function.  results  identification of fgfr2 mutations in melanoma cell lines and uncultured primary and metastatic melanoma tumors  we screened an initial panel of 47 melanoma cell lines for the presence of mutations in the fgfr1 to fgfr4 genes by sequencing. following the identification of mutations in fgfr2, 66 additional melanoma cell lines were screened for fgfr2 mutations using a combination of denaturing high-performance liquid chromatography and sequencing leading to the identification of fifteen different fgfr2 mutations (table 1 ). none of these mutations were observed in a panel of lymphocyte dna from 150 caucasian controls (data not shown), suggesting that these mutations arose somatically. of the 15 cell lines carrying fgfr2 mutations, 11 also carried mutations in either braf or nras (table 1). no mutations were identified in either fgfr1 or fgfr3, whereas one mutation at a nonconserved codon was found in fgfr4, p716r.  view inlineview popup table 1. summary of melanoma samples carrying fgfr2 mutations next, we screened a panel of 28 metastatic melanoma tumors and a panel of 72 vertical growth phase primary melanomas representing samples from different melanoma </span><span style=\"background-color: hsl(120, 100.00%, 94.09%); opacity: 0.81\" title=\"0.026\">subtypes</span><span style=\"opacity: 0.80\"> for the presence of mutations in fgfr2 gene. we identified mutations in 3 of 28 metastatic samples and 5 of 72 primary tumors (table 1). subset analysis revealed fgfr2 mutations in 1 of 17 (6%) superficial spreading melanomas, 2 of 10 (20%) lentigo maligna melanomas, 1 of 28 (4%) acral melanomas, and 1 of 17 (6%) mucosal melanomas. for three of these tumors, we were able to extract dna from surrounding normal tissue, and in all </span><span style=\"background-color: hsl(0, 100.00%, 99.31%); opacity: 0.80\" title=\"-0.001\">cases</span><span style=\"opacity: 0.80\"> the mutation was only present in the tumor, confirming the somatic origin of these mutations. reminiscent of the mutator phenotype seen in the breast cancer kinome screen (13), one metastatic tumor was found to carry four different fgfr2 mutations (v77m, e574k, s688f, and p708s), which were not </span><span style=\"background-color: hsl(0, 100.00%, 98.17%); opacity: 0.80\" title=\"-0.005\">further</span><span style=\"opacity: 0.80\"> analyzed because there was a high probability that these mutations represented passenger mutations rather than driver mutations. notably, two primary tumors carried nonsense mutations (table 1). the majority (20 of 22) of fgfr2 mutations occurred at residues that are conserved across fgfr1 to fgfr4 or at residues that are conserved in three of four of the fgf receptors as well as conserved in fgfr2 across evolution (data not shown). mutations in both nras and braf were observed in tumors with fgfr2 mutations at similar frequencies to that reported in the literature.  mapping of fgfr2 mutations within known fgfr2 structures  extracellular mutations. to gain insights into how the different fgfr2 mutations might affect receptor function, we analyzed them in the light of our crystallographic data on fgfrs. the e219k, g227e, v248d, r251q, and g271e mutations, which affect residues in the extracellular region of fgfr2, were mapped onto the crystal structure of the 2:2:2 fgf2-fgfr1c-heparin dimer (pdb entry 1fq9; ref. 14; fig. 1 ). we intentionally chose this structure instead of the crystal structure of the 1:1 fgf2-fgfr2c monomer to evaluate the effect of the mutations on heparin/heparan sulfate and fgf-induced fgfr dimerization. in the dimer, two centrally located fgfrs interact with each other directly and each receptor makes contacts with both ligands as well.  figure 1. download figureopen in new tabdownload powerpoint figure 1. mapping of fgfr2c extracellular mutations identified in melanoma tumors/cell lines onto the known fgf-fgfr crystal structures suggests that these mutations should impair fgfr2c activity. the locations of mutated residues in the extracellular region of fgfr2 are mapped onto the ribbon diagram of 2:2:2 fgf2-fgfr1c-heparin complex (pdb id: 1fq9; ref. 14). fgf is colored orange and the extracellular ligand binding region of fgfr is colored as follows: d2 in green, d3 in cyan, and d2-d3 linker in gray. the two heparin oligosaccharides in the dimer are rendered in sticks. to assist the viewer, certain  strands of d2 and d3 domains are labeled. a to e. close-up view of the region where the mutated receptor residues are located. in each image, in addition to the mutated residue, other relevant receptor residues are shown as ball-and-sticks. a. the molecular interactions of g226 and v247, which correspond to the mutated g227 and v248 in fgfr2. g226 in f strand makes hydrogen bonds with l245 in g strand and contributes to proper f-g strand pair formation and, hence, to the overall stability of d2. v247 engages in hydrophobic contacts with other residues in the interior core of d2 and thus plays a critical role in folding of d2 as well. to emphasize these hydrophobic contacts, the surface of v247 and neighboring residues are shown in cyan (v247) and purple meshes, respectively. b. r250, homologous to r251 in fgfr2, makes three hydrogen bonds with fgf ligand. mutation of r251 to q should substantially reduce the binding affinity of mutated fgfr2 toward fgf ligand. c. the molecular environments of g270 and n304 in fgfr1, which correspond to the mutated g271 and g305 in fgfr2. g270 plays a role in formation of turn between a and b by making hydrogen bonds with other residues in d3. therefore, mutation of g271 in fgfr2 should negatively affect the tertiary folding of d3. n304, on the other hand, </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the large loop region between strand c and c, which protrudes from d3 and does not play any apparent role in the structural integrity of d3. the surfaces of the conserved cysteines (c277 and c341) in the interior core of d3 are shown as gray mesh. d. the proximity of e159 and t212 to the heparin oligosaccharide chains in the dimer is emphasized. e159 and t212 correspond to e160 and h213 in fgfr2, respectively, and mutation of these residues in melanoma could alter the interaction of mutated fgfr2s with heparin/heparan sulfate. e. the important role of d218 is highlighted in the binding of the second fgf in the 2:2 fgf-fgfr dimer and hence dimerization. mutation of e219 in fgfr2, which corresponds to d218 in fgfr1, should negatively affect the ability of the receptor to undergo ligand- and heparan sulfateinduced dimerization. atom coloring is as follows: nitrogen in blue, oxygen in red, and sulfur in yellow. hydrogen bonds are shown as dashed lines. letters n and c denote the nh2 and cooh termini of fgfr1c, respectively. ligand interacts with the second and third immunoglobulin domains (hereafter referred to as d2 and d3) and the interconnecting d2-d3 linker. the r251q mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the highly conserved d2-d3 linker region (fig. 1b). in the structure, r250, the residue homologous to r251 of fgfr2, makes three hydrogen bonds with its primary fgf2, suggesting that the r251q mutation should diminish the binding affinity of fgfr2 toward fgf2 (fig. 1b). by contrast, the e219k mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the secondary ligand binding site on d2. in the dimer, d218, which corresponds to e219 of fgfr2, makes a hydrogen bond with k35 of the secondary fgf2 ligand (fig. 1e). therefore, based on the structure, e219k should not affect the ability of fgfr2c to bind fgf2, but it should reduce the ability of fgfr2 to undergo fgf- and heparin-induced dimerization.  other extracellular mutations including v248d, g227e, and g271e are predicted to destabilize the tertiary fold of either d2 (v248d and g227e) or d3 (g271e). v247, which corresponds to v248 of fgfr2, is situated at the cooh-terminal end of the g strand in d2 where its side chain is surrounded by other hydrophobic residues inside the hydrophobic core of d2 (fig. 1a). substitution of v248 with the charged aspartic acid should be detrimental to the tertiary folding of d2. g226, which corresponds to g227 in fgfr2c, </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the start of the f strand in d2 and is spatially in close vicinity to v247 (fig. 1a). g270, which corresponds to g271 in fgfr2, </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the loop between the a and b strands in d3 (fig. 1c). each glycine residue provides hydrogen bonds that contribute to the proper -strand formation in d2 and d3, respectively. therefore, based on the structure, these substitutions should cause local structural perturbations, which could ultimately affect the folding of the whole d2 and d3, respectively. destabilization of d2 and d3 is predicted to impair the correct intramolecular disulfide bridge formation in d2 or d3, leading to exposure of the conserved cysteines and allowing them to form intermolecular disulfide-bridged receptor dimers. this destabilization can also interfere with the correct processing/maturation and trafficking of the mutated receptor through the endoplasmic reticulum (er)/golgi, resulting in receptor mislocalization.  based on the structure, the e160a and h213y mutations are likely to affect the interaction of fgfr2 with heparan sulfate. in the structure, e159 of fgfr1, which corresponds to e160 of fgfr2, facilitates the conformation of ga helix in d2, on which a key lysine residue involved in heparan sulfate binding resides (fig. 1d). therefore, destabilization of the ga helix </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to the e160a mutation should negatively affect fgfr2-heparan sulfate interaction and thereby attenuate fgf- and heparan sulfatemediated fgfr dimerization. t212, the residue corresponding to h213 of fgfr2, points into the heparin binding canyon of the 2:2:2 fgf2-fgfr1-heparan sulfate dimer (fig. 1d), and the bulkier tyrosine side chain at this location could cause steric clashes with heparan sulfate oligosaccharide molecules, leading to inefficient receptor dimerization.  kinase domain mutations. more than half of the melanoma mutations affect the conserved tyrosine kinase domain of fgfr2, and mapping of these mutations onto unphosphorylated and phosphorylated fgfr2 kinase domain structures (pdb id: 2psq and 2pvf; ref. 15) suggests that they also represent loss-of-function mutations (fig. 2 ). notably, m640 and i642 localize onto either end of the 8 strand (fig. 2), a secondary structure element that precedes the activation loop (a-loop) of the kinase domain. the hydrophobic side chains of these residues are in contact with residues in the e helix in the innermost core of the c-lobe of the kinase domain. substitutions of either residues with the smaller hydrophobic residues isoleucine or valine, respectively, would cause a cavity within the core of the kinase that should induce local structural perturbations that ultimately affect the conformation of the activation loop. the a648t mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the activation loop of the kinase domain (fig. 2e) and is also expected to lead to loss of function. this is because in the phosphorylated activated kinase domain of fgfr2 (pdb id: 2pvf; ref. 15), the methyl group of a648 makes hydrophobic contact with m537 in the c helix, which contributes to the correct positioning/orientation of the c helix relative to the cooh-terminal lobe of the kinase and, hence, to the productive alignment of the catalytic residues for the phosphotransfer reaction (15). a threonine is unfavorable at this location because its larger side chain is predicted to sterically hinder the c helix from approaching sufficiently close to the cooh-terminal lobe, and consequently, the catalytic residues would not be aligned for the phosphotransfer reaction.  figure 2. download figureopen in new tabdownload powerpoint figure 2. fgfr2c kinase domain mutations identified in melanoma tumors/cell lines mapped onto the crystal structures of the fgfr2 kinase domain suggest that these mutations should impair the kinase activity of fgfr2. the positions of the mutated residues are mapped onto the crystal structure of unphosphorylated wild-type fgfr2 kinase domain (pdb entry: 2psq; ref. 15). the coloring of the intracellular tyrosine kinase domain is as follows: the nh2-terminal lobe of kinase is in light blue; the cooh-terminal lobe is in bright blue; the activation loop is in magenta; the kinase hinge region is in green; and the nh2-terminal tail of the kinase is colored wheat. note that atp (not shown) binds in the cleft between the n-lobe and c-lobe of the kinase domain. to assist the viewer, certain  strands and  helices, which are relevant for explanation of the effects of the melanoma mutations, are labeled. a. e636, which </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the sharp turn between 7 and 8 strands, makes hydrogen bond with s563 in the kinase hinge region. mutation of this residue to lysine could influence the relative disposition of the n-lobe to the c-lobe, and hence alter the tyrosine kinase activity of fgfr2. b. the m640 and i642 side chains (red mesh) point into the inner hydrophobic core of the kinase domain and engage c606 and m614 (cyan mesh). mutation of either m640 or i642 to smaller hydrophobic residues should weaken the extent of these core hydrophobic contacts and lead to a reduction in kinase activity. c. the hydrogen bonds between r759 and side chains of negatively charged e731 (in helix h) and d756 (in helix i) are shown. in addition to the loss of these hydrogen bonds, introduction of a stop codon at this location will also truncate the helix i by one helical turn. these structural changes should reduce the stability of the kinase domain, specifically making the kinase domain temperature sensitive and ultimately leading to receptor loss of function. d. close-up view of the region where e475 is located. the side chain of e475 engages in two strong hydrogen bonds with the side chain and backbone atom of t555 (located in the loop between 4 and 5 strands), which facilitate the conformation of the 4-5 loop as well of the region preceding e475 itself. this in turn promotes the hydrophobic contacts between l468, t555 (the methyl group), and l560 from these regions and l528, v532, and m535 in the catalytically important c helix. to emphasize this point, the side chains of these hydrophobic residues are shown as colored mesh. e. close-up view of the region where d530 and a648 are located. in unphosphorylated wild-type fgfr2k structure, d530 does not play any role. however, in the a-loop phosphorylated activated fgfr2k structure (pdb entry: 2pvf; ref. 15), the side chain of d530 engages in salt bridge with r664 in the a-loop and contributes to a-loop conformation in the active state (not shown). the side chain of a648, located at the beginning of the a-loop, packs against the hydrophobic side chains of m537 in the c helix. replacement of a648 with threonine is expected to lead to loss of function because the larger side chain of threonine will impede c helix from nearing sufficiently close enough to the a-loop in the active fgfr2k. f. close-up view of the kinase region where g701 is located. g701 plays a role in stabilizing the conformation of the loop between the helices f and g, which is in hydrogen-bonding contacts with k668 in the a-loop. the g701s mutation should destabilize the f-g loop and indirectly affect the a-loop conformation. the e636k mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the loop region between the 7 and 8 strands (fig. 2a). in the structure, the side chain of e636 makes a solvent-exposed hydrogen bond with s568, which is located in the hinge region of the kinase domain, suggesting that it may play a role in regulation of the spatial positioning of the nh2-terminal and cooh-terminal lobes of the kinase relative to each other. based on the structure, the e636k mutation should influence the kinase activity of the fgfr2 kinase domain. the e475 is part of the we consensus motif that defines approximately the nh2-terminal boundary of the conserved kinase domain in many receptor tyrosine kinases. e475 engages in two strong hydrogen bonds with t555 whose methyl group is in hydrophobic contact with residues in the c helix (fig. 2d). hence, these interactions hold the c helix in an orientation found in unphosphorylated inhibited kinases. in the phosphorylated activated kinase domains, however, these hydrogen bonds are weaker and, concomitantly, the c-helix moves closer toward the c-lobe. loss of these hydrogen bonds </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to the e475k mutation could relieve this inhibition and potentially lead to activation of the kinase domain. alternatively, the e475k mutation could introduce structural instability in the n-lobe of the kinase and reduce the half life of the mutant fgfr2. the g701s mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the loop between f and g helices (fig. 2f) and facilitates the conformation of this loop region, which makes several hydrogen bonds with k668 in the activation loop. therefore, the structural perturbations induced by the g701s mutation could indirectly lead to reduced kinase activity. the r759q mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> to the cooh-terminal end of the i helix, the last secondary structure element of the kinase domain. in the structure, r759 makes hydrogen bonds with two other residues in its vicinity (fig. 2c), and loss of these hydrogen bonds </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to the r759q mutation could also induce subtle structural changes leading to a loss in kinase activity.  the l770v mutation should reduce the ability of fgfr2 to activate the phospholipase c (plc) signaling pathway. l770 follows y769, the major phosphorylation site of fgfr2 that serves as the docking site for sh2 domains of plc and thus is required for plc phosphorylation and activation by the activated fgfr2. nuclear magnetic resonance solution structure of plc sh2 domain in complex with phosphopeptide indicates that the sh2 domain of plc interacts with ptyr and residues at +1 to +6 positions relative to the ptyr. hence, the l770v mutation should reduce the affinity of sh2 domains to the activated fgfr2. the l770v mutation could also reduce y769 phosphorylation because the substrate binding pocket of the kinase domain prefers a leucine over a valine next to the tyrosine phosphorylation site (16). the reduced phosphorylation of y769 could, in turn, reduce the ability of fgfr2 to activate the plc pathway. table 2 summarizes the different mechanisms of loss of function by the melanoma mutations inferred from crystal structures. consistent with these analyses predicting that the fgfr2 mutations are loss-of-function mutations, two melanoma samples harbored nonsense mutations and a third sample carried a splicing mutation that would result in premature termination of fgfr2 protein translation (table 1).  view inlineview popup table 2. summary of in silico analysis of fgfr2 mutations analysis of fgfr2 melanoma mutations in vitro  to test our structural predictions for the extracellular mutations on fgfr2 function, we expressed recombinant wild-type and mutant ectodomains in e. coli, and </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> isolated the inclusion bodies enriched in the proteins and subjected them to in vitro refolding. as a positive control, we also used the s252w mutant ectodomain. we have previously shown that the s252w mutation has no adverse effect on folding of fgfr ectodomain (17). we were unable to refold in vitro the ectodomain proteins containing the g227e, v248d, and g271e mutations, consistent with structural predictions that these mutations are detrimental to the tertiary fold of d2 and d3 domains. by contrast, the r251q mutant ectodomain folded in vitro with yields similar to those of the wild-type and s252w mutant receptors. we </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> used surface plasmon resonance to analyze the effect of the r251q mutation on ligand binding affinity of fgfr2c. consistent with structural predictions, the r251q mutation severely impaired the ability of fgfr2c to bind multiple fgf ligands including fgf2 (fig. 3 ) and fgf1, fgf4, fgf7, fgf8, and fgf10 (data not shown). on the other hand, as expected, the s252w mutation enhanced the affinity of fgfr2c toward fgf2 and other fgfs (refs. 17-19; data not shown).  figure 3. download figureopen in new tabdownload powerpoint figure 3. surface plasmon resonance analysis shows that the r251q mutation abrogates ligand binding. a and b. increasing concentrations of wild-type (wt; a) or r251q fgfr2c mutant ectodomain (b) were injected over a cm5 chip onto which full-length human fgf2 was immobilized. the biosensor chip response is plotted as a function of time. surface plasmon resonance analysis was done as previously described (19). to test the effect of kinase domain mutations, we prepared recombinant wild-type and mutant kinase domain proteins harboring the e475k, d530n, i642v, and a648t melanoma mutations and subjected them to in vitro kinase assays. consistent with structural predictions, the d530n, i642v, and a648t mutant kinase domains exhibited reduced kinase activity relative to the wild-type kinase domain (fig. 4 ). interestingly, the e475k mutant exhibited a slight increase in kinase activity relative to the wild-type kinase domain. notably, the protein expression yield for the e475k mutant kinase was severalfold less than that of wild-type kinase, suggesting that this mutation was partially destabilizing the kinase domain.  figure 4. download figureopen in new tabdownload powerpoint figure 4. fgfr2 kinase domains harboring melanoma mutations have reduced tyrosine autophosphorylation activity. wild-type fgfr2k and four mutant fgfr2ks, each harboring e475k, d530n, i642v, or a648t single mutation, are subjected to in vitro kinase assays. columns, average of three independent experiments; bars, sd. the tyrosine autophosphorylation activities of these wild-type and mutant fgfr2ks were determined using a continuous spectrophotometric assay as previously described (43). analysis of fgfr2 mutations in cultured cells  we next analyzed the effect of several mutations (g227e, r251q, g271e, e475k, d530n, i642v, and a648t) on the receptor trafficking and maturation of the full-length fgfr2c. three different fgfr2 mutants harboring c278f, s252w, or k517r served as controls. the c278f activating mutation associated with crouzon syndrome was used to assess the effect of destabilizing d3 on both receptor maturation and activation. this mutation has been shown to result in impaired receptor processing, constitutive dimerization, and increased signaling from intracellular compartments (20). in contrast, the s252w mutation observed in patients with apert syndrome has no effect on receptor folding (19). the k517r mutant receptor served as a kinase-deficient receptor. k517 coordinates the phosphate groups of atp, and its mutation to arginine in the homologous fgfr3 has been previously shown to abolish kinase activity (21).  following transfection of wild-type fgfr2 receptor into hek293 cells, two major fgfr2-immunoreactive bands (110 and 130 kda) are observed. pngase digestion resulted in the reduction in size of both bands to a single 98-kda band (open arrow), confirming that these two bands are differently glycosylated receptor populations (fig. 5a ). endohf digestion, which is known to remove n-linked high-mannose moieties but not complex carbohydrates from glycosylated proteins, resulted in the reduction of the 110-kda band to 98 kda but had no effect on the mobility of the 130-kda band. these data show that the 110-kda form represents immature, partially processed receptor present in the er and early golgi, whereas the endohf-resistant 130-kda band represents the fully glycosylated mature receptor. densitometric analysis indicates that the endohf-resistant 130-kda mature form comprises 60% of the wild-type fgfr2 (fig. 5a and b). the r251q and s252w mutations did not reduce the relative proportion of the endoh-resistant 130-kda mature form, which is fully consistent with our structural predictions of no adverse effects on protein stability by these mutations. in contrast, only 30% of g227e, v248d, g271e, and c278f mutants were endohf resistant, suggesting impaired receptor processing of these fgfr2 mutants. this is also consistent with our structural predictions that these mutations will adversely affect the structural integrity of the d2 or d3 domain of fgfr2 (fig. 5b, top). there was a small increase in the relative proportion of the 110-kda band for the e475k mutant (fig. 5b, bottom), consistent with the decreased protein yield observed in the in vitro kinase assays.  figure 5. download figureopen in new tabdownload powerpoint figure 5. mutations in fgfr2 impair receptor processing and localization. a. confirmation of posttranslational glycosylation of wild-type fgfr2 by pngase digestion of both the 130-kda band (solid arrow) and 110-kda band (stippled arrow) to a single 98-kda band (open arrow) in hek293 cells. sensitivity to digestion with endohf of only the 110-kda band indicates this 110-kda band to be an immature, partially processed receptor predominantly present in the er and early golgi. b. endohf digestion of fgfr2 in hek293 lysates transiently transfected with wild-type fgfr2 or mutant fgfr2. densitometry analysis revealed that the endoh-resistant 130-kda band comprises 60% of the wild-type fgfr2 but only 30% of g227e, v248d, g271e, and c278f mutants. c. localization of wild-type and mutated fgfr2 following transient transfection into hek293 cells and immunofluorescent staining. impaired receptor trafficking is evident for g227e, v248d, g271e, and c278f compared with trafficking of the wild-type receptor to the cell surface. d. immunofluorescent analysis of the colocalization of fgfr2 (red, left) and the er-resident marker protein disulfide isomerase (green, middle) confirms the overall retention of the g227e mutant receptor in the er compartment in hek293 cells (yellow, right) when compared with the cell surface localization of wild-type fgfr2 and e219k (mapping onto the known extracellular crystal structure predicted the latter mutation to have no effect on misfolding). to confirm altered cellular localization of the mutants, we performed immunofluorescence studies in transiently transfected hek293 cells. in cells transfected with wild-type fgfr2, positive immunofluorescence signal was distributed primarily in the plasma membrane and also in the rough er and golgi (fig. 5c). in cells transfected with the g227e mutant (fig. 5c), however, there was positive perinuclear staining, particularly evident around the nuclear membrane, as well as a dispersed reticular staining indicative of retention in the er. immunofluorescence also showed mislocalization to intracellular membranes and decreased cell surface expression for fgfr2 mutants v248d, g271e, and c278f. in contrast, immunofluorescence of the a648t mutant shows clear cell surface localization in keeping with the structural predictions and endohf analyses. to confirm the er localization of the g227e mutant, we </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> performed similar immunofluorescence studies looking for colocalization with the er-resident protein, protein disulfide isomerase, which revealed a high degree of colocalization of the two proteins, confirming impaired receptor localization (fig. 5d). in contrast, marked cell surface localization and limited colocalization with protein disulfide isomerase were observed for both the wild-type fgfr2 and the e219k mutation, predicted to have no detrimental effect on receptor folding.  baf3 proliferation assay and p44/42 mitogen-activated protein kinase phosphorylation in response to fgf2 as readout of receptor function  we next analyzed the effects of the melanoma mutations on fgfr2c activity. given that melanocytes and melanoma cell lines express other fgfrs whose activation may mask the ability to specifically evaluate fgfr2 signaling and function, we used the murine interleukin-3 (il-3)dependent pro-b baf3 cell line to determine receptor activity. the baf3 cell line is routinely used as a model system for the evaluation of fgfr function because it does not express endogenous fgf ligands or receptors. although baf3 cell proliferation and survival is normally dependent on il-3, activated tyrosine kinase signaling can substitute for il-3 to maintain cell viability and proliferation. we included the n549k fgfr2 receptor as a constitutively activated positive control because this activating mutation has been reported in endometrial cancer (</span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">) and an identical mutation at the paralogous positions in fgfr3 has been associated with hypochondroplasia (n540k; ref. 22). polyclonal stable lines were generated following lentiviral transduction and selection in geneticin. proliferation assays were </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> done in the absence of il-3 and in the presence of 1 nmol/l fgf2 and 10 g/ml heparin, and cell viability was assayed after 5 days. as anticipated, the constitutively active n549k mutant resulted in levels of proliferation above that achieved by the wild-type receptor, whereas expression of all mutant constructs including g227e, g271e, e475k, i642v, and a648t fgfr2 resulted in reduced cell proliferation compared with wild-type fgfr2 (fig. 6a ), verifying that these mutations impaired or abrogated the ability of fgfr2 to stimulate cell proliferation in response to ligand in an in vivo setting. surprisingly, the n549k mutation did not cause constitutive baf3 proliferation in the absence of ligand stimulation.  figure 6. download figureopen in new tabdownload powerpoint figure 6. cell proliferation in response to fgf2 in baf3 cell lines stably transduced with wild-type and mutant fgfr2_ires_neo expression constructs. a. all novel mutations identified in melanoma result in a decrease in baf3 proliferation when compared with wild-type fgfr2. gray columns, basal proliferation in the absence of ligand; black columns, proliferation in response to 1 nmol/l fgf2. columns, average of at least two independent experiments assayed in triplicate. b. analysis of phospho-p44/42 mapk on stimulation with exogenous rfgf2 shows that, with the exception of the n549k positive control, mutations impair the ability of the receptor to activate this pro-proliferation pathway compared with wild-type receptor. c. western blot showing equal expression of fgfr2 by all mutants except empty vector in the absence () and presence (+) of fgf2. d. graphical representation of p44/42 mapk phosphorylation expressed as a percentage of phosphorylation (on ligand stimulation) of that of wild-type receptor. as a </span><span style=\"background-color: hsl(0, 100.00%, 98.17%); opacity: 0.80\" title=\"-0.005\">further</span><span style=\"opacity: 0.80\"> assessment of receptor impairment </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to mutation, we examined the ability of the receptor to activate the pro-proliferative p44/42 mitogen-activated protein kinase (mapk) signaling pathway in response to exogenous rfgf2. with the exception of the n549k known activating mutant, all mutations resulted in a reduction to varying levels in the ability of the receptor to induce phosphorylation of p44/42 mapk (fig. 6b-d).  reintroduction of fgfr2 into melanoma cells  we next sought to determine whether reintroduction of fgfr2 could suppress proliferation of melanoma cells. reverse transcription-pcr with isoform-specific primers and subsequent sequencing revealed normal human melanocytes and the majority of melanoma cell lines expressed the mesenchymal fgfr2c isoform (data not shown). for this reason, our reintroduction studies used the fgfr2c isoform. we overexpressed fgfr2c in melanoma cell lines that lacked fgfr2 expression by reverse transcription-pcr (sbcl2) and in melanoma cell lines that expressed mutant fgfr2 (ao4, d22, uacc2534) and assessed proliferation. surprisingly, reintroduction of fgfr2 failed to suppress the proliferation of these melanoma cell lines (fig. 7a ). expression of fgfr2 following lentiviral transduction was confirmed by western blot analysis for each cell line (fig. 7b). similar results were also achieved in both the d22 and uacc2534 melanoma cell lines (data not shown).  figure 7. download figureopen in new tabdownload powerpoint figure 7. reintroduction of fgfr2 failed to suppress proliferation of sbcl2 or ao4 melanoma cells. a. melanoma cell lines sbcl2 and ao4 were stably transduced with empty vector, fgfr2, or fgfr2 k517r lentivirus, and proliferation was assessed using the sulforhodamine b assay. overexpression of fgfr2 or kinase-dead fgfr2 k517r had no effect on the proliferation of these melanoma cell lines. b. fgfr2 expression was confirmed by western blot analysis. discussion  in this study, we report novel fgfr2 mutations in 15 of 116 melanoma cell lines and 8 of 100 uncultured metastases and primary tumors. to determine the ratio of nonsynonymous to synonymous mutations in our data set, we went back and examined the synonymous variants we identified. because we did not have constitutional dna for these samples, we excluded them as probable somatic mutations if they had been previously identified in dbsnp. several additional synonymous variants occurred alongside each other in multiple individuals, consistent with the presence of a rare germ-line haplotype block. following the exclusion of these putative snps, four synonymous variants remained; therefore, the conservative ratio of nonsynonymous to synonymous mutations in our data set was 23:4, which exceeds the 2:1 ratio one would expect if these were random passenger mutations, and supporting a role for these mutations in melanoma pathogenesis. furthermore, the mutation spectrum is characteristic of those induced by uv radiation (i.e., 70% of the mutations were c:g&gt;t:a substitutions with one cc:gg&gt;tt:aa tandem mutation). the presence of characteristic uv mutations in fgfr2 suggests that mutation of this gene is possibly an early event in transformation, and we anticipate that </span><span style=\"background-color: hsl(0, 100.00%, 98.17%); opacity: 0.80\" title=\"-0.005\">further</span><span style=\"opacity: 0.80\"> sequencing analysis in a greater number of primary tumors and nevi will shed light on when during melanoma development these mutations arise. it should be noted that the mutation frequency reported in this study may be an underestimate. this is based on the fact that 40 cell lines were screened by denaturing high-performance liquid chromatography rather than sequencing, exons 2 to 5 were not screened in this latter panel of cell lines, and not all primary tumors provided high-quality sequence for all exons.  insights gained from mapping these mutations onto known fgfr2 crystal structure, combined with in vitro and in vivo functional studies, show that these mutations lead to loss of receptor activity through different mechanisms, including loss of ligand binding (r251q), incomplete processing of the receptor and retention in the er (v248d, g227e, g271e), and complete or partial loss of tyrosine kinase activity (a648t, d530n, i642v). future studies confirming the structural predictions described herein may also reveal that fgfr2 loss of function can occur through impaired fgf- and heparan sulfateinduced receptor dimerization and reduced binding to plc.  the loss-of-function nature of our identified melanoma mutations is also consistent with published literature. in support of d530n resulting in fgfr2 loss of function, a germ-line d771n mutation at the equivalent codon in ret has been associated with hirschsprung&#x27;s disease, commonly </span><span style=\"background-color: hsl(0, 100.00%, 98.86%); opacity: 0.80\" title=\"-0.002\">due</span><span style=\"opacity: 0.80\"> to loss-of-function mutations in ret (23). in addition, while these studies were under way, the a648t fgfr2 mutation was identified in the germ line of affected patients in two independent families with autosomal dominant lacrimo-ariculo-dento-digital (ladd) syndrome (omim 149730; ref. 24). consistent with our in vitro and in vivo data, a </span><span style=\"background-color: hsl(0, 100.00%, 98.35%); opacity: 0.80\" title=\"-0.004\">recent</span><span style=\"opacity: 0.80\"> study showed that the a648t mutation abrogated the tyrosine kinase activity of fgfr2 (25). as discussed earlier, we propose that this mutation creates a steric barrier for the movement of the kinase n-lobe toward the c-lobe, which is predicted to take place when the kinase transits from the basal low activity state to the active state. however, precise definition of the mechanism by which a648t mutation impairs fgfr2 kinase activity awaits the resolution of crystal structure of the mutant kinase domain as was recently carried out for an a628t ladd mutation. in contrast to the a648t mutation, the a628t mutation </span><span style=\"background-color: hsl(0, 100.00%, 99.72%); opacity: 0.80\" title=\"-0.000\">maps</span><span style=\"opacity: 0.80\"> onto the catalytic loop of the kinase domain, and the crystal structure of the mutant kinase shows that this mutation acts by introducing steric conflicts that alter the productive arrangement of the neighboring key catalytic residues (26).  a comparison of the mutations identified in melanoma to those our lab previously identified in endometrial cancer highlights the differences in the mutation spectra (fig. 8 ). the majority of the endometrial mutations (17 of 20) are identical to fgfr2 and fgfr3 mutations previously reported in the germ line, a finding similar to that observed for fgfr3 where a comparison of fgfr3 mutations reported in cancer with germ-line mutations associated with skeletal disorders shows a remarkable concordance (reviewed in refs. 27, 28). in contrast, the more than 20 fgfr2 mutations we identified in melanoma were all novel, supporting our findings that the mutations are distinct and different from the many gain-of-function mutations previously identified in the fgfrs.  figure 8. download figureopen in new tabdownload powerpoint figure 8. schematic representation of fgfr2 mutations highlighting the spectrum difference of somatic mutations in melanoma compared with endometrial cancer, as well as germ-line mutations in craniosynostosis syndromes. a. novel somatic mutations in fgfr2 identified in melanoma cell lines and uncultured tumors are presented in red above the schematic representation of the protein and numbered relative to fgfr2c (np_000132.1). b. somatic mutations in fgfr2 identified in primary endometrial cancers and cell lines are presented in green (top) above the schematic representation of the protein and are numbered relative to fgfr2b (np_075259.2; ref. </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">). bottom, germ-line mutations associated with a variety of craniosynostosis syndromes and numbered relative to fgfr2c (np_000132.1; http://www.hgmd.cf.ac.uk/ac/index.php). novel mutations are underlined. four somatic fgfr2 endometrial mutations, although not previously reported in the germ line, have an identical missense change reported in the paralogous position in fgfr3c in a skeletal chondrodysplasia (indicated with **; http://www.hgmd.cf.ac.uk/ac/index.php). note that the majority of somatic mutations in endometrial cancer parallel those identified in the germ line, whereas every melanoma mutation was novel. this discovery of loss-of-function mutations in a receptor tyrosine kinase is unexpected, given the conventional wisdom that activation of receptor tyrosine kinases (e.g., kit, flt3, and fgfr3) drives tumorigenesis and our previous report documenting the presence of activating mutations in fgfr2 in endometrial cancer (</span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">). however, it should not be surprising given that a variety of receptor tyrosine kinase signaling pathways can induce either proliferation or differentiation depending on the cell type. indeed, cre-lox transgenic mice, with fgfr2b having been deleted in cells expressing keratin v, develop spontaneous papillomas. moreover, following 7,</span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">-dimethylbenz(a)anthracene and </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">-o-tetradecanoylphorbol-13-acetate treatment, these transgenic mice developed an increased number of papillomas and carcinomas compared with wild-type mice, providing support that fgfr2 can act as a tumor suppressor gene in some cell types (29). furthermore, loss of fgfr2 expression has been associated with several cancers including prostate and bladder (30-32). reintroduction of fgfr2 resulted in decreased growth in vitro and reduced tumorigenicity in vivo of bladder carcinoma cells (33), prostate tumor cells (34-36), and salivary adenocarcinoma cells (37).  based on these data and our structural, biochemical, and biological data supporting the characterization of the fgfr2 mutations identified in melanoma as loss of function, it was indeed surprising that reintroduction of fgfr2 failed to suppress proliferation of melanoma cell lines. there are several possible explanations for these apparent disparate results. one difference between the previously published reintroduction results and the current melanoma study is that they involve different fgfr2 splice forms. the previous studies were all carried out in cells of </span><span style=\"background-color: hsl(0, 100.00%, 99.64%); opacity: 0.80\" title=\"-0.000\">epithelial</span><span style=\"opacity: 0.80\"> origin with the fgfr2 iiib isoform, whereas our studies evaluated proliferation following reintroduction of fgfr2 iiic, the isoform of fgfr2 expressed in mesenchymal cells and in the melanocytes, which are derived from the neural crest. the lack of suppression of proliferation of melanoma cells following overexpression of fgfr2 may therefore reflect a cell lineagespecific function of fgfr2. another possible explanation is that loss or abrogation of fgfr2 is an early event and those melanoma cell lines have acquired additional genetic aberrations that prevent suppression of proliferation following fgfr2 reintroduction. alternatively, abrogation of fgfr2 may not contribute to melanoma pathogenesis by removing a growth inhibitory signal. indeed, although tumor suppressors classically function to regulate cell cycle, accumulating evidence suggests that additional subsets of genes exist, which function to suppress tumorigenesis not by inhibiting proliferation but by altering other aspects of the malignant phenotype, including loss of contact inhibition, angiogenesis, cell migration, and metastasis. therefore, the pathogenic </span><span style=\"background-color: hsl(120, 100.00%, 95.15%); opacity: 0.81\" title=\"0.020\">consequence</span><span style=\"opacity: 0.80\"> of loss of fgfr2 function may not directly involve cell cycle regulation but instead affect another aspect of tumorigenesis. the caveat exists that loss of fgfr2 might not be a driver of cell transformation but rather these mutations occur as a </span><span style=\"background-color: hsl(120, 100.00%, 95.15%); opacity: 0.81\" title=\"0.020\">consequence</span><span style=\"opacity: 0.80\"> of the oncogenic state; however, their selection in 10% of melanoma samples would argue that they must provide a selective advantage to the cells in which they arose. this is consistent with a relatively high frequency and high nonsynonymous to synonymous ratio arguing against their presence as passenger mutations.  elucidation of the mechanism by which abrogation of fgfr2 contributes to melanoma is also complicated by our current lack of understanding about the differing cellular functions of the multiple splice forms of these receptors, particularly in the melanocytic system. there are additional splice variants affecting the ligand binding domain (2 ig, 3 ig), cytoplasmic juxtamembrane domain (+vt, vt), and cooh-terminal domain of fgfr2 (c1, c3), and the role of these splice variants in normal melanocyte function and melanoma pathogenesis is currently unknown. our lab is currently investigating the relative expression levels of these various isoforms and their functional significance in melanocytes and melanoma. fgfr2 may also have cellular functions distinct from typical receptor tyrosine kinase signaling. there have been reports that fgfr1 binds cbp and rsk in the nucleus (38), and we and others have observed that fgfr2 localized to the nucleus in various cell types (39).8 lastly, we and others have shown that fgfr2 can heterodimerize with fgfr1 at least in vitro (40)8 and fgfrs have also been shown to bind to other molecules at the cell surface, including neural cell adhesion molecule (41). we postulate that a more complete understanding of fgfr biology will allow us to identify the mechanism by which loss of fgfr2 contributes to melanoma initiation and/or progression.  in conclusion, we provide strong genetic, biochemical, in vitro and in vivo evidence that loss-of-function fgfr2 mutations occur in a subset of melanomas. although we have yet to uncover the biological </span><span style=\"background-color: hsl(120, 100.00%, 95.15%); opacity: 0.81\" title=\"0.020\">consequence</span><span style=\"opacity: 0.80\"> of loss of fgfr2 function in melanoma, we propose that fgfr2 should join the list of those genes that play context-dependent opposing roles in tumorigenesis. although many genes have been shown to have context-dependent opposing roles in cancer (e.g., notch and tgf; ref. 42), fgfr2 may be the first gene in which both loss-of-function and gain-of-function mutations have been reported in different tissue types. ongoing work in our laboratory is targeted toward understanding the molecular mechanisms by which loss of fgfr2 contributes to melanoma pathogenesis.  materials and methods  see supplementary methods for details on mutation detection.  expression, refolding, and purification of ectodomain of fgfr2  dna fragment encoding residues 147 to 366 of fgfr2c was amplified by pcr and subcloned into pet-28a bacterial expression vector using ncoi and hindiii cloning sites. point mutations (g227e, v248d, r251q, s252w, and g271e) were introduced using quikchange site-directed mutagenesis kit (stratagene). bl21(de3) competent cells were transformed with wild-type or mutant expression constructs and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 5 h at 37c. cells were lysed using french press and centrifuged; the inclusion bodies containing wild-type or mutant proteins were dissolved in 6 mol/l guanidinium hydrochloride plus 10 mmol/l dtt in 100 mmol/l tris-hcl buffer (ph 8.0). the solubilized fgfr2 proteins were subjected to an in vitro refolding by slow dialysis against 25 mmol/l hepes buffer (ph 7.5) containing 150 mmol/l nacl, 10% glycerol, and l-cysteine. the correctly refolded wild-type and mutant proteins were </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> sequentially purified by heparin affinity and size exclusion chromatography. surface plasmon resonance analysis was done as described previously (19).  expression and purification of fgfr2 kinase domain  the dna fragment encoding residues 458 to 768 of human fgfr2 was subcloned into pet bacterial expression vectors with an nh2-terminal 6his-tag to aid in protein purification. point mutations (e475k, d530n, i642v, and a648t) were introduced using quikchange site-directed mutagenesis kit (stratagene). the bacterial strain bl21(de3) cells were transformed with wild-type or mutant fgfr2 kinase expression constructs, and protein expression was induced with 1 mmol/l isopropyl-l-thio-b-d-galactopyranoside for 4 h at ambient temperature. the cells were </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> lysed using a french press, and the soluble kinase proteins were purified by using sequential ni2+-chelating, anion exchange and size exclusion chromatography. the purity of the proteins was estimated to be &gt;98% based on sds-page analysis. purified wild-type and mutant kinase proteins were concentrated to at least 10 mg/ml using a centricon-10. the autophosphorylation activity of the wild-type and mutant kinases was compared using a continuous spectrophotometric kinase assay according to the published protocol (43).  site-directed mutagenesis of full-length fgfr2  all mutations were introduced to the pcdna3_fgfr2 wild-type plasmid (nm_000141) using the quikchange ii xl site-directed mutagenesis kit (stratagene) according to the manufacturer&#x27;s instructions. additionally, mutagenesis primers were designed to introduce a novel but silent (noncoding change) restriction enzyme recognition site to allow for fast, preliminary screening of clones. the full list of primer sequences can be found in supplementary table s1. after restriction enzyme screening, the entire coding sequence of fgfr2 was sequenced for each clone to confirm the presence of the intended mutation and to ensure that no other mutations were introduced during the mutagenesis </span><span style=\"background-color: hsl(0, 100.00%, 73.86%); opacity: 0.91\" title=\"-0.217\">process</span><span style=\"opacity: 0.80\">. plasmid dna was </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> isolated using qiagen endofree plasmid dna isolation kits, and dna with a260/a280 &gt; 1.8 was used in all subsequent experiments.  cell lines and transfections  all cell lines described in this series of experiments were maintained under standard culturing conditions as described by the american type culture collection or deutsche sammlung von mikroorganismen und zellkulturen (baf3 cell line). transient transfections of 1 g total dna (consisting of 250 g wild-type or mutant fgfr2 + 750 g empty plasmid) were achieved with fugene 6 (roche) at a 3:1 ratio of reagent/dna according to the manufacturer&#x27;s instructions.  cell lysis, endoglycosidase digestions, and western blotting  at 36 to 48 h posttransfection, cells were washed with ice-cold pbs and lysed in ice-cold lysis buffer [20 mmol/l hepes (ph 7.4), 1% triton x-100, 2 mmol/l egta, 10% glycerol, complete protease inhibitor cocktail (roche), 1 mmol/l naf, 1 mmol/l na3vo4, 100 mol/l phenylmethylsulfonyl fluoride]. for pngase and endohf digestions, 20 g of lysate were subjected to digestion according to the manufacturer&#x27;s recommendations. protein was resolved on 3% to 8% tris-acetate gels and transferred onto polyvinylidene difluoride membranes (invitrogen). after blocking the membranes with 5% skim milk/tbst, membranes were probed with 1:2,000 bek c-17 (santa cruz biotechnology).  immunofluorescence  cells were transfected as described and plated onto eight-well chamber slides. after 36 to 48 h, cells were fixed with 4% paraformaldehyde, permeabilized in 0.2% triton, and incubated with 1:300 bek c-17 and 1:1,000 antiprotein disulfide isomerase (invitrogen) and </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> detected with goat ant-rabbit cy3 (1:200; jackson immunoresearch) and goat anti-mouse alexa fluor 488 (1:1,000; invitrogen), respectively. slides were mounted with pro long gold antifade reagent and imaged under a zeiss lsm 5 pascal confocal microscope.  lentivirus production  293ft cells on poly-d-lysinecoated plates were transfected with lentiviral packaging vectors and ptyf.fgfr2.ires.neo (or mutant fgfr2) using superfect (qiagen) at a 4:1 superfect/dna ratio as per manufacturer&#x27;s instructions. media containing the virus were collected at 24 and 40 h, pooled, filtered through a 0.45-um low protein binding filter, and stored at 80c until use. viral stocks were titered in hek293 cells following serial dilution in a six-well plate, selection for 14 d in 800 g/ml geneticin, staining with crystal violet, and colony counting.  baf3 proliferation assays  baf3 cells (5  105) were infected with a multiplicity of infection of 1 (5  105 viral transducing units) for 23 h, </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> placed under selective pressure (1,200 g/ml geneticin) for 14 d in the presence of 5 ng/ml il-3. transduced, stably selected baf3 cells containing wild-type and mutant fgfr2 were maintained under selection in rpmi/10% fetal bovine serum containing 50 nmol/l -mercaptoethanol, 100 units/ml penicillin/100 g/ml streptomycin sulfate, 1,000 g/ml geneticin, and supplemented with 5 ng/ml murine il-3 (r&amp;d sytems). before the proliferation assay, cells were counted and washed in the above media containing no il-3. cells were plated at 1  104 per well in triplicate in a 96-well plate in media containing 16.7 ng/ml (1 nmol/l) fgf2 and 10 g/ml heparin. half the volume of media was removed at day 3 and replaced with an equal volume containing fresh ligand at 2 concentration and no il-3. proliferation was assessed using the vialight plus cell proliferation/cytotoxity kit (lonza rockland, inc.) according to the manufacturer&#x27;s instructions.  melanoma proliferation assays  sbcl2 melanoma cells were cultured in 2% tumor medium containing a 4:1 mixture of mcdb 153 medium with 1.5 g/l sodium bicarbonate and leibovitz&#x27;s l-15 medium with 2 mmol/l l-glutamine supplemented with 0.005 mg/ml bovine insulin, 1.68 mmol/l cacl2, and 2% fetal bovine serum. ao4, d22, and uacc2534 cells were grown in rpmi supplemented with 10% fetal bovine serum. sbcl2, ao4, d22, and uacc2534 melanoma cells were transduced with empty vector, fgfr2, or kinase-dead fgfr2 (k517r) at a multiplicity of infection of 2 in the presence of 6 g/ml polybrene. cells were selected for at least 14 d in 500 to 800 g/ml g418. cells (2,000-4,000) were plated per well in a 96-well plate in full growth media. the next day, cells were washed in pbs and switched to low-serum media overnight (0.5% fetal bovine serum for sbcl2 and 1% for ao4, d22, and uacc2534). cells were </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> stimulated with 10 ng/ml fgf2 and 10 g/ml heparin in low-serum media and proliferation was assessed using the sulforhodamine b assay. fresh fgf2 and heparin were added on day 4.  p44/42 phosphorylation  to examine p44/42 mapk phosphorylation, cells were washed in pbs and resuspended in baf media containing no fetal bovine serum and no il-3 (baf starve media) for 2 h. cells were pelleted and resuspended in either baf starve media with 10 g/ml heparin only (unstimulated or ) or baf starve media with heparin and 1nmol/l rfgf2 (stimulated or +) for 10 min at 37c. after 10 min, the cells were pelleted by brief centrifugation and immediately lysed in ice-cold lysis buffer. twenty micrograms of total protein were resolved on a 4% to </span><span style=\"background-color: hsl(0, 100.00%, 99.71%); opacity: 0.80\" title=\"-0.000\">12</span><span style=\"opacity: 0.80\">% bis tris gel and probed for phosho-p44/42 mapk. blots were </span><span style=\"background-color: hsl(0, 100.00%, 98.59%); opacity: 0.80\" title=\"-0.003\">then</span><span style=\"opacity: 0.80\"> stripped and reprobed for total p44/42 mapk (cell signaling technologies).  phosphorylation was quantitated using image j9 and presented as the ratio of phosphorylated over total p44/42 mapk for each mutant divided by the same ratio for the fgf2-stimulated wild-type receptor.</span>\n",
       "    </p>\n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "    \n",
       "\n",
       "\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "execution_count": 226,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Analyzing individual predictions. Let's check some predictions from the validation set. You see a summary of various vectorizer's contribution at the top, and then below you can see features highlighed in text.\n",
    "eli5.show_prediction(model, doc=X.values[1], vec=vectorizer)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 3.2 Word2vec + Model"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3.2.1 Document-trained w2vec + xgboost"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [],
   "source": [
    "clf = Pipeline([('vect', MeanEmbeddingVectorizer(w2vec)),\n",
    "                ('clf',  xgb.XGBClassifier(objective=\"multi:softprob\", random_state=42))])                           "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0.4626865671641791\n"
     ]
    }
   ],
   "source": [
    "clf.fit(X_tr, y_tr)  \n",
    "predicted = clf.predict(X_val)\n",
    "acc=np.mean(predicted == y_val)\n",
    "print(acc)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Lessons learned\n",
    "\n",
    " * Text pre-processing helps \n",
    " * W2vec: domain corpus helps \n",
    " * A small document: w2vec may not outperform tfidf\n",
    " * One-hot encodings is not good for tree based methods \n",
    " * Knowing what featurews are useful/useless for prediction is helpful \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
